<!DOCTYPE html>
<html lang="en-US">
<head>
<!-- Original URL: http://www.aimsci.com/ros/html/?page_id=1560
Date Downloaded: 12/25/2019 3:40:10 PM !-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	<meta charset="UTF-8" />
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
	<link rel="pingback" href="http://www.aimsci.com/ros/html/xmlrpc.php" />

	<script type="text/javascript">
		document.documentElement.className = 'js';
	</script>

	<script>var et_site_url='index.html';var et_post_id='1560';function et_core_page_resource_fallback(a,b){"undefined"===typeof b&&(b=a.sheet.cssRules&&0===a.sheet.cssRules.length);b&&(a.onerror=null,a.onload=null,a.href?a.href=et_site_url+"/?et_core_page_resource="+a.id+et_post_id:a.src&&(a.src=et_site_url+"/?et_core_page_resource="+a.id+et_post_id))}
</script><title>2017; 4(12):389–417 | ss_site_title</title>
<link rel='dns-prefetch' href='//fonts.googleapis.com' />
<link rel='dns-prefetch' href='//s.w.org' />
<link rel="alternate" type="application/rss+xml" title="ss_site_title &raquo; Feed" href="_c0b1fbb3" />
<link rel="alternate" type="application/rss+xml" title="ss_site_title &raquo; Comments Feed" href="_387b1518" />
		<script type="text/javascript">
			window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/11\/72x72\/","ext":".png","svgUrl":"https:\/\/s.w.org\/images\/core\/emoji\/11\/svg\/","svgExt":".svg","source":{"concatemoji":"wp-includes/js/wp-emoji-release.min_e0220a6d.js"}};
			!function(a,b,c){function d(a,b){var c=String.fromCharCode;l.clearRect(0,0,k.width,k.height),l.fillText(c.apply(this,a),0,0);var d=k.toDataURL();l.clearRect(0,0,k.width,k.height),l.fillText(c.apply(this,b),0,0);var e=k.toDataURL();return d===e}function e(a){var b;if(!l||!l.fillText)return!1;switch(l.textBaseline="top",l.font="600 32px Arial",a){case"flag":return!(b=d([55356,56826,55356,56819],[55356,56826,8203,55356,56819]))&&(b=d([55356,57332,56128,56423,56128,56418,56128,56421,56128,56430,56128,56423,56128,56447],[55356,57332,8203,56128,56423,8203,56128,56418,8203,56128,56421,8203,56128,56430,8203,56128,56423,8203,56128,56447]),!b);case"emoji":return b=d([55358,56760,9792,65039],[55358,56760,8203,9792,65039]),!b}return!1}function f(a){var c=b.createElement("script");c.src=a,c.defer=c.type="text/javascript",b.getElementsByTagName("head")[0].appendChild(c)}var g,h,i,j,k=b.createElement("canvas"),l=k.getContext&&k.getContext("2d");for(j=Array("flag","emoji"),c.supports={everything:!0,everythingExceptFlag:!0},i=0;i<j.length;i++)c.supports[j[i]]=e(j[i]),c.supports.everything=c.supports.everything&&c.supports[j[i]],"flag"!==j[i]&&(c.supports.everythingExceptFlag=c.supports.everythingExceptFlag&&c.supports[j[i]]);c.supports.everythingExceptFlag=c.supports.everythingExceptFlag&&!c.supports.flag,c.DOMReady=!1,c.readyCallback=function(){c.DOMReady=!0},c.supports.everything||(h=function(){c.readyCallback()},b.addEventListener?(b.addEventListener("DOMContentLoaded",h,!1),a.addEventListener("load",h,!1)):(a.attachEvent("onload",h),b.attachEvent("onreadystatechange",function(){"complete"===b.readyState&&c.readyCallback()})),g=c.source||{},g.concatemoji?f(g.concatemoji):g.wpemoji&&g.twemoji&&(f(g.twemoji),f(g.wpemoji)))}(window,document,window._wpemojiSettings);
		</script>
		<meta content="Divi v.3.19.7" name="generator" /><style type="text/css">
img.wp-smiley,
img.emoji {
	display: inline !important;
	border: none !important;
	box-shadow: none !important;
	height: 1em !important;
	width: 1em !important;
	margin: 0 .07em !important;
	vertical-align: -0.1em !important;
	background: none !important;
	padding: 0 !important;
}
</style>
<link rel='stylesheet' id='wp-block-library-css' href='wp-includes/css/dist/block-library/style.min_e0220a6d.css' type='text/css' media='all' />
<link rel='stylesheet' id='divi-fonts-css' href='../../../fonts.googleapis.com/css_6b0aa6b1.css' type='text/css' media='all' />
<link rel='stylesheet' id='divi-style-css' href='wp-content/themes/Divi/style_1e2ae7de.css' type='text/css' media='all' />
<link rel='stylesheet' id='dashicons-css' href='wp-includes/css/dashicons.min_e0220a6d.css' type='text/css' media='all' />
<script type='text/javascript' src='wp-includes/js/jquery/jquery_85ec8bd8.js'></script>
<script type='text/javascript' src='wp-includes/js/jquery/jquery-migrate.min_e021f9e6.js'></script>
<link rel='https://api.w.org/' href='index_ba315afa.php' />
<link rel="EditURI" type="application/rsd+xml" title="RSD" href="xmlrpc_b9b66041.php" />
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="wp-includes/wlwmanifest.xml" /> 
<meta name="generator" content="WordPress 5.0.8" />
<link rel="canonical" href="_500daa5e.html" />
<link rel='shortlink' href='_500daa5e.html' />
<link rel="alternate" type="application/json+oembed" href="index_6b89d1a9.php" />
<link rel="alternate" type="text/xml+oembed" href="index_865870d0.php" />
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" /><link rel="stylesheet" id="et-divi-customizer-global-cached-inline-styles" href="wp-content/cache/et/global/et-divi-customizer-global-15766348400072.min.css" onerror="et_core_page_resource_fallback(this, true)" onload="et_core_page_resource_fallback(this)" /></head>
<body class="page-template-default page page-id-1560 et_pb_button_helper_class et_fixed_nav et_show_nav et_cover_background et_pb_gutter windows et_pb_gutters3 et_primary_nav_dropdown_animation_fade et_secondary_nav_dropdown_animation_fade et_pb_footer_columns4 et_header_style_left et_right_sidebar et_divi_theme et-db et_minified_js et_minified_css">
	<div id="page-container">

	
	
			<header id="main-header" data-height-onload="66">
			<div class="container clearfix et_menu_container">
							<div class="logo_container">
					<span class="logo_helper"></span>
					<a href="index.html">
						<img src="wp-content/uploads/2016/03/ROS-blue-square.png" alt="ss_site_title" id="logo" data-height-percentage="54" />
					</a>
				</div>
							<div id="et-top-navigation" data-height="66" data-fixed-height="40">
											<nav id="top-menu-nav">
						<ul id="top-menu" class="nav"><li id="menu-item-1503" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1503"><a href="_ad6635bd.html">2016</a>
<ul class="sub-menu">
	<li id="menu-item-168" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-168"><a href="_2e83399d.html">2016 Vol 1 No 1</a>
	<ul class="sub-menu">
		<li id="menu-item-155" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-155"><a href="_468cdbca.html">2016; 1(1):1–8</a></li>
		<li id="menu-item-154" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-154"><a href="_468cdbcc.html">2016; 1(1):9–21</a></li>
		<li id="menu-item-153" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-153"><a href="_468cdbbe.html">2016; 1(1):22–37</a></li>
		<li id="menu-item-152" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-152"><a href="_73d222cd.html">2016; 1(1):38–52</a></li>
		<li id="menu-item-151" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-151"><a href="_73d222cf.html">2016; 1(1):53–58</a></li>
		<li id="menu-item-150" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-150"><a href="_73d222d1.html">2016; 1(1):59–64</a></li>
		<li id="menu-item-149" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-149"><a href="_73d222d3.html">2016; 1(1):65–80</a></li>
		<li id="menu-item-165" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-165"><a href="_2e833998.html">2016; 1(1):81–98</a></li>
	</ul>
</li>
	<li id="menu-item-208" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-208"><a href="_d12aae32.html">2016 Vol 1 No 2</a>
	<ul class="sub-menu">
		<li id="menu-item-207" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-207"><a href="_d12aae34.html">2016; 1(2):99–109</a></li>
		<li id="menu-item-206" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-206"><a href="_d12aae36.html">2016; 1(2):110–116</a></li>
		<li id="menu-item-205" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-205"><a href="_d12aae38.html">2016; 1(2):117–130</a></li>
		<li id="menu-item-204" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-204"><a href="_d12aae2a.html">2016; 1(2):131–140</a></li>
		<li id="menu-item-203" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-203"><a href="_d3f8ab89.html">2016; 1(2):141–156</a></li>
		<li id="menu-item-202" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-202"><a href="_d3f8ab8b.html">2016; 1(2):157–164</a></li>
		<li id="menu-item-201" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-201"><a href="_d3f8ab8d.html">2016; 1(2):165–177</a></li>
	</ul>
</li>
	<li id="menu-item-289" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-289"><a href="_d3f8aba4.html">2016 Vol 1 No 3</a>
	<ul class="sub-menu">
		<li id="menu-item-273" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-273"><a href="_d12aae53.html">2016; 1(3):178–188</a></li>
		<li id="menu-item-277" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-277"><a href="_d12aae57.html">2016; 1(3):189–198</a></li>
		<li id="menu-item-280" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-280"><a href="_d12aae4c.html">2016; 1(3):199–206</a></li>
		<li id="menu-item-517" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-517"><a href="_8bdbc57e.html">2016; 1(3):207–215</a></li>
		<li id="menu-item-286" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-286"><a href="_d3f8abaf.html">2016; 1(3):216–227</a></li>
		<li id="menu-item-292" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-292"><a href="_76a02046.html">2016; 1(3):228–244</a></li>
	</ul>
</li>
	<li id="menu-item-587" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-587"><a href="_d3f8ac0b.html">2016 Vol 2 No 4</a>
	<ul class="sub-menu">
		<li id="menu-item-607" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-607"><a href="_e9345108.html">2016; 2(4):245–263</a></li>
		<li id="menu-item-576" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-576"><a href="_d12aaeb3.html">2016; 2(4):264–271</a></li>
		<li id="menu-item-677" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-677"><a href="_2e833a37.html">2016; 2(4):272–289</a></li>
		<li id="menu-item-651" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-651"><a href="_73d2236e.html">2016; 2(4):290–297</a></li>
		<li id="menu-item-641" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-641"><a href="_468cdc66.html">2016; 2(4):298–307</a></li>
		<li id="menu-item-692" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-692"><a href="_d3f8ac2e.html">2016; 2(4):308–314</a></li>
	</ul>
</li>
	<li id="menu-item-735" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-735"><a href="_468cdc8b.html">2016 Vol 2 No 5</a>
	<ul class="sub-menu">
		<li id="menu-item-741" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-741"><a href="_468cdc85.html">2016; 2(5):315–324</a></li>
		<li id="menu-item-751" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-751"><a href="_73d2238b.html">2016; 2(5):325–338</a></li>
		<li id="menu-item-766" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-766"><a href="_2e833a59.html">2016; 2(5):339–354</a></li>
		<li id="menu-item-781" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-781"><a href="_d12aaef1.html">2016; 2(5):355–360</a></li>
		<li id="menu-item-807" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-807"><a href="_e9344f3d.html">2016; 2(5):361–370</a></li>
		<li id="menu-item-713" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-713"><a href="_8bdbc5c3.html">2016; 2(5):371–387</a></li>
	</ul>
</li>
	<li id="menu-item-844" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-844"><a href="_73d221a8.html">2016 Vol 2 No 6</a>
	<ul class="sub-menu">
		<li id="menu-item-868" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-868"><a href="_2e833876.html">2016; 2(6):388–395</a></li>
		<li id="menu-item-879" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-879"><a href="_d12aad11.html">2016; 2(6):396–403</a></li>
		<li id="menu-item-914" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-914"><a href="_8bdbc3f6.html">2016; 2(6):404–416</a></li>
		<li id="menu-item-886" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-886"><a href="_d3f8aa66.html">2016; 2(6):417–420</a></li>
		<li id="menu-item-834" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-834"><a href="_468cdaa1.html">2016; 2(6):421–431</a></li>
		<li id="menu-item-897" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-897"><a href="_76a01f00.html">2016; 2(6):432–439</a></li>
	</ul>
</li>
</ul>
</li>
<li id="menu-item-1507" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-ancestor current_page_ancestor menu-item-has-children menu-item-1507"><a href="_22c86351.html">2017</a>
<ul class="sub-menu">
	<li id="menu-item-940" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-940"><a href="_468cdac6.html">2017 Vol 3 No 7</a>
	<ul class="sub-menu">
		<li id="menu-item-932" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-932"><a href="_a3e5662c.html">2017; 3(7):1–11</a></li>
		<li id="menu-item-1007" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1007"><a href="_5141f038.html">2017; 3(7):12–25</a></li>
		<li id="menu-item-1017" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1017"><a href="_5141f039.html">2017; 3(7):26–37</a></li>
		<li id="menu-item-1030" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1030"><a href="_ec1045b.html">2017; 3(7):38–46</a></li>
		<li id="menu-item-944" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-944"><a href="_73d221c7.html">2017; 3(7):47–65</a></li>
		<li id="menu-item-1039" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1039"><a href="_bf30705.html">2017; 3(7):66–80</a></li>
	</ul>
</li>
	<li id="menu-item-1074" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1074"><a href="_deadc002.html">2017 Vol 3 No 8</a>
	<ul class="sub-menu">
		<li id="menu-item-1069" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1069"><a href="_bf30708.html">2017; 3(8):81–95</a></li>
		<li id="menu-item-1132" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1132"><a href="_71b58a6f.html">2017; 3(8):96–110</a></li>
		<li id="menu-item-1128" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1128"><a href="_b70473a4.html">2017; 3(8):111–120</a></li>
		<li id="menu-item-1112" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1112"><a href="_71b58a6d.html">2017; 3(8):121–126</a></li>
		<li id="menu-item-1098" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1098"><a href="_694b9260.html">2017; 3(8):127–134</a></li>
		<li id="menu-item-1084" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1084"><a href="_deadbff3.html">2017; 3(8):135–147</a></li>
	</ul>
</li>
	<li id="menu-item-1174" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1174"><a href="_2c66a13d.html">2017 Vol 3 No 9</a>
	<ul class="sub-menu">
		<li id="menu-item-1194" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1194"><a href="_2c66a12f.html">2017; 3(9):148–183</a></li>
		<li id="menu-item-1286" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1286"><a href="_1cc6f704.html">2017; 3(9):184–188</a></li>
		<li id="menu-item-1177" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1177"><a href="_cf0e15d8.html">2017; 3(9):189–196</a></li>
		<li id="menu-item-1186" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1186"><a href="_fc535cd0.html">2017; 3(9):197–207</a></li>
		<li id="menu-item-1299" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1299"><a href="_a764c971.html">2017; 3(9):208–217</a></li>
		<li id="menu-item-1312" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1312"><a href="_82897a77.html">2017; 3(9):218–236</a></li>
	</ul>
</li>
	<li id="menu-item-1347" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1347"><a href="_6a7fd84b.html">2017 Vol 4 No 10</a>
	<ul class="sub-menu">
		<li id="menu-item-1352" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1352"><a href="_82897a7b.html">2017; 4(10):237–250</a></li>
		<li id="menu-item-1379" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1379"><a href="_afcec184.html">2017; 4(10):251–265</a></li>
		<li id="menu-item-1400" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1400"><a href="_6d4dd597.html">2017; 4(10):266–274</a></li>
		<li id="menu-item-1360" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1360"><a href="_6d4dd5a3.html">2017; 4(10):275–283</a></li>
		<li id="menu-item-1371" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1371"><a href="_caa66109.html">2017; 4(10):284–289</a></li>
		<li id="menu-item-1388" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1388"><a href="_6a7fd83f.html">2017; 4(10):290–297</a></li>
	</ul>
</li>
	<li id="menu-item-1423" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1423"><a href="_a4fc3dba.html">2017 Vol 4 No 11</a>
	<ul class="sub-menu">
		<li id="menu-item-1419" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1419"><a href="_d50f8217.html">2017; 4(11):298–302</a></li>
		<li id="menu-item-1429" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1429"><a href="_d50f8218.html">2017; 4(11):303–319</a></li>
		<li id="menu-item-1438" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1438"><a href="_32680d7e.html">2017; 4(11):320–335</a></li>
		<li id="menu-item-1458" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1458"><a href="_32680d80.html">2017; 4(11):336–350</a></li>
		<li id="menu-item-1478" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1478"><a href="_32680d82.html">2017; 4(11):351–361</a></li>
		<li id="menu-item-1490" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1490"><a href="_ed19243d.html">2017; 4(11):362–371</a></li>
	</ul>
</li>
	<li id="menu-item-1530" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-ancestor current-menu-parent current_page_parent current_page_ancestor menu-item-has-children menu-item-1530"><a href="_3ad20582.html">2017 Vol 4 No 12</a>
	<ul class="sub-menu">
		<li id="menu-item-1591" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1591"><a href="_982a90dd.html">2017; 4(12):372–381</a></li>
		<li id="menu-item-1551" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1551"><a href="_982a90e9.html">2017; 4(12):382–388</a></li>
		<li id="menu-item-1562" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-1560 current_page_item menu-item-1562"><a href="_500daa5e.html">2017; 4(12):389–417</a></li>
		<li id="menu-item-1538" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1538"><a href="_8020eeb9.html">2017; 4(12):418–433</a></li>
		<li id="menu-item-1605" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1605"><a href="_fb1f16f8.html">2017; 4(12):434–440</a></li>
		<li id="menu-item-1526" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1526"><a href="_c56fd7ee.html">2017; 4(12):441–444</a></li>
	</ul>
</li>
</ul>
</li>
<li id="menu-item-1679" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1679"><a href="_70814493.html">2018</a>
<ul class="sub-menu">
	<li id="menu-item-1667" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1667"><a href="_888ae6c1.html">2018 Vol 5 No 13</a>
	<ul class="sub-menu">
		<li id="menu-item-1671" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1671"><a href="_a09488f0.html">2018; 5(13):1–14</a></li>
		<li id="menu-item-1757" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1757"><a href="_339c5360.html">2018; 5(13):15–34</a></li>
		<li id="menu-item-1742" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1742"><a href="_a8fe80f2.html">2018; 5(13):35–45</a></li>
		<li id="menu-item-1683" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1683"><a href="_b5d02dba.html">2018; 5(13):46–55</a></li>
		<li id="menu-item-1703" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1703"><a href="_be3a25c7.html">2018; 5(13):56–67</a></li>
		<li id="menu-item-1714" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1714"><a href="_a6308399.html">2018; 5(13):68–77</a></li>
	</ul>
</li>
	<li id="menu-item-1810" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1810"><a href="_7c880e31.html">2018 Vol 5 No 14</a>
	<ul class="sub-menu">
		<li id="menu-item-1814" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1814"><a href="_21fd8024.html">2018; 5(14):78–85</a></li>
		<li id="menu-item-1835" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1835"><a href="_c4a4f4c1.html">2018; 5(14):86–106</a></li>
		<li id="menu-item-1860" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1860"><a href="_7c880e36.html">2018; 5(14):107–117</a></li>
		<li id="menu-item-1802" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1802"><a href="_a8fe80e8.html">2018; 5(14):118–125</a></li>
		<li id="menu-item-1847" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1847"><a href="_9491b064.html">2018; 5(14):126–133</a></li>
		<li id="menu-item-1870" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1870"><a href="_7c880e37.html">2018; 5(14):134–144</a></li>
	</ul>
</li>
	<li id="menu-item-1902" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1902"><a href="_b5054a94.html">2018 Vol 5 No 15</a>
	<ul class="sub-menu">
		<li id="menu-item-1934" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1934"><a href="_6fb66161.html">2018; 5(15):145–158</a></li>
		<li id="menu-item-1956" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1956"><a href="_3fa31d05.html">2018; 5(15):159–166</a></li>
		<li id="menu-item-1898" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1898"><a href="_ac9b5288.html">2018; 5(15):167–175</a></li>
		<li id="menu-item-1911" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1911"><a href="_27997ad1.html">2018; 5(15):176–198</a></li>
		<li id="menu-item-1924" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1924"><a href="_6fb66160.html">2018; 5(15):199–208</a></li>
		<li id="menu-item-1944" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1944"><a href="_6fb66162.html">2018; 5(15):209–219</a></li>
	</ul>
</li>
	<li id="menu-item-1993" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1993"><a href="_125dd5f2.html">2018 Vol 6 No 16</a>
	<ul class="sub-menu">
		<li id="menu-item-2036" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2036"><a href="_ae9a7bbd.html">2018; 6(16):220–247</a></li>
		<li id="menu-item-2007" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2007"><a href="_5141f05b.html">2018; 6(16):248–252</a></li>
		<li id="menu-item-1989" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1989"><a href="_fa5433c2.html">2018; 6(16):253–266</a></li>
		<li id="menu-item-2020" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2020"><a href="_ec1047b.html">2018; 6(16):267–275</a></li>
		<li id="menu-item-2046" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2046"><a href="_ae9a7bc4.html">2018; 6(16):276–288</a></li>
		<li id="menu-item-2056" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2056"><a href="_ae9a7bc3.html">2018; 6(16):289–298</a></li>
	</ul>
</li>
	<li id="menu-item-2086" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2086"><a href="_ae9a7bb8.html">2018 Vol 6 No 17</a>
	<ul class="sub-menu">
		<li id="menu-item-2156" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2156"><a href="_89bf2cc1.html">2018; 6(17):299–310</a></li>
		<li id="menu-item-2082" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2082"><a href="_23fca94c.html">2018; 6(17):311–324</a></li>
		<li id="menu-item-2102" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2102"><a href="_71b58a8f.html">2018; 6(17):325–337</a></li>
		<li id="menu-item-2119" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2119"><a href="_145cff29.html">2018; 6(17):338–348</a></li>
		<li id="menu-item-2134" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2134"><a href="_2c66a156.html">2018; 6(17):349–362</a></li>
		<li id="menu-item-2177" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2177"><a href="_59abe861.html">2018; 6(17):363–370</a></li>
		<li id="menu-item-2169" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2169"><a href="_b70473c7.html">2018; 6(17):371–372</a></li>
	</ul>
</li>
	<li id="menu-item-2200" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2200"><a href="_5c79e5ae.html">2018 Vol 6 No 18</a>
	<ul class="sub-menu">
		<li id="menu-item-2204" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2204"><a href="_7a1f8294.html">2018; 6(18):373–388</a></li>
		<li id="menu-item-2242" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2242"><a href="_34d09962.html">2018; 6(18):389–395</a></li>
		<li id="menu-item-2213" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2213"><a href="_d7780df8.html">2018; 6(18):396–405</a></li>
		<li id="menu-item-2226" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2226"><a href="_bf6e6bc8.html">2018; 6(18):406–413</a></li>
		<li id="menu-item-2257" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2257"><a href="_6215e068.html">2018; 6(18):414–427</a></li>
		<li id="menu-item-2278" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2278"><a href="_4bd5501.html">2018; 6(18):428–444</a></li>
	</ul>
</li>
</ul>
</li>
<li id="menu-item-2315" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2315"><a href="_2530ef33.html">2019</a>
<ul class="sub-menu">
	<li id="menu-item-2311" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2311"><a href="_6d4dd5c1.html">2019 Vol 7 No 19</a>
	<ul class="sub-menu">
		<li id="menu-item-2306" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2306"><a href="_d274d05.html">2019; 7(19):1–9</a></li>
		<li id="menu-item-2386" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2386"><a href="_d274cfd.html">2019; 7(19):10–20</a></li>
		<li id="menu-item-2321" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2321"><a href="_caa66125.html">2019; 7(19):21–29</a></li>
		<li id="menu-item-2338" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2338"><a href="_52763638.html">2019; 7(19):30–46</a></li>
		<li id="menu-item-2362" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2362"><a href="_82897a9b.html">2019; 7(19):47–54</a></li>
		<li id="menu-item-2372" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2372"><a href="_82897a9a.html">2019; 7(19):55–63</a></li>
	</ul>
</li>
	<li id="menu-item-2427" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2427"><a href="_1a5e6b6d.html">2019 Vol 7 No 20</a>
	<ul class="sub-menu">
		<li id="menu-item-2770" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2770"><a href="_a63083be.html">2019; 7(20):64–77</a></li>
		<li id="menu-item-2653" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2653"><a href="_5877a27f.html">2019; 7(20):78–85</a></li>
		<li id="menu-item-2740" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2740"><a href="_cdd9d011.html">2019; 7(20):86–92</a></li>
		<li id="menu-item-2743" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2743"><a href="_5877a27e.html">2019; 7(20):93–105</a></li>
		<li id="menu-item-2895" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2895"><a href="_21fd8046.html">2019; 7(20):106–120</a></li>
		<li id="menu-item-2919" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2919"><a href="_c4a4f4e3.html">2019; 7(20):121–128</a></li>
	</ul>
</li>
	<li id="menu-item-2960" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2960"><a href="_ca40ef92.html">2019 Vol 7 No 21</a>
	<ul class="sub-menu">
		<li id="menu-item-3026" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3026"><a href="_694b92a9.html">2019; 7(21):129–135</a></li>
		<li id="menu-item-3117" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3117"><a href="_c6a41e07.html">2019; 7(21):136–147</a></li>
		<li id="menu-item-3029" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3029"><a href="_fa5433e5.html">2019; 7(21):148–153</a></li>
		<li id="menu-item-3135" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3135"><a href="_59abe87d.html">2019; 7(21):154–160</a></li>
		<li id="menu-item-3047" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3047"><a href="_6fb66183.html">2019; 7(21):161–175</a></li>
		<li id="menu-item-3085" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3085"><a href="_c6a41e10.html">2019; 7(21):176–187</a></li>
	</ul>
</li>
	<li id="menu-item-3197" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3197"><a href="_59abe878.html">2019 Vol 8 No 22</a>
	<ul class="sub-menu">
		<li id="menu-item-3291" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3291"><a href="_4bd5522.html">2019; 8(22):188–195</a></li>
		<li id="menu-item-3234" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3234"><a href="_7a1f82b1.html">2019; 8(22):196–201</a></li>
		<li id="menu-item-3360" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3360"><a href="_c7d863ed.html">2019; 8(22):202–212</a></li>
		<li id="menu-item-3218" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3218"><a href="_b9d27134.html">2019; 8(22):213–221</a></li>
		<li id="menu-item-3270" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3270"><a href="_1cc6f752.html">2019; 8(22):222–230</a></li>
		<li id="menu-item-3330" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3330"><a href="_82897ab8.html">2019; 8(22):231–244</a></li>
	</ul>
</li>
	<li id="menu-item-3418" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3418"><a href="_32680dbe.html">2019 Vol 8 No 23</a>
	<ul class="sub-menu">
		<li id="menu-item-3451" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3451"><a href="_254c95d.html">2019; 8(23):245–257</a></li>
		<li id="menu-item-3500" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3500"><a href="_a4fc3df2.html">2019; 8(23):258–266</a></li>
		<li id="menu-item-3552" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3552"><a href="_3ad205c1.html">2019; 8(23):267–275</a></li>
		<li id="menu-item-3470" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3470"><a href="_77b6f6f8.html">2019; 8(23):276–286</a></li>
		<li id="menu-item-3536" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3536"><a href="_22c86393.html">2019; 8(23):287–296</a></li>
		<li id="menu-item-3579" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3579"><a href="_500daa9c.html">2019; 8(23):297–311</a></li>
	</ul>
</li>
	<li id="menu-item-3644" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3644"><a href="_5877a2a3.html">2019 Vol 8 No 24</a>
	<ul class="sub-menu">
		<li id="menu-item-3732" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3732"><a href="_60e19aa5.html">2019; 8(24):312–322</a></li>
		<li id="menu-item-3666" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3666"><a href="_cdd9d037.html">2019; 8(24):323–331</a></li>
		<li id="menu-item-3705" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3705"><a href="_b5d02e07.html">2019; 8(24):332–340</a></li>
		<li id="menu-item-3756" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3756"><a href="_d643c837.html">2019; 8(24):341–357</a></li>
		<li id="menu-item-3786" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3786"><a href="_78eb3cd5.html">2019; 8(24):358–371</a></li>
	</ul>
</li>
</ul>
</li>
<li id="menu-item-3889" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3889"><a href="_4f42c764.html">2020</a>
<ul class="sub-menu">
	<li id="menu-item-3863" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3863"><a href="_9f3de38.html">2020 Vol 9 No 25</a>
	<ul class="sub-menu">
		<li id="menu-item-3886" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3886"><a href="_9491b0a4.html">2020; 9(25):1–12</a></li>
		<li id="menu-item-3914" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3914"><a href="_9f3de2f.html">2020; 9(25):13–21</a></li>
		<li id="menu-item-4021" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4021"><a href="_27997b14.html">2020; 9(25):22–35</a></li>
		<li id="menu-item-4096" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4096"><a href="_694b92cd.html">2020; 9(25):36–47</a></li>
	</ul>
</li>
</ul>
</li>
</ul>						</nav>
					
					
					
										<div id="et_top_search">
						<span id="et_search_icon"></span>
					</div>
					
					<div id="et_mobile_nav_menu">
				<div class="mobile_nav closed">
					<span class="select_page">Select Page</span>
					<span class="mobile_menu_bar mobile_menu_bar_toggle"></span>
				</div>
			</div>				</div> <!-- #et-top-navigation -->
			</div> <!-- .container -->
			<div class="et_search_outer">
				<div class="container et_search_form_container">
					<form role="search" method="get" class="et-search-form" action="http://www.aimsci.com/ros/html/">
					<input type="search" class="et-search-field" placeholder="Search &hellip;" value="" name="s" title="Search for:" />					</form>
					<span class="et_close_search_field"></span>
				</div>
			</div>
		</header> <!-- #main-header -->
			<div id="et-main-area">
	
<div id="main-content">


	<div class="container">
		<div id="content-area" class="clearfix">
			<div id="left-area">


			
				<article id="post-1560" class="post-1560 page type-page status-publish hentry">

				
					<h1 class="entry-title main_title">2017; 4(12):389–417</h1>
				
				
					<div class="entry-content">
					<h1><span style="color: #808000;"><strong>RESEARCH ARTICLES</strong></span></h1>
<h1><a href="../index.php/ros/article/view/108/225.html"><img class="alignright wp-image-231 " src="wp-content/uploads/2016/03/PDF-logo-blue-30.png" alt="PDF logo-blue 30" width="53" height="67" /></a></h1>
<p>&nbsp;</p>
<h2><strong>Characterization of Two <em>Arabidopsis</em> L-Gulono-1,4-lactone Oxidases, AtGulLO3 and AtGulLO5, Involved in Ascorbate Biosynthesis</strong></h2>
<p>&nbsp;</p>
<p><strong>Siddique I. Aboobucker<sup>1,a</sup>, Walter P. Suza<sup>1,a</sup>, and Argelia Lorence<sup>1,2</sup></strong></p>
<p><sup>1</sup>Arkansas Biosciences Institute, Arkansas State University, P.O. Box 639, State University, AR 72467, USA; <sup>a</sup>Current address: 2104 Agronomy Hall, Iowa State University, Ames, IA 50011, USA; <sup>2</sup>Department of Chemistry and Physics, Arkansas State University, P.O. Box 419, State University, AR 72467, USA</p>
<p>Correspondence: alorence@astate.edu (A.L.)</p>
<p><em>Aboobucker</em><em> SI et al. Reactive Oxygen Species 4(12):389–417, 2017; </em><em>©2017 Cell Med Press</em></p>
<p><a href="../index.php/ros/article/view/108.html"><em>http://dx.doi.org/10.20455/ros.2017.861</em></a></p>
<p><em>(Received: June 30, 2017; Revised: July 18, 2017; Accepted: July 19, 2017)</em></p>
<p><strong>ABSTRACT</strong> <strong>|</strong> L-Ascorbic acid (AsA, vitamin C) is an essential antioxidant for plants and animals. There are four known ascorbate biosynthetic pathways in plants: the L-galactose, L-gulose, D-galacturonate, and<em> myo</em>-inositol routes. These pathways converge into two AsA precursors: L-galactono-1,4-lactone and L-gulono-1,4-lactone (L-GulL). This work focuses on the study of L-gulono-1,4-lactone oxidase (GulLO), the enzyme that works at the intersect of the gulose and inositol pathways. Previous studies have shown that feeding L-gulono-1,4-lactone to multiple plants leads to increased AsA. There are also reports showing GulLO activity in plants. We describe the first detailed characterization of a plant enzyme specific to oxidize L-GulL to AsA. We successfully purified a recombinant <em>Arabidopsis</em> GulLO enzyme (called AtGulLO5) in a transient expression system. The biochemical properties of this enzyme are similar to the ones of bacterial isozymes in terms of substrate specificity, subcellular localization, use of flavin adenine dinucleotide (FAD) as electron acceptor, and specific activity. AtGulLO5 is an exclusive dehydrogenase with an absolute specificity for L-GulL as substrate thus differing from the existing plant L-galactono-1,4-lactone dehydrogenases and mammalian GulLOs. Feeding L-GulL to <em>N. benthamiana</em> leaves expressing <em>AtGulLO5</em> constructs led to increased foliar AsA content, but it was not different from that of controls, most likely due to the observed low catalytic efficiency of AtGulLO5. Similar results were also obtained with another member of the AtGulLO family (AtGulLO3) that appears to have a rapid protein turnover. We propose that AsA synthesis through L-GulL in plants is regulated at the post-transcriptional level by limiting GulLO enzyme availability.</p>
<p><strong>KEYWORDS</strong> <strong>|</strong> Aldonolactone oxidoreductase; Ascorbate; GulLO; L-Gulono-1,4-lactone oxidase; Vitamin C</p>
<p><strong>ABBREVIATIONS </strong><strong>|</strong> ALO, arabinono lactone oxidase; AsA, L-ascorbic acid; AtGulLO, <em>Arabidopsis</em> L-gulono-1,4-lactone oxidase; DTT, dithiothrietol; EDTA, ethylenediaminetetraacetic acid; FAD, flavin adenine dinucleotide; GlcUR, D-glucuronate reductase; GLDH, L-galactono-1,4-lactone dehydrogenase; GME, GDP-D-mannose-3′,5′-epimerase; GNL, gulonolactonase; GulLDH, L-GulL dehydrogenase; GulLO, L-gulono-1,4-lactone oxidase; L-Gal, L-galactose; L-GalL, L-galactono-1,4-lactone; L-Gul, L-gulose; LGulL, L-gulono-1,4-lactone; MIOX,<em> myo</em>-inositol oxygenase; MPA, metaphosphoric acid; VAO, vanillyl alcohol oxidase</p>
<hr />
<p><strong>CONTENTS</strong></p>
<p><span style="line-height: 1.7em;">1. Introduction</span></p>
<p><span style="line-height: 1.7em;">2. Materials and Methods</span></p>
<p>2.1. Chemicals</p>
<p>2.2. Plant Materials and Growth Conditions</p>
<p>2.3. Ascorbate Measurements</p>
<p>2.4. GulLO Constructs in the pBIB-Kan Binary Vector</p>
<p>2.5. Transient Expression of Recombinant AtGulLO5 in N. benthamiana</p>
<p>2.6. Purification of Recombinant AtGulLO5-EK-6xHIS</p>
<p>2.7. Enterokinase Cleavage of 6xHIS from AtGulLO5-EK-6xHIS</p>
<p>2.8. GulL Dehydrogenase and Oxidase Activity Assays</p>
<p>2.9. <em>A. thaliana</em> Transgenic Plant Generation</p>
<p>2.10. Analysis of <em>A. thaliana</em> Transgenic Status by PCR and RT-PCR</p>
<p>2.11. Total Protein Extraction, SDS-PAGE, and Western Blotting</p>
<p>2.12. Substrate Feeding via Petiole in <em>N. benthamiana</em> and <em>A. thaliana</em></p>
<p>2.13. Statistical Analysis</p>
<p><span style="line-height: 1.7em;">3. Results</span></p>
<p>3.1. Phylogenetic Analysis and Expression Patterns of Putative AtGulLOs and AtGLDH</p>
<p>3.2. Purification and Characterization of Recombinant AtGulLO5 Transiently Expressed in <em>N. benthamiana</em></p>
<p>3.3. <em>In planta</em> Function of AtGulLO3 and AtGulLO5 in <em>N. benthamiana</em> Transient Expression System Showed no Significant Increase in Foliar AsA Content</p>
<p>3.4. <em>A. thaliana</em> AtGulLO3 and AtGulLO5 Transgenic Plants Did Not Have Increased Foliar AsA Content</p>
<p>3.5. AtGulLO3 Protein Appears to Be Regulated Post-Transcriptionally by Rapid Turnover</p>
<p><span style="line-height: 1.7em;">4. Discussion</span></p>
<p><span style="line-height: 1.7em;">5. Conclusions</span></p>
<hr />
<p><strong style="line-height: 1.7em;">1. INTRODUCTION</strong></p>
<p>L-Ascorbic acid (ascorbate, AsA), besides being the vitamin that prevents scurvy, has multiple applications as a therapeutic for human health in the treatment of common cold, wound healing, and cancer among other illnesses [1]. We cannot synthesize vitamin C on our own because of the lack of a functional L-gulono-1,4-lactone oxidase (GulLO), the terminal enzyme in the pathway [2], therefore we need to acquire this vitamin from fresh fruits and vegetables.</p>
<p>Ascorbate is a primary metabolite in plants, which possesses many functions. It is one of the most abundant molecules in green leaves and represents 10% of the total soluble carbohydrate pool in favorable conditions. Ascorbate efficiently scavenges free radicals and other reactive oxygen species formed during normal metabolism [3]. The roles of AsA in plants are diverse including being an antioxidant and being involved in the defense against ozone, sulphur dioxide and UV-B radiation [4]. Ascorbate also modulates cell growth, cell division, cell elongation and tolerance to stresses [3, 5].</p>
<p>Metabolic engineering attempts to increase AsA in plants have resulted in limited success [6] partly due to an incomplete understanding of the AsA metabolic network. Plants appear to have multiple biosynthetic pathways for AsA. The first one was reported in 1998 [7] to proceed from GDP-D-mannose, GDP-L-galactose, L-galactose (L-Gal) and L-galactono-1,4-lactone (L-GalL). The operation of this pathway has been confirmed in multiple plant species. A second pathway uses a D-galacturonate reductase to convert D-galacturonate to AsA as demonstrated in strawberry [8]. The immediate precursor in these two pathways is L-GalL, which is oxidized to AsA via L-galactono-1,4-lactone dehydrogenase (GLDH; <strong>Figure 1</strong>). A third pathway was proposed to proceed from L-gulose (L-Gul), an epimerization product of GDP-D-mannose-3′,5′-epimerase (GME), in addition to L-Gal. GME co-purified with a heat shock protein suggesting the operation of this pathway under stress [9]. Lorence et al. [10] proposed a fourth pathway by showing that <em>myo</em>-inositol (MI) can also be channeled towards the production of AsA by overexpressing a <em>myo-inositol oxygenase</em> (<em>MIOX</em>) gene in <em>Arabidopsis thaliana</em>. A purple acid phosphatase, AtPAP15 supplying MI to this pathway further supports the operation of the MI route in plants [11]. <em>Myo</em>-inositol must undergo four intermediate steps to become AsA in plants and the proposed enzymes for these steps are: MIOX, D-glucuronate reductase (GlcUR), gulonolactonase (GNL) and L-gulono-1,4-lactone oxidase<strong> (</strong>GulLO).</p>
<p><a href="wp-content/uploads/2017/09/figure-1-3.png"><img class="alignnone size-medium wp-image-1565" src="wp-content/uploads/2017/09/figure-1-3-300x189.png" alt="" width="300" height="189" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-1-3-300x189.png 300w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-1-3-768x484.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-1-3-1024x646.png 1024w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-1-3-400x250.png 400w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-1-3-1080x681.png 1080w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-1-3.png 1476w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p><strong>FIGURE 1</strong>. <strong>Four different pathways lead to ascorbate biosynthesis in plants involving <em>myo</em>-inositol, L-gulose, L-galactose and D-galacturonate as main precursors.</strong> These pathways merge in the formation of two alternative immediate precursors: L-galactono-1,4-lactone and L-gulono-1,4-lactone. Conversion of these to the final product is catalyzed by L-galactono-1,4-lactone dehydrogenase (GLDH) and by L-gulono-1,4-lactone oxidase (GulLO), respectively. Oxidized ascorbate (a.k.a. dehydroascorbate) can be recycled back to ascorbate, the reduced form of the antioxidant.</p>
<p>The enzyme, GulLO, is well characterized in animals, fungi, bacteria, and protozoa [12‒15]. Substrate specificity is one of the interesting properties of GulLO when compared to the last enzyme of the L-Gal pathway, L-galactono-1,4-lactone dehydrogenase (GLDH). Rat GulLO (RnGulLO) can use both L-GalL and L-gulono-1,4-lactone (L-GulL) as substrates [12, 13, 15], while plant GLDHs are very specific to L-GalL and have limited activity or none towards L-GulL [16‒20]. An increase in AsA content was observed by the overexpression of  RnGulLO in tobacco [21], potato [22], tomato [23] and <em>Arabidopsis</em> [24], whereas no increased AsA was observed in tobacco when GLDH was overexpressed [25]. This suggests that the flux through L-GulL can also be used to increase AsA in plants irrespective of the availability of the L-GalL pool.</p>
<p>The conversion of L-GulL to AsA (terminal step in both the gulose and inositol pathways; <strong>Figure 1</strong>) has been observed by multiple researchers in different plant species. This conversion, however, is less efficient than the one of L-GalL to AsA [9, 21, 25‒31]. Scarce information is available about the enzyme responsible for this reaction in plants. On the other hand, information about GLDH, the enzyme that is specific for L-GalL is abundant. Plant GLDHs are highly specific to L-GalL [16‒19, 32] or at least have high catalytic efficiency to L-GalL [20, 33]. Although it could be speculated that the observed slow utilization of L-GulL versus L-GalL in different plant species is due to the marginal activity of GLDH towards L-GulL, multiple authors [9, 20, 29] have predicted the existence of a GulLO enzyme. Baig et al. [28] predicted that this enzyme would be specific to L-GulL. GulLO enzyme activity has been detected in hypocotyl homogenates of kidney beans [31], cytosolic and mitochondrial fractions of <em>Arabidopsis</em> cell cultures [29], and potato tubers [9, 33] but no enzyme has been characterized in detail to date. In an experiment by Maruta et al. [34], tobacco BY-2 cell cultures over-expressing three putative <em>Arabidopsis</em> GulLOs (AtGulLO2, 3 and 5) increased AsA after L-GulL feeding. However, the specificity of these enzymes between L-GulL and L-GalL was not characterized and their biochemical properties were not studied.</p>
<p>Here we describe the characterization of two AtGulLO members, AtGulLO3 and AtGulLO5. We report for the first time, the characterization of a recombinant AtGulLO5 (At2g46740), transiently expressed in <em>N. benthamiana,</em> and demonstrate its activity to convert L-GulL to AsA. The enzyme is a dehydrogenase with an absolute specificity for L-GulL thus differing from the existing plant GLDHs (specific to L-GalL) or mammalian GulLOs. We also found that AtGulLO5 has a low catalytic efficiency in comparison to similar isozymes. We propose that <em>in planta</em> this enzyme may need an effector molecule to increase its efficiency post-transcriptionally. Another member of the AtGulLO family, AtGulLO3 (At5g11540), may also be post-transcriptionally regulated in a different mechanism than AtGulLO5, i.e., by rapid protein turnover. These findings led us to propose that AsA synthesis via L-GulL in plants is regulated at the post-transcriptional level by limited GulLO enzyme availability. This phenomenon offers an explanation for the previously observed slow utilization of L-GulL versus L-GalL in different plant species, which is also discussed herein.</p>
<p><strong style="line-height: 1.7em;">2. MATERIALS AND METHODS</strong></p>
<p><strong>2.1. Chemicals</strong></p>
<p>L-Gulono-1,4-lactone, D-gluconic acid-γ-lactone, cytochrome C from equine heart, flavin adenine dinucleotide (FAD), reduced glutathione, metaphosphoric acid and ascorbate oxidase were purchased from Sigma-Aldrich (St. Louis, MO, USA).  Dithiothrietol (DTT), 2-(N-morpholino) ethane sulfonic acid (MES), Tris, NaCl, glycine, sodium dodecyl sulfate (SDS), and glycerol were purchased from Fisher Scientific (Fair Lawn, NJ, USA). Ethylenediaminetetraacetic acid (EDTA) was acquired from Promega (Madison, WI, USA). D-Galactono-1,4-lactone was purchased from Carbomer (San Diego, CA, USA). L-Galactono-1,4-lactone was from Santa Cruz Biotechnology (Dallas, TX, USA) and L-galactose was from TCI America (Portland, OR, USA).</p>
<p><strong>2.2. Plant Materials and Growth Conditions</strong></p>
<p><em>Arabidopsis thaliana</em> var. Columbia (Col-0, stock CS-60000), SALK 036899, and SALK 008089 seeds were obtained from the Arabidopsis Biological Resource Center (ABRC, Columbus, OH, USA). After sterilizing with 50% bleach, seeds were planted on petri plates containing solidified Murashige and Skoog (MS) media [35] and 50 mg/l kanamycin for transgenic or SALK lines. The plates were vernalized for 3 days at 4<sup>o</sup>C and placed in a growth chamber for 7 days with the following conditions: 23<sup>o</sup>C constant temperature, 65% relative humidity, 150 µmol m<sup>‒2</sup> s<sup>‒1</sup> light intensity, and 16:8 h photoperiod. Robust seedlings were transferred to 5 × 5 cm pots containing Arabidopsis Growth Medium (Lehle Seeds, Round Rock, TX, USA) and incubated in the same growth chamber. Plants at developmental stage 5.1 [36] were used to collect rosette leaves. After harvesting, the tissue was frozen immediately in liquid nitrogen, and stored at ‒80<sup>o</sup>C until analysis. To do substrate feeding assays, <em>A. thaliana</em> plants were grown in the same way as described above except the photoperiod was 12:12 h and rosette leaves were from plants at developmental stage 5.1 [36].</p>
<p><em>Nicotiana benthamiana</em> seeds were obtained from Dr. Sue Tolin (Department of Plant Pathology, Physiology and Weed Science, Virginia Polytechnic Institute and State University, Blacksburg, VA, USA) and grown as previously described [37]. Approximately, 2 to 3 <em>N. benthamiana</em> seeds were sown in 12 cm square pots containing Pro-mix BX soil (Premier Horticulture, Canada) and were fertilized with Osmocote 14-14-14 (Scotts, Canada). Vermiculite was overlaid on top of the seeds. Pots were watered and covered with a dome until ~3 weeks at which time the domes were gradually vented over a 2‒3-day period. The plants were grown in a growth chamber with the following conditions: 25<sup>o</sup>C (day)/21<sup>o</sup>C (night) temperature, 65% relative humidity, 150 µmol m<sup>‒2</sup>s<sup>‒1</sup> light intensity, and 16:8 h photoperiod.</p>
<p><strong>2.3. Ascorbate Measurements</strong></p>
<p>Baseline AsA content of untransformed <em>N. benthamiana</em> plants was determined by collecting all healthy leaves from 6-week old plants and measuring reduced, oxidized, and total AsA content. Whole leaves were cut without the petiole, weighed, flash frozen in liquid nitrogen and stored at ‒80<sup>o</sup>C until analysis. Samples were taken at 10:00 am, 1‒2 h after the lights in the growth chamber go on. Reduced, oxidized and total AsA were measured using an enzyme-based spectrophotometric method as previously described [10]. Briefly plant tissue (frozen or fresh) was pulverized in liquid nitrogen and 6% (w/v) metaphosphoric acid (MPA) was added to a ratio of 60 mg tissue: 750 µl MPA. The homogenate was centrifuged at 13,000 × g for 15 min and the supernatant was used for AsA measurement. Total AsA is the sum of reduced and oxidized ascorbic acid measured individually. Reduced AsA was measured, in a 1 mL reaction mixture containing 100 mM potassium phosphate buffer pH 6.9 and 50 µl of plant extract, by recording the decrease in absorbance at 265 nm for 1 min after the addition of 1 µl ascorbate oxidase (Sigma). Oxidized AsA was measured by recording the absorbance at 265 nm, before and 20 min after the addition of 1 µl 200 mM DTT to a 1 mL reaction mixture. An extinction coefficient of 14.3 mM<sup>‒1</sup>cm<sup>‒1</sup> was used for calculations. Ascorbate values are the mean of at least 5 biological replicates measured in duplicate and are reported as µmol per gram fresh weight (FW).</p>
<p><strong>2.4. GulLO Constructs in the pBIB-Kan Binary Vector</strong></p>
<p>To make <em>AtGulLO3</em> constructs, template cDNA was synthesized by reverse transcribing 1 µg of RNA with oligo-dT primers by LongRange reverse transcriptase (Qiagen, Valencia, CA, USA). Total RNA was isolated—from a pool of <em>A. thaliana</em> rosette and cauline leaves, siliques, flowers, and roots—using the RNeasy® plant mini kit following the manufacturer’s protocol (Qiagen). In order to amplify <em>AtGulLO3</em> and <em>AtGulLO3-6xHIS </em>(<strong>Figure 4A</strong> and <strong>B</strong>)<em>,</em> the following PCR conditions were used with PrimeSTAR<sup>TM</sup> HS DNA polymerase (Takara, Madison, WI): an initial denaturation of 94<sup>o</sup>C for 30 s was followed by 35 cycles. Each cycle consisted of 98<sup>o</sup>C for 10 s, 53<sup>o</sup>C for 15 s and 72<sup>o</sup>C for 2 min. The PCR was completed with a final extension of 72<sup>o</sup>C for 5 min. Primers for PCR amplification were synthesized at Invitrogen (Carlsbad, CA, USA) and their sequences are presented in <strong>Table 1</strong>. Primer pairs for <em>AtGulLO3</em> were AtGulLO3_F and AtGulLO3_R. For <em>AtGulLO3-6xHIS</em>, they were AtGulLO3_F and AtGulLO3-6xHIS_R.</p>
<table width="624">
<tbody>
<tr>
<td colspan="2" width="624"><strong>TABLE 1. Primers used to make <em>GulLO</em> constructs and to do the molecular characterization of <em>Arabidopsis</em> transgenic lines</strong></td>
</tr>
<tr>
<td width="156"><strong>Primer name</strong></td>
<td width="468"><strong>Primer sequence (5’ to 3’)</strong></td>
</tr>
<tr>
<td width="156">AtGulLO3_F</td>
<td width="468">C<u>GGTACC</u>ATGCGTTACTCTCATACTCTC</td>
</tr>
<tr>
<td width="156">AtGulLO3_R</td>
<td width="468">C<u>GAGCTC</u>TTACATCACTATAACTTGAGC</td>
</tr>
<tr>
<td width="156">AtGulLO3-6xHIS_R</td>
<td width="468">C<u>GAGCTC</u>TTA<em>ATGATGATGATGATGATG</em>CATCACTATAACTTGAGC</td>
</tr>
<tr>
<td width="156">AtGulLO3-RT_F</td>
<td width="468">GCAACGGAGTCAACGACTTTC</td>
</tr>
<tr>
<td width="156">AtGulLO3-RT_R</td>
<td width="468">TGCCTTGTCTTCCGATAACC</td>
</tr>
<tr>
<td width="156">AtGulLO5_F</td>
<td width="468">CGG<u>GGTACC</u>ATGGCATTTGGTTATTCTCCTTC</td>
</tr>
<tr>
<td width="156">AtGulLO5_R</td>
<td width="468">C<u>GAGCTC</u>TTACTTAGGTACATGTGTACAGACTCTAG</td>
</tr>
<tr>
<td width="156">AtGulLO5-6xHIS_R</td>
<td width="468">C<u>GAGCTC</u>TTA<em>ATGATGATGATGATGATG</em>CTTAGGTACATGTGTACAGACTC</td>
</tr>
<tr>
<td width="156">AtGulLO5-EK-6xHIS_R</td>
<td width="468">C<u>GAGCTC</u>TTA<em>ATGATGATGATGATGATG</em><strong>CTTGTCGTCGTCATC</strong>CTTAGGTACATGTGTACAGACTC</td>
</tr>
<tr>
<td width="156">EF1α-A_F</td>
<td width="468">TGAGCACGCTCTTCTTGCTTTCA</td>
</tr>
<tr>
<td width="156">EF1α-A_R</td>
<td width="468">GGTGGTGGCATCCATCTTGTTACA</td>
</tr>
<tr>
<td width="156">nptII_F</td>
<td width="468">AGAGGCTATTCGGCTATGAC</td>
</tr>
<tr>
<td width="156">nptII_R</td>
<td width="468">AGCTCTTCAGCAATATCACG</td>
</tr>
<tr>
<td width="156">RnGulLO_F</td>
<td width="468">C<u>GGTACC</u>ATGGTCCATGGGTACAAAG</td>
</tr>
<tr>
<td width="156">RnGulLO-6xHIS_R</td>
<td width="468">C<u>GAGCTC</u>TTA<em>ATGATGATGATGATGATG</em>GTAGAAGACTTTCTCCAGGTACG</td>
</tr>
<tr>
<td width="156">TEV</td>
<td width="468">CAAACGAATCTCAAGCAATCAAG</td>
</tr>
<tr>
<td colspan="2" width="624"><em>Note: KpnI and SacI restriction sites in the forward and reverse primers, respectively, for cloning are underlined. Sequences encoding a poly HIS tail are italicized and the enterokinase (EK) recognition sequence is in bold. F, forward; R, reverse; RT, reverse transcription.</em></td>
</tr>
</tbody>
</table>
<p>To make <em>AtGulLO5</em> constructs, template cDNA was synthesized as mentioned above using the roots of <em>A. thaliana </em>seedlings. The PCR conditions to amplify <em>AtGulLO5</em> and <em>AtGulLO5-6xHIS</em> were: 98<sup>o</sup>C for 10 s, 53<sup>o</sup>C for 15 s, 72<sup>o</sup>C for 2 min using PrimeSTAR<sup>TM</sup> HS DNA polymerase (Takara, Madison, WI) for 35 cycles, after initial denaturation of 94<sup>o</sup>C for 30 s and completed with final extension of 72<sup>o</sup>C for 5 min. The following were the primer pairs used for making the <em>AtGulLO5</em> constructs: AtGulLO5_F and AtGulLO5_R (for <em>AtGulLO5</em>), AtGulLO5_F and AtGulLO5-6xHIS_R (for <em>AtGulLO5-6xHIS</em>). To make the <em>AtGulLO5-EK-6xHIS </em>construct, PCR conditions for 35 cycles were: 98<sup>o</sup>C for 10 s, 61<sup>o</sup>C for 30 s, 72<sup>o</sup>C for 30 s using Phusion High-Fidelity DNA polymerase (New England BioLabs, Ipswich, MA) after initial denaturation of 98<sup>o</sup>C for 30 s and completed with a final extension of 72<sup>o</sup>C for 5 min. The primer pairs were AtGulLO5_F and AtGulLO5-EK-6XHIS_R and their sequences are in <strong>Table 1</strong>.</p>
<p>PCR products were purified using the QIAquick PCR purification kit by following the manufacturer’s protocol (Qiagen). Purified PCR products were digested with <em>KpnI</em> and <em>SacI </em>(Promega) overnight at 37<sup>o</sup>C. A modified binary vector, pBIB-Kan [38] was also digested as above, in parallel. The digested PCR products and vector were purified and ligated at 16<sup>o</sup>C overnight and transformed to <em>E. coli </em>NEB 10-b cells for <em>AtGulLO5-EK-6xHIS </em>construct or DH5α for <em>AtGulLO3</em>, <em>AtGulLO3-6xHIS</em>, <em>AtGulLO5,</em> and <em>AtGulLO5-6xHIS</em> constructs. Successful clones were screened by extraction and restriction digestion of plasmid DNA. Positive clones were sequenced at the University of Chicago DNA sequencing and genotyping facility. Binary vectors carrying the gene constructs were transformed into <em>Agrobacterium tumefaciens</em> strain LBA 4404 (for transforming <em>N. benthamiana</em>) or GV3101 (for transforming <em>A. thaliana</em>) by the freeze-thaw method.</p>
<p>The <em>RnGulLO</em> cDNA was amplified from the binary vector pBIN19 carrying the <em>RnGulLO </em>cDNA [21] using <em>pfu</em> DNA polymerase (Agilent Technologies, Santa Clara, CA, USA) and the following PCR conditions: an initial denaturation step at 94<sup>o</sup>C for 5 min followed by 35 cycles of each denaturation at 94<sup>o</sup>C for 1 min, annealing at 60<sup>o</sup>C for 1 min and extension at 72<sup>o</sup>C for 90 s. The final extension was at 72<sup>o</sup>C for 10 min. The primer pairs, RnGulLO_F and RnGulLO-6xHIS_R (<strong>Table 1</strong>) were used to add a poly HIS tail to the 3’ end of the cDNA. The PCR product was purified using a gel purification kit (Qiagen). Poly ‘A’ overhangs were added by Taq polymerase (Promega) to the PCR product. Ligation to the pDrive cloning vector (Qiagen) and transformation into <em>E. coli</em> JM109 (Promega) cells were done as mentioned above. After verifying the sequence of positive clones, the <em>RnGulLO</em> fragment was digested with <em>KpnI</em> and <em>SacI</em> and ligated to the linearized pBIB-Kan vector to generate the <em>RnGulLO-6xHIS</em> construct illustrated in <strong>Figure 4B</strong>. Successful clones were sequenced and transformed into <em>A. tumefaciens</em> strain LBA 4404 to transform <em>N. benthamiana</em>.</p>
<p><strong>2.5. Transient Expression of Recombinant AtGulLO5 in <em>N. benthamiana</em></strong></p>
<p>Four to six weeks old <em>N. benthamiana</em> plants were grown as previously described [37] and vacuum infiltrated with <em>A. tumefaciens</em> carrying the <em>GulLO</em> constructs. Optimal expression time was experimentally determined by collecting and analyzing consistently the third leaf from the top of the plant 24, 48, 72 and 96 h post-infiltration. In all subsequent experiments, all healthy-looking leaves were collected 48 h post-infiltration, frozen immediately in liquid nitrogen and stored at ‒80<sup>o</sup>C until further processing.</p>
<p><strong>2.6. Purification of Recombinant AtGulLO5-EK-6xHIS</strong></p>
<p>Recombinant AtGulLO5-EK-6xHIS was purified from <em>N. benthamiana</em> leaves by a two-step purification procedure involving cation exchange and nickel affinity chromatography (<strong>Figure 6</strong>). Protein preparations were made fresh for individual experiments. Briefly, leaf tissue was pulverized in liquid nitrogen and proteins were extracted in 20 mM Tris-HCl pH 8.0, 0.6% v/v protease inhibitor cocktail (Sigma-Aldrich), 1 mM DTT (Buffer A). The supernatant obtained after centrifugation was exchanged to 20 mM MES-NaOH pH 6.0, 5 mM reduced glutathione (Buffer B) in a 10-DG desalting column (Bio-Rad, Hercules, CA, USA). This fraction was then loaded onto a cation exchanger, UNOSphere S Support (Bio-Rad) and washed with Buffer B containing 50 mM NaCl (Buffer C). The basic proteins were then eluted with Buffer B containing 500 mM NaCl (Buffer D) and exchanged to 50 mM sodium phosphate pH 7.4, 300 mM NaCl, 40 mM imidazole, 0.1 mM DTT (Buffer E) and loaded to a nickel affinity column (HIS60 Ni Superflow; Clontech, CA, USA). The column was washed with Buffer E and the bound proteins were eluted with Buffer E containing 300 mM imidazole (Buffer F). The eluate from the nickel column was concentrated and exchanged to 20 mM Tris-HCl pH 8.0, 50 mM NaCl, 20 mM CaCl<sub>2</sub> (Buffer G) using an AMICON® ultra centrifugal filter (Millipore, Billerica, MA, USA). Total soluble protein concentration was estimated by the Bradford method [39] using Coomassie G-250 dye and bovine serum albumin (Thermo Sci., Rockford, IL, USA) as a standard. Protein fractions from purification procedure were separated by SDS-PAGE and were silver stained using Pierce® Silver Stain Kit according to manufacturer’s protocol (Thermo Sci.).</p>
<p><strong>2.7. Enterokinase Cleavage of 6xHIS from AtGulLO5-EK-6xHIS</strong></p>
<p>Two hundred µg of purified protein were digested at 23<sup>o</sup>C for 16 h in a 500 µl final volume of Buffer G with 20 units of recombinant enterokinase (New England Biolabs, Ipswich, MA, USA). One unit of enterokinase is defined as the amount of protein needed to digest 25 µg of an MBP fusion protein (test substrate, New England Biolabs) to 95% completion in 16 h or less at 25°C.</p>
<p><strong>2.8. GulL Dehydrogenase and Oxidase Activity Assays</strong></p>
<p>L-Gulono-1,4-lactone dehydrogenase activity was measured by following the reduction of cytochrome C at 550 nm at 25<sup>o</sup>C as previously described [33]. One mL of the reaction contained 50 mM Tris-HCl pH 8.0, 121 µM cytochrome C, 50 mM L-GulL (unless specified), 100 µM FAD, and an aliquot of the purified enzyme. A parallel incubation with equal amount of boiled enzyme was carried out as control and any background is subtracted to get activity units. One unit was defined as 1 µmol of AsA produced per min, which is equivalent to 2 µmol of cytochrome C reduced. An extinction coefficient of 17.3 mM<sup>‒1</sup>cm<sup>‒1</sup> for cytochrome C was used for enzyme activity calculations. For the enzyme kinetics experiment, individual reactions were monitored for 15 min in an Evolution 300 spectrophotometer (Thermo Scientific) and the initial rates were calculated using the VISIONPro software by the chord method. Enzyme kinetic analysis was performed in the SigmaPlot® 11.2 software (San Jose, CA, USA).</p>
<p>L-Gulono-1,4-lactone oxidase activity was measured as described [40] with slight modifications. The reaction mixture contained 50 mM Tris-HCl pH 8.0 buffer, 50 mM reduced glutathione, 200 mM L-GulL and the reaction was initiated by adding an aliquot of the purified enzyme to a final volume of 1 mL and incubated at 37<sup>o</sup>C for 20 min. The above reaction was stopped by adding 0.1 mL of 50% TCA, incubated in ice for 20 min, centrifuged at 13,000 x g for 15 min and the supernatant was analyzed by HPLC to detect AsA. A Luna 5 µm HILIC 200Å column with dimension 150 x 4.6 mm (Phenomenex, Torrance, CA, USA) was used. The column was connected to a Dionex Ultimate 3000 system equipped with a photo diode array detector. The mobile phase had 100 mM ammonium acetate, pH 5.8 (5%) for the full 15 min run at 25<sup>o</sup>C. Initial conditions were for 2.5 min: 90% acetonitrile and 5% water. In the next 5 min, the percentage of water was gradually increased from 5 to 45 in a gradient and the percentage of acetonitrile was decreased from 90 to 50. These conditions were kept constant for 0.5 min. In the next 1 min, initial conditions were restored and the column was equilibrated for 6 min.</p>
<p><strong>2.9.<em> A. thaliana</em> Transgenic Plant Generation</strong></p>
<p>To generate transgenic<em> A. thaliana</em> plants, transformation was done with <em>A. tumefaciens</em> strain GV3101 carrying the <em>AtGulLO3</em> or <em>AtGulLO5</em> constructs by the floral dip method [41]. Putative transformants were obtained by selecting seedlings on MS plates containing 500 mg/L carbenicillin and 50 mg/L kanamycin. Primary transformants were screened for the presence of the transgene and the kanamycin resistance gene (<em>nptII</em>) by PCR. Foliar AsA content was measured in rosette leaves of the primary transformants at the developmental stage 6.9 [36]. The progeny—T2 (for <em>AtGulLO5</em>) or T3 (for <em>AtGulLO3</em>) —was used for AsA measurement, substrate feeding assays, transgene expression, and protein expression verification.</p>
<p><strong>2.10. Analysis of <em>A. thaliana</em> Transgenic Status by PCR and RT-PCR</strong></p>
<p>PCR was performed to verify the presence of the transgene and kanamycin resistance gene (<em>nptII</em>) in <em>A. thaliana</em> putative transgenic lines with GoTaq polymerase (Promega). Genomic DNA was extracted from leaves of the putative transgenic plants and used. The PCR conditions to amplify full length <em>AtGulLO3-6xHIS</em> or <em>AtGulLO5-6xHIS</em> transgene were as follows: initial denaturation at 94<sup>o</sup>C for 3 min followed by 94<sup>o</sup>C for 30 s, 53<sup>o</sup>C for 1 min, 72<sup>o</sup>C for 2 min; 30 cycles and a final denaturation at 72<sup>o</sup>C for 5 min. The primer sequences were TEV and AtGulLO3-6xHIS_R for <em>AtGulLO3-6xHIS</em> or TEV and AtGulLO5-6xHIS_R to amplify <em>AtGulLO5-6xHIS</em> (<strong>Table 1</strong>). The following PCR conditions were used for the amplification of <em>nptII</em> gene. The initial denaturation was for 3 min at 94<sup>o</sup>C, followed by 35 cycles of 94<sup>o</sup>C for 30 s, 50<sup>o</sup>C for 1 min and 72<sup>o</sup>C for 45 s and a final extension step at 72<sup>o</sup>C for 5 min. The primer sequences used were nptII_F and nptII_R (<strong>Table 1</strong>).</p>
<p>Expression of the <em>AtGulLO3-6xHIS</em> transgene in <em>A. thaliana</em> was also verified by RT-PCR by purifying total RNA from two individual plants for each line using an RNeasy® plant mini kit (Qiagen). RNA from the two plants belonging to the same line was pooled. The iScript cDNA synthesis kit (Bio-Rad) was used to synthesize the first strand cDNA from 1 µg of RNA using a mix of oligo-dT and random hexamer primers included in the kit. The reference gene, elongation factor (<em>EF1α-A</em>), and <em>AtGulLO3-6xHIS</em> were amplified using GoTaq Mastermix (Promega) by PCR with the following conditions as: Initial denaturation was at 94<sup>o</sup>C for 4 min followed by 35 cycles of 94<sup>o</sup>C for 30 s, 56<sup>o</sup>C for 1 min, 72<sup>o</sup>C for 30 s. No final extension was done. The primer pairs for <em>AtGulLO3</em> were AtGulLO3-RT_F and AtGulLO3-RT_R; for <em>EF1α-A</em>, they were EF1α-A_F and EF1α-A_R. The primer sequences are presented in <strong>Table 1</strong>.</p>
<p><strong>2.11. Total Protein Extraction, SDS-PAGE, and Western Blotting</strong></p>
<p>To determine recombinant AtGulLO3-6xHIS or AtGulLO5-6xHIS protein expression levels, total protein was extracted from <em>N. benthamiana</em> or <em>A. thaliana</em> leaves. Tissue was pulverized in liquid nitrogen and protein was solubilized in extraction buffer (150 mM Tris-HCl pH 6.8, 30% glycerol, 6% SDS, 5 mM EDTA pH 8.0) at a ratio of 1 g tissue per 2 mL extraction buffer. After centrifuging the homogenate at 13,000 x g for 15 min, the supernatant containing total protein was recovered. To separate protein samples by SDS-PAGE, 6.5 µl of total protein were mixed with 2.5 µl SDS-loading buffer and 1 µl DTT (500 mM), boiled for 10 min at 70<sup>o</sup>C and separated by SDS-PAGE on 10% precast minigels with a Tris-MOPS buffer system (Expedeon, San Diego, CA). For Western blot, SDS-PAGE separated proteins were transferred electrophoretically to nitrocellulose membrane in transblotting buffer (25 mM Tris base, 192 mM glycine and 20% methanol). Recombinant AtGulLO3-6xHIS or AtGulLO5-6xHIS was detected by α-HIS-AP antibody at a 1:2,000 dilution (Invitrogen, Carlsbad, CA, USA) and CDP-star, a chemiluminescent substrate for alkaline phosphatase (Roche Diagnostics, Indianapolis, IN, USA).</p>
<p><strong>2.12. Substrate Feeding via Petiole in <em>N. benthamiana</em> and <em>A. thaliana</em></strong></p>
<p>The substrate, L-GulL (50 mM), was freshly prepared by dissolving it in double-distilled water and aliquoted in 1.5 ml microfuge tubes. Second and third leaves from the top of 6-weeks old <em>N. benthamiana</em> plants 48 h post-infiltration with test constructs, empty vector, or untransformed, were cut from the stem and the petiole was immersed in the microfuge tubes containing either the substrate or water as control. Leaves were incubated at 25<sup>o</sup>C for 8 h (10:00 am to 6:00 pm) in continuous light (150 µmol m<sup>‒2</sup>s<sup>‒1</sup>) and whole leaves without petiole were weighed, frozen in liquid nitrogen and stored at ‒80<sup>o</sup>C until analysis. For substrate feeding assays in <em>A. thaliana</em>, leaves of plants at developmental stage 5.1 [36] were cut and the petiole was immersed in the freshly prepared substrate solutions or water. Substrate concentrations were 5 or 10 mM as mentioned in respective figures. The incubation conditions were 23<sup>o</sup>C and continuous light (100 µmol m<sup>‒2</sup>s<sup>‒1</sup>) for 16 h. Whole leaves without petiole were weighed, frozen in liquid nitrogen, and stored at ‒80<sup>o</sup>C until analysis.</p>
<p><strong>2.13. Statistical Analysis</strong></p>
<p>All data was analyzed by one-way ANOVA at a confidence interval of 95% using the Minitab® 16.1.1 software (State College, PA, USA).</p>
<p><strong style="line-height: 1.7em;">3. RESULTS</strong></p>
<p><strong>3.1. Phylogenetic Analysis and Expression Patterns of Putative AtGulLOs and AtGLDH</strong></p>
<p>To find putative GulLOs in the <em>Arabidopsis</em> genome, a BLASTP search was done against the <em>Arabidopsis</em> protein database (www.arabidopsis.org) using the well characterized <em>Rattus norvegicus</em> GulLO’s (RnGulLO) sequence as query. BLASTP result revealed the presence of seven candidates in <em>Arabidopsis, </em>and they were designated as AtGulLO1 to AtGulLO7 according to Maruta et al. [34] (<strong>Figure 2</strong>). GulLO belongs to the family of aldonolactone oxidoreductases. The similarity of amino acids among the AtGulLOs is between 22 and 28% to RnGulLO. A high similarity of the protein sequences is not found between the terminal enzymes in AsA biosynthesis among cross-species [15]. However, conserved features in this family of enzymes were found, such as an N-terminus flavin adenine dinucleotide (FAD)-binding domain (Pfam Id: 01565) and an arabinono lactone oxidase (ALO) catalytic domain (Pfam Id: 04030). The sequences of these domains were identified in all the candidate proteins by doing a BLAST search in conserved domains of Pubmed. The only exception to this was AtGulLO7 that does not have an N-terminus FAD binding domain. Further, these domains were also present in GLDHs characterized from some selected plant species (<strong>Figure 2A</strong>). The domain sequences were individually compared to that of RnGulLO using bl2seq to get a similarity score. The similarity of ALO domains is 20‒33% to RnGulLO and that of FAD binding domains to RnGulLO is 28‒36% (<strong>Figure 2A</strong>). An uncharacterized plant specific FAD-dependent oxidoreductase domain (TIGR01677) was found in all the putative GulLO candidates during the conserved domain search.</p>
<p><a href="wp-content/uploads/2017/09/figure-2-2.png"><img class="alignnone size-medium wp-image-1568" src="wp-content/uploads/2017/09/figure-2-2-300x206.png" alt="" width="300" height="206" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-2-2-300x206.png 300w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-2-2-768x526.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-2-2-1024x702.png 1024w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-2-2-1080x740.png 1080w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p><strong>FIGURE 2.</strong> <strong>Domains and phylogenetic analysis of GulLOs and GLDHs.</strong> Panel A, cartoon representation of domains found in rat GulLO (RnGulLO), <em>Arabidopsis</em> GulLOs and GLDH from various plant sources. Domain sequences of individual proteins were obtained from conserved domains in Pubmed and compared to that of RnGulLO using the Global Align program. The length of the domains was drawn in relation to RnGulLO, which is assumed to be 100%. The accession numbers of the sequences used are as follows: RnGulLO (P10867.3), <em>Arabidopsis</em> homologs (AtGulLO1 to AtGulLO7), and GLDHs from <em>Nicotiana tabacum</em> (NtGLDH; BAB13368.1), <em>Brassica oleracea</em> (BoGLDH; O47881.1), <em>Arabidopsis</em> <em>thaliana</em> (AtGLDH; NP_190376.1), <em>Ipomoea batatas</em> (IbGLDH; BAA34995.1) and <em>Solanum lycopersicum</em> (SlGLDH; NP_001234603.1). Panel B, a phylogenetic tree was constructed to show the similarity among the protein sequences in panel A. Bootstrap values shown are from 1000 replicates. Phylogenetic analyses were conducted in MEGA5. The two chosen <em>Arabidopsis</em> candidates, AtGulLO3 and AtGulLO5, are underlined.</p>
<p>Phylogenetic analysis was conducted in MEGA5 for AtGulLOs, RnGulLO and GLDHs using a maximum likelihood method for similarity of the protein sequences based on the JTT matrix-based model. The analysis showed that most AtGulLOs and GLDHs form two separate clades except AtGulLO3. RnGulLO and AtGulLO3 (At5g11540) lie close to each other and they are separated from the AtGulLO and GLDH clades (<strong>Figure 2B</strong>). Based on this, we speculated that AtGulLO3 may have a similar function as RnGulLO and AtGulLO3 was chosen for further study.</p>
<p>Further, two independent <em>Arabidopsis</em> T-DNA insertion lines (SALK 036899 and SALK 008089) were obtained from the ABRC SALK collection, to evaluate the involvement of <em>AtGulLO3</em> in AsA biosynthesis. In a segregated population of the two SALK lines, after selection on media containing kanamycin, AsA content was measured. The foliar AsA content of these putative knockouts was ~20% lower compared to wild type control plants (data not shown). This further supported the potential involvement of <em>AtGulLO3</em> in AsA synthesis.</p>
<p>Next, we performed a search at Genevestigator [42] to learn more about the expression pattern of these putative <em>GulLOs</em>. High abundance of <em>AtGulLO</em> transcripts is found during the early developmental stages of <em>A. thaliana</em> (<strong>Supplemental Figure 1</strong>) and in roots (<strong>Supplemental Figure 2</strong>). Unlike <em>AtGLDH</em>, the expressions of many <em>AtGulLOs</em> seem to be inducible (<strong>Supplemental Figure 3</strong>). Among all the <em>AtGulLO</em> candidates, <em>AtGulLO5</em> was of particular interest because of it being more strongly inducible than any other <em>AtGulLO</em> (<strong>Supplemental Figure 3B</strong>), high abundance in roots (<strong>Supplemental Figure 2</strong>) and its specific expression during the early developmental stages of <em>A. thaliana </em>(<strong>Supplemental Figure 1</strong>). Roots are the very organs that are exposed to various biotic and abiotic stresses and increased AsA content in plants is known to exhibit tolerance to abiotic stress. Maruta et al. [34] demonstrated <em>in vivo</em> the oxidation of L-GulL to AsA by three <em>AtGulLOs</em> including <em>AtGulLO5</em> in BY-2 cell suspension cultures. These facts lead us to choose <em>AtGulLO5</em> for further characterization.</p>
<p><a href="wp-content/uploads/2017/09/figure-3-2.png"><img class="alignnone size-medium wp-image-1569" src="wp-content/uploads/2017/09/figure-3-2-203x300.png" alt="" width="203" height="300" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-3-2-203x300.png 203w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-3-2-768x1134.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-3-2-694x1024.png 694w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-3-2-1080x1594.png 1080w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-3-2.png 1210w" sizes="(max-width: 203px) 100vw, 203px" /></a></p>
<p><strong>FIGURE 3.</strong> <strong>Multiple sequence alignment of AtGulLO5 with well characterized aldonolactone oxidoreductases.</strong> AtGulLOs and other GulLOs were aligned using ClustalW and manual alignment. Conservation of residues interacting with parts of the FAD cofactor (pyrophosphate, ADP, isoalloxazine and adenine) in the alignment is shown (G, glycine; H, histidine; s, small: G, A, S, T; h, hydrophobic: I, L, V; -, any residue). The residues present in gaps and termini are indicated by numbers in parentheses. The residue indicating oxygen reactivity is boxed. Also highlighted are the Glu-Arg pair in the catalytic site of AtGLDH and the cysteine residue known to be involved with thiol modifying agents in AtGLDH (arrow). Conservation of HWXK motif is also shown. The sequences used are: AtGLDH (<em>Arabidopsis</em>; Swiss-Prot: Q9SU56), AtGulLO1(<em>Arabidopsis</em>; GenBank: NP_564393.1), AtGulLO2 (<em>Arabidopsis</em>; GenBank: NP_182198.2), AtGulLO3 (<em>Arabidopsis</em>; GenBank: NP_196715.1), AtGulLO4 (<em>Arabidopsis</em>; GenBank: NP_200460.1), AtGulLO5 (<em>Arabidopsis</em>; GenBank: NP_182197.1), AtGulLO6 (<em>Arabidopsis</em>; GenBank: NP_182199.1), AtGulLO7 (<em>Arabidopsis</em>; GenBank: NP_200458.1), RnGulLO (rat; Swiss-Prot: P10867), ScALO (<em>S. cerevisiae</em>; Swiss-Prot: P54783), PgGUO (<em>P. griseoroseum</em>; Swiss-Prot: Q671X8), TcGalLO (<em>T. cruzi</em>; Swiss-Prot: Q4DPZ5), MtGulLDH (<em>M. tuberculosis</em>; Swiss-Prot: O06804).</p>
<p>Multiple sequence alignments were performed with AtGulLOs and well characterized aldonolactone oxidoreductases one from each of animals, fungi, protozoa, bacteria and plants using the ClustalW algorithm (<strong>Figure 3</strong>) followed by manual adjustment. All AtGulLO members except AtGulLO7 share high similarity among each other. We found significant similarity in the N termini and C termini of AtGulLOs and other GulLOs (<strong>Figure 3</strong>). An N terminus FAD-binding domain and residues that interact with parts of FAD molecule are conserved in all AtGulLOs including AtGulLO3 and AtGulLO5 (<strong>Figure 3</strong>). Aldonolactone oxidoreductases belong to the vanillyl alcohol oxidase (VAO) family and FAD binds to a histidine in the C terminus of VAO, the only protein known to have a crystal structure in this family [43]. This histidine is conserved in the HWXK motif in the C termini of all aldonolactone oxidoreductases [15], including AtGulLO3 and AtGulLO5 (<strong>Figure 3</strong>). A cysteine residue known to interact with thiol groups is also conserved in AtGulLO3 and AtGulLO5 (<strong>Figure 3</strong>).</p>
<p><strong>3.2. Purification and Characterization of Recombinant AtGulLO5 Transiently Expressed in <em>N. benthamiana</em></strong></p>
<p>To characterize AtGulLO5, we tested the enzyme activity <em>in vitro</em>. First, the recombinant expression of AtGulLO3-6xHIS and AtGulLO5-6xHIS proteins were confirmed by Western blot in <em>N. benthamiana</em> leaf samples collected 24, 48, 72 and 96 h post-infiltration. The time at which the protein is best expressed was found to be 48 h and this time point was used in all subsequent experiments (<strong>Figure 5</strong>).</p>
<p>The AtGulLO5 enzyme has 590 amino acids with a theoretical mass of 65 kDa and a pI of 8.2. It has at least one transmembrane domain as predicted [44]. AtGulLO5-EK-6xHIS (<strong>Figure 4C</strong>) was transiently expressed in <em>N. benthamiana </em>leaves for 48 h and was next purified by a two-step purification strategy as outlined (<strong>Figure 6A</strong>). The recombinant protein was almost completely extracted from leaf tissue in buffer with pH 8.0 consistent with the report that it is a cell -wall associated protein ([our unpublished observation and Ref. [44]). The final purified protein yielded a single band in SDS-PAGE followed by silver stain analysis (<strong>Figure 6B</strong>).</p>
<p>Recombinant enterokinase was successfully used to digest the 6xHIS tag from the recombinant protein since a low signal was seen in Western blot probed with anti-HIS-AP antibody compared to undigested protein (<strong>Figure 6C</strong>). Also, migration of the digested protein is distinguishable from the undigested protein in SDS-PAGE with silver stain (<strong>Figure 6B</strong>; Lane 5).</p>
<p><a href="wp-content/uploads/2017/09/figure-4-3.png"><img class="alignnone size-medium wp-image-1585" src="wp-content/uploads/2017/09/figure-4-3-300x247.png" alt="" width="300" height="247" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-4-3-300x247.png 300w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-4-3-768x632.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-4-3-1024x842.png 1024w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-4-3-1080x888.png 1080w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-4-3.png 1526w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p><strong>FIGURE 4.</strong><strong> Schematic representation of <em>GulLO</em> molecular constructs in the pBIB-Kan binary vector</strong>. Panel A, the constructs were made with either <em>AtGulLO3</em> or <em>AtGulLO5</em> ORF and are driven by a double-enhancer 35S constitutive promoter, a transcriptional enhancer (TEV) and a nopaline synthase terminator (NOS). Panel B, to facilitate detection and purification, a 6x-HIS tag was added in frame at the C-terminus of <em>AtGulLO3</em>, <em>AtGulLO5</em> or <em>RnGulLO</em> ORFs. C, a third construct was made only for <em>AtGulLO5</em> by adding an enterokinase (EK) recognition sequence between the ORF and the 6x-HIS tag for removal of the tag after purification.</p>
<p><a href="wp-content/uploads/2017/09/figure-5-2.png"><img class="alignnone size-medium wp-image-1586" src="wp-content/uploads/2017/09/figure-5-2-202x300.png" alt="" width="202" height="300" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-5-2-202x300.png 202w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-5-2-768x1142.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-5-2-688x1024.png 688w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-5-2.png 962w" sizes="(max-width: 202px) 100vw, 202px" /></a></p>
<p><strong>FIGURE 5.</strong><strong> Recombinant AtGulLO5-6xHIS and AtGulLO3-6xHIS protein expression in <em>N. benthamiana</em></strong>. Panel A, Western blot with α-HIS-AP antibody to detect the transiently expressed AtGulLO5-6xHIS protein in the total protein fraction of <em>N. benthamiana</em> leaves at indicated time points post-infiltration. EV, leaves transformed with empty pBIB-Kan vector at 48 h. Panel B, Western blot as described in panel A but for AtGulLO3-6xHIS protein.</p>
<p><a href="wp-content/uploads/2017/09/figure-6-2.png"><img class="alignnone size-medium wp-image-1587" src="wp-content/uploads/2017/09/figure-6-2-300x264.png" alt="" width="300" height="264" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-6-2-300x264.png 300w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-6-2-768x677.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-6-2-1024x902.png 1024w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-6-2-1080x951.png 1080w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p><strong>FIGURE 6.</strong> <strong>Two-step purification of recombinant AtGulLO5-EK-6xHIS expressed in <em>N. benthamiana </em>leaves.</strong> Panel A, protein purification scheme used to purify AtGulLO5-EK-6xHIS. Panel B, silver stain of SDS-PAGE for the fractions obtained from the purification procedure. The different lanes are: molecular weight marker (lane 1), crude extract (lane 2), elution from cation exchange chromatography (lane 3), elution from nickel affinity chromatography (lane 4) and protein digested with recombinant enterokinase (lane 5). Data is representative of at least twelve independent purifications. Panel C, Western blot with HIS-AP antibody to detect the AtGulLO5-EK-6xHIS protein before and after enterokinase digestion. The lane numbers are as in panel B.</p>
<p>The specific enzyme activity of our preparation was determined in both oxidase and dehydrogenase assays as described in materials and methods. Surprisingly, recombinant AtGulLO5 did not exhibit oxidase activity with any of the substrates tested (<strong>Table 2</strong>). However, it exhibited dehydrogenase activity with L-GulL (specific activity: 4.7 ± 0.3 mU/mg protein) which was not observed with any other substrates tested including L-GalL (<strong>Table 2</strong>). The characteristic flavin spectrum was absent and the enzyme activity was the same with or without exogenous addition of 100 µM FAD (data not shown). This is similar to the findings with the L-GulL dehydrogenase (GulLDH) enzyme from <em>Mycobacterium tuberculosis</em> [14].</p>
<table width="624">
<tbody>
<tr>
<td colspan="3" width="624"><strong>TABLE 2.</strong> <strong>Specific activity (dehydrogenase and oxidase) of the purified recombinant AtGulLO5 with different substrates</strong></td>
</tr>
<tr>
<td rowspan="2" width="263"><strong>Substrates Tested</strong><p />
<p /><strong>(200 mM)</strong></td>
<td colspan="2" width="361"><strong>Specific Activity (mU/mg protein)</strong></td>
</tr>
<tr>
<td width="213"><strong>Dehydrogenase (25<sup>o</sup>C)</strong></td>
<td width="149"><strong>Oxidase (37<sup>o</sup>C)</strong></td>
</tr>
<tr>
<td width="263">L-Gulono-1,4-lactone</td>
<td width="213">4.7 ± 0.3</td>
<td width="149">0</td>
</tr>
<tr>
<td width="263">L-Galactono-1,4-lactone</td>
<td width="213">0</td>
<td width="149">0</td>
</tr>
<tr>
<td width="263">D-Galactono-1,4-lactone</td>
<td width="213">0</td>
<td width="149">ND</td>
</tr>
<tr>
<td width="263">D-Gluconic acid-γ-lactone</td>
<td width="213">0</td>
<td width="149">ND</td>
</tr>
<tr>
<td colspan="3" width="624"><em>Note: Data are means ± SD (n=3). ND, not determined.</em></td>
</tr>
</tbody>
</table>
<p>Enzyme kinetic analysis was performed with L-GulL at a concentration of 5 mM to 400 mM. Kinetic constants were calculated from Michaelis‒Menten and Lineweaver‒Burk plots (<strong>Figure 7A </strong>and <strong>7B</strong>). The enzyme obeyed Michaelis‒Menten steady-state kinetics with an apparent K<sub>m</sub> of 33.8 mM and V<sub>max</sub> of 4.5 nmol AsA min<sup>‒1</sup> mg protein<sup>‒1</sup> (K<sub>cat</sub> = 0.005 s<sup>‒1</sup>).</p>
<p><a href="wp-content/uploads/2017/09/figure-7-1.png"><img class="alignnone size-medium wp-image-1571" src="wp-content/uploads/2017/09/figure-7-1-300x133.png" alt="" width="300" height="133" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-7-1-300x133.png 300w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-7-1-768x339.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-7-1-1024x453.png 1024w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-7-1-1080x477.png 1080w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p><strong>FIGURE 7.</strong> <strong>Enzyme kinetics of</strong> <strong>the recombinant AtGulLO5.</strong> Steady-state kinetic parameters were determined for the dehydrogenase activity with 5 to 400 mM L-gulono-1,4-lactone and 121 µM cytochrome C. Initial velocity, V<sub>o</sub> was defined as nmol of L-gulono-1,4-lactone oxidized per min per mg of the recombinant protein (mU mg<sup>-1</sup> protein). Michaelis-Menten (panel A) and double-reciprocal Lineweaver-Burke plots (panel B) are shown. Measurements were made in triplicate in three independent experiments; the values obtained in a representative experiment are shown.</p>
<p>Enzyme activity was inhibited when L-GulL concentration was increased to 400 mM (<strong>Figure 7A </strong>and <strong>7B</strong>). Leferink et al. [20] reported that AtGLDH activity was highly dependent on the ionic strength of the solution. So, we tested the effect of NaCl on AtGulLO5 activity. In contrast to results by Leferink et al. [20], AtGulLO5 enzyme activity was inhibited with NaCl at the concentration range tested (<strong>Figure 8A</strong>). Effects of temperature and pH in AtGulLO5 activity were also studied. The activity of the enzyme increased by up to 3-fold when the temperature was changed from 20<sup>o</sup>C to 25<sup>o</sup>C and the same was true until 35<sup>o</sup>C. At 40<sup>o</sup>C, the activity increased by at least another 1-fold (<strong>Figure 8B</strong>). The enzyme had a higher activity between pH 8.0 and 9.0 compared to lower pH (<strong>Figure 8C</strong>).</p>
<p><a href="wp-content/uploads/2017/09/figure-8.png"><img class="alignnone size-medium wp-image-1572" src="wp-content/uploads/2017/09/figure-8-155x300.png" alt="" width="155" height="300" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-8-155x300.png 155w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-8-768x1483.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-8-530x1024.png 530w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-8.png 962w" sizes="(max-width: 155px) 100vw, 155px" /></a></p>
<p><strong>FIGURE 8.</strong> <strong>Effects of NaCl (panel A), temperature (panel B) and pH (panel C) on the activity of the recombinant AtGulLO5.</strong> Panels A and B, the pH was maintained at 8.0. Measurements were made in duplicate; mean values ± SD are shown.</p>
<p><strong>3.3. In planta</strong><strong> Function of AtGulLO3 and AtGulLO5 in <em>N. benthamiana</em> Transient Expression System Showed no Significant Increase in Foliar AsA Content</strong></p>
<p>To demonstrate GulLO activity of <em>AtGulLO3</em> and <em>AtGulLO5</em> in vivo, a transient expression system utilizing <em>N. benthamiana</em> plants was chosen. Plant systems are advantageous to test the function by measuring AsA in leaves after infiltrating with the putative ORFs, since all plants make AsA. A similar system was used by Laing et al. [45] and Bulley et al. [46] to identify two of the enzymes involved in the L-galactose pathway.</p>
<p>First, the baseline AsA content was determined in all healthy leaves of <em>N. benthamiana</em>. Ascorbate content is higher in younger leaves (top; 4.05 ± 0.37 µmol/g FW) and it decreases as the leaves age (bottom; 1.88 ± 0.69 µmol/g FW; <strong>Figure 9</strong>). This pattern of AsA content decreasing as foliar tissue ages has been observed in <em>Arabidopsis</em> [47] but not in rice [48]. Next, AsA content was measured in <em>N. benthamiana</em> leaves, collected 48 h post infiltration in plants expressing <em>AtGulLO3-6xHIS</em>, <em>AtGulLO5-6xHIS</em>, <em>RnGulLO-6xHIS</em> (positive control)<em>,</em> or an empty pBIB-Kan vector (negative control), since, 48 h post infiltration has the highest recombinant protein expression (<strong>Figure 5</strong>). No significant increase in AsA content was found in the leaves expressing <em>AtGulLO3 </em>or<em> AtGulLO5 </em>constructs, although some increase in AsA was observed in leaves infiltrated with the <em>RnGulLO-6xHIS</em> construct (<strong>Figure 10</strong>). From this we learned that the substrate, L-GulL, was likely limiting in the leaves of <em>N. benthamiana</em> plants expressing the <em>AtGulLO</em> constructs.</p>
<p><a href="wp-content/uploads/2017/09/figure-9.png"><img class="alignnone size-medium wp-image-1573" src="wp-content/uploads/2017/09/figure-9-300x131.png" alt="" width="300" height="131" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-9-300x131.png 300w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-9-768x335.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-9-1024x446.png 1024w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-9-1080x471.png 1080w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-9.png 1507w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p><strong>FIGURE 9.</strong> <strong>Total, reduced (light grey) and oxidized (dark grey), AsA content of untransformed <em>N. benthamiana</em> leaves of different ages.</strong> Leaf numbers are as shown in the cartoon representation of the plant. Data shown are means ± SD (n = 5). One-way ANOVA was performed at a 95% confidence interval. Means that do not share a letter were significantly different (P &lt; 0.001).</p>
<p><a href="wp-content/uploads/2017/09/figure-10.png"><img class="alignnone size-medium wp-image-1574" src="wp-content/uploads/2017/09/figure-10-300x263.png" alt="" width="300" height="263" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-10-300x263.png 300w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-10-768x673.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-10.png 958w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p><strong>FIGURE 10.</strong><strong>  </strong><strong>Foliar AsA content of <em>N. benthamiana</em> infiltrated with GulLO constructs. </strong>Total, reduced (light grey), and oxidized (dark grey), AsA content of <em>N. benthamiana</em> leaves infiltrated with <em>AtGulLO3-6xHIS</em>, <em>AtGulLO5-6xHIS</em>, <em>RnGulLO-6xHIS,</em> or empty pBIB-Kan vector constructs. Data are means ± SD (n = 5). One-way ANOVA was performed at a 95% confidence interval and means that do not share a letter were significantly different (P &lt; 0.001).</p>
<p>In order to overcome the speculated substrate limitation, L-GulL or water was fed through the petioles of <em>N. benthamiana</em> leaves transiently expressing <em>AtGulLO3</em> or <em>AtGulLO5</em> constructs as described in materials and methods. Second and third leaves (metabolically active) from the top were used for AsA measurement in the feeding experiments for consistency. Ascorbate content was found to be increased in all leaves fed with L-GulL compared to water fed controls (<strong>Figure. 11A </strong>and<strong> 11B</strong>). However, no distinctive increase in AsA was observed in the leaves expressing <em>AtGulLO3</em> or <em>AtGulLO5</em> constructs compared to negative controls (<strong>Figure 11A </strong>and<strong> 11B</strong>). Plants not subjected to infiltration had more AsA in both the substrate and water fed groups.</p>
<p><a href="wp-content/uploads/2017/09/figure-11.png"><img class="alignnone size-medium wp-image-1575" src="wp-content/uploads/2017/09/figure-11-178x300.png" alt="" width="178" height="300" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-11-178x300.png 178w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-11-768x1297.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-11-606x1024.png 606w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-11-1080x1824.png 1080w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-11.png 1229w" sizes="(max-width: 178px) 100vw, 178px" /></a></p>
<p><strong>FIGURE 11.</strong><strong>  </strong><strong>Foliar AsA content of <em>N. benthamiana,</em> expressing AtGulLO3 or AtGulLO5, fed with substrate. </strong>Panel A, total, reduced (light grey), and oxidized (dark grey), AsA content of <em>N. benthamiana</em> leaves transiently expressing either <em>AtGulLO5</em> or <em>AtGulLO5-6xHIS</em> compared to plants infiltrated with the empty pBIB-Kan vector, and plants not subjected to infiltration (untransformed). Leaves were fed via petiole as described in materials and methods with 50 mM L-gulono-1,4-lactone or water. Data shown are means ± SD (n = 10). One-way ANOVA was performed at a 95% confidence. Means that do not share a letter were significantly different (P&lt;0.001). Data is representative of three independent experiments having similar results. Panel B, foliar AsA content of <em>N. benthamiana</em> expressing AtGulLO3 constructs, similar to what is described in panel A. Data are means ± SD (n = 3). One-way ANOVA was performed at a 95% confidence interval and means that do not share a letter were significantly different (P &lt; 0.001).</p>
<p><strong>3.4.<em> A. thaliana</em></strong><strong> AtGulLO3 and AtGulLO5 Transgenic Plants Did Not Have Increased Foliar AsA Content</strong></p>
<p>Stable <em>A. thaliana</em> transgenic lines were generated with <em>AtGulLO3-6xHIS</em> or <em>AtGulLO5-6xHIS</em> constructs and seventeen putative <em>AtGulLO3-6xHIS</em> transgenic lines were obtained after growing in selection media and fourteen putative lines were obtained for <em>AtGulLO5-6xHIS</em>. The presence of the <em>AtGulLO3, AtGulLO5</em> and <em>nptII</em> transgenes was verified by PCR (data not shown). Three lines (L1, L12 and L15), for <em>AtGulLO3-6xHIS</em>, were chosen for further study based on their <em>AtGulLO</em> mRNA expression (data not shown). Two best lines (L6 and L14), for <em>AtGulLO5-6xHIS</em>, were chosen for further study. Next, the progeny of the chosen lines was grown to get a segregating population and AsA precursors were fed through leaf petioles. The foliar AsA content was measured and no significant increase was observed in the transgenic lines compared to negative controls (<strong>Figure 12A </strong>and<strong> 12B</strong>). This observation is similar to that of the previous results obtained in <em>N. benthamiana</em> expressing <em>AtGulLO3</em> or <em>AtGulLO5</em> constructs (<strong>Figure 11</strong>).</p>
<p><a href="wp-content/uploads/2017/09/figure-12.png"><img class="alignnone size-medium wp-image-1576" src="wp-content/uploads/2017/09/figure-12-219x300.png" alt="" width="219" height="300" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-12-219x300.png 219w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-12-768x1051.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-12-748x1024.png 748w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-12-1080x1478.png 1080w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-12.png 1469w" sizes="(max-width: 219px) 100vw, 219px" /></a></p>
<p><strong>FIGURE 12.</strong><strong>  </strong><strong>Foliar AsA content of <em>A. thaliana, </em>expressing AtGulLO3 or AtGulLO5, fed with substrate.</strong> Panel A, total, reduced (light grey) and oxidized (dark grey) leaf AsA content of three T2 generation transgenic <em>A. thaliana </em>lines expressing <em>AtGulLO3-6xHIS </em>or empty pBIB-Kan vector. L-Gulono-1,4-lactone (10 mM) or water was fed through leaf petioles as described in “materials and methods”. Data are means ± SD (n = 3). One-way ANOVA was performed at a 95% confidence interval and means that do not share a letter were significantly different (P &lt; 0.001). Panel B, foliar AsA content of <em>A. thaliana</em> expressing AtGulLO5 constructs, similar to what is described in panel A. Data are means ± SD (n = 5). One-way ANOVA was performed at a 95% confidence interval and means that do not share a letter were significantly different (P &lt; 0.001).</p>
<p><strong>3.5. AtGulLO3 Protein Appears to Be Regulated Post-Transcriptionally by Rapid Turnover</strong></p>
<p>In an attempt to understand the reason(s) for not having increased foliar AsA in the transgenic <em>Arabidopsis</em> lines, protein expressions of AtGulLO3 and AtGulLO5 were measured. Western blot performed with the <em>Arabidopsis</em> AtGulLO5-6xHIS transgenic lines (L6 and L14) showed the presence of AtGulLO5-6xHIS protein expression in the lines tested (data not shown).</p>
<p>On the other hand, two plants were sampled at random for each of the two homozygous AtGulLO3-6xHIS <em>Arabidopsis</em> lines, L12 and L15. In contrast to AtGulLO5, no protein expression was found in either of the lines tested for AtGulLO3-6xHIS (<strong>Figure 13B</strong>). However, both the endogenous and transgenic <em>AtGulLO3</em> transcript expression was found in both L12, L15 and also in the wild type (control) plants by RT-PCR (<strong>Figure 13A</strong>). When transiently expressed in <em>N. benthamiana</em> leaves, AtGulLO3-6xHIS protein was found to be slightly expressed at 48 h and no expression was found thereafter up to 96 h (<strong>Figure 5B</strong>). Taken together, our results suggest that AtGulLO3 is short lived when expressed in both transient (<em>N. benthamiana</em>) and stable (<em>A. thaliana</em>) platforms indicating that its expression at protein level is tightly regulated.</p>
<p><a href="wp-content/uploads/2017/09/figure-13.png"><img class="alignnone size-medium wp-image-1577" src="wp-content/uploads/2017/09/figure-13-300x300.png" alt="" width="300" height="300" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-13-300x300.png 300w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-13-150x150.png 150w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-13-768x770.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-13.png 962w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p><strong>FIGURE 13.</strong><strong>  </strong><strong>AtGulLO3 protein and mRNA detection in transgenic <em>A. thaliana</em>. </strong>Panel A, <em>AtGulLO3</em> transcript was detected by RT-PCR in two <em>A. thaliana</em> transgenic lines (L12 and L15) as described in “materials and methods”. The elongation factor gene, <em>EF1α-A,</em> was used as a reference. EV, Empty Vector. Panel B, recombinant AtGulLO3-6xHIS was detected by Western blot with HIS-AP antibody in total protein fraction of two <em>A. thaliana </em>AtGulLO3-6xHIS transgenic lines. M, Marker; EV, <em>A. thaliana </em>transformed with empty pBIB-kan vector as a negative control; (+), HSP90 α-HIS protein as a positive control.</p>
<table width="624">
<tbody>
<tr>
<td colspan="7" width="624"><strong>TABLE 3.</strong> <strong>Comparison of biochemical parameters of aldonolactone oxidoreductases characterized from five kingdoms</strong></td>
</tr>
<tr>
<td rowspan="2" width="113"><strong>Properties</strong></td>
<td colspan="2" width="168"><strong>              Plants</strong></td>
<td rowspan="2" width="78"><strong>GulLDH (Bacteria)</strong></td>
<td rowspan="2" width="83"><strong>GulLO (Mammals)</strong></td>
<td rowspan="2" width="88"><strong>ALO </strong><p />
<p /><strong>(Yeast)</strong></td>
<td rowspan="2" width="94"><strong>GalLO</strong><p />
<p /><strong>(Protozoa)</strong></td>
</tr>
<tr>
<td width="72"><strong>GulLO<sup>a</sup></strong></td>
<td width="96"><strong>GLDH</strong></td>
</tr>
<tr>
<td width="113">Molecular mass (kDa)</td>
<td width="72">65</td>
<td width="96">56</td>
<td width="78">61</td>
<td width="83">51</td>
<td width="88">56‒60</td>
<td width="94">57 (<em>T. cruzi</em>);<p />
<p />59 (<em>T. brucei</em>)</td>
</tr>
<tr>
<td width="113">Relative substrate specificity</td>
<td width="72"></td>
<td width="96"></td>
<td width="78"></td>
<td width="83"></td>
<td width="88"></td>
<td width="94"></td>
</tr>
<tr>
<td width="113">       L-Galactono-1,4-lactone</td>
<td width="72">0</td>
<td width="96">100</td>
<td width="78">0</td>
<td width="83">87</td>
<td width="88">87</td>
<td width="94">100 (<em>T. cruzi</em>)<p />
<p />74 (<em>T. brucei</em>)</td>
</tr>
<tr>
<td width="113">       L-Gulono-1,4-lactone</td>
<td width="72">100</td>
<td width="96">0‒20</td>
<td width="78">100</td>
<td width="83">100</td>
<td width="88">24</td>
<td width="94">3‒9</td>
</tr>
<tr>
<td width="113">       D-Arabinono-1,4-lactone</td>
<td width="72">ND</td>
<td width="96">ND</td>
<td width="78">ND</td>
<td width="83">ND</td>
<td width="88">100</td>
<td width="94">96 (<em>T. cruzi</em>)<p />
<p />100 (<em>T. brucei</em>)</td>
</tr>
<tr>
<td width="113">K<sub>m </sub>(preferred substrate)</td>
<td width="72">33.8 mM</td>
<td width="96">0.12–3.3 mM;<p />
<p />13.9 mM (L-GulL)<sup>c</sup></td>
<td width="78">5.5 mM (<em>Mt</em>);<p />
<p />34.8 mM (<em>Go</em>)<sup>d</sup></td>
<td width="83">0.066 mM (rat);<p />
<p />0.15 mM (goat)</td>
<td width="88">44.1 mM;<p />
<p />24 mM<sup>f</sup></td>
<td width="94">0.16 mM (<em>T. cruzi</em>);<p />
<p />0.05 mM (<em>T. brucei</em>)</td>
</tr>
<tr>
<td width="113">Electron acceptor</td>
<td width="72">Cyt C</td>
<td width="96">Cyt C</td>
<td width="78">Cyt C, PMS</td>
<td width="83">O<sub>2</sub></td>
<td width="88">O<sub>2</sub>, PMS<sup>f</sup></td>
<td width="94">Cyt C</td>
</tr>
<tr>
<td width="113">Subcellular localization</td>
<td width="72">Cell wall<sup>b</sup></td>
<td width="96">Mitochondria</td>
<td width="78">Cell wall<sup>e</sup></td>
<td width="83">Microsomes</td>
<td width="88">Mitochondria</td>
<td width="94">Glycosomes</td>
</tr>
<tr>
<td colspan="7" width="624"><em>Note: </em><em>Cyt C, cytochrome C; Go, Gluconobacter oxydans; PMS, Phenazine mthosulfate; Mt, Mycobacterium tuberculosis, and ND, not determined. References: <sup>a</sup>this study, <sup>b</sup>[44] and our unpublished observation, <sup>c</sup>[20], <sup>d</sup>[49], <sup>e</sup>[50], and <sup>f</sup>[51].</em></td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<table width="624">
<tbody>
<tr>
<td colspan="7" width="624"><strong>TABLE 4. Differential AsA synthesis with L-GulL or L-GalL substrate feeding in plants</strong></td>
</tr>
<tr>
<td rowspan="2" width="126"><strong>Species</strong><p />
<p /><strong>(Scientific name)</strong></td>
<td rowspan="2" width="102"><strong>Explant used</strong></td>
<td colspan="2" width="174"><strong>L-GalL or L-galactose</strong></td>
<td colspan="2" width="174"><strong>L-GulL</strong></td>
<td rowspan="2" width="48"><strong>Refs</strong></td>
</tr>
<tr>
<td width="102"><strong>Substrate concentration (mM)</strong></td>
<td width="72"><strong>(% AsA increase)</strong></td>
<td width="102"><strong>Substrate concentration (mM)</strong></td>
<td width="72"><strong>(% AsA increase)</strong></td>
</tr>
<tr>
<td rowspan="2" width="126">Apple<p />
<p />(<em>Malus domestica</em>)</td>
<td width="102">Flesh peel, flesh, and seed of young fruit</td>
<td width="102">L-GalL (10)<p />
<p />L-Gal (10)</td>
<td width="72">142<p />
<p />142</td>
<td width="102">L-GulL (10)</td>
<td width="72">100</td>
<td rowspan="2" width="48">[53]</td>
</tr>
<tr>
<td width="102">Flesh peel, flesh, and seed of mature fruit</td>
<td width="102">L-GalL (10)<p />
<p />L-Gal (10)</td>
<td width="72">132<p />
<p />127</td>
<td width="102">L-GulL (10)</td>
<td width="72">100</td>
</tr>
<tr>
<td width="126">Apple<p />
<p />(<em>Malus domestica</em>)</td>
<td width="102">Leaf disk</td>
<td width="102">L-GalL (15)<p />
<p />L-Gal (15)</td>
<td width="72">300<p />
<p />300</td>
<td width="102">L-GulL (15)</td>
<td width="72">150</td>
<td width="48">[54]</td>
</tr>
<tr>
<td width="126">Arabidopsis<p />
<p /><em>(Arabidopsis thaliana)</em></td>
<td width="102">Cell culture</td>
<td width="102">L-GalL (15)<p />
<p />L-Gal (15)</td>
<td width="72">3180<p />
<p />6800</td>
<td width="102">L-GulL (15)</td>
<td width="72">970</td>
<td width="48">[29]</td>
</tr>
<tr>
<td width="126">Bean<p />
<p />(<em>Phaseolus vulgaris</em>)</td>
<td width="102">Bean shoots</td>
<td width="102">L-GalL (28.1)</td>
<td width="72">255</td>
<td width="102">L-GulL (28.1)</td>
<td width="72">85</td>
<td width="48">[28]</td>
</tr>
<tr>
<td width="126">Black Currant<p />
<p />(<em>Ribes nigrum </em>L.)</td>
<td width="102">Flowers</td>
<td width="102">L-GalL (25)<p />
<p />L-Gal (25)</td>
<td width="72">235<p />
<p />296</td>
<td width="102">L-GulL (25)</td>
<td width="72">108</td>
<td width="48">[55]</td>
</tr>
<tr>
<td width="126">Cress<p />
<p />(<em>Lepidium sativum</em>)</td>
<td width="102">Germinating cress seedlings</td>
<td width="102">L-GalL (28.1)</td>
<td width="72">482</td>
<td width="102">L-GulL (56.1)</td>
<td width="72">148</td>
<td width="48">[26]</td>
</tr>
<tr>
<td width="126">Kiwi<p />
<p />(<em>Actinidia deliciosa</em>)</td>
<td width="102">Flesh discs</td>
<td width="102">L-GalL (10)<p />
<p />L-Gal (10)</td>
<td width="72">168<p />
<p />162</td>
<td width="102">L-GulL (10)</td>
<td width="72">150</td>
<td width="48">[56]</td>
</tr>
<tr>
<td width="126">Pea<p />
<p />(<em>Pisum sativum</em> L.)</td>
<td width="102">Embryonic axes</td>
<td width="102">L-GalL (25)</td>
<td width="72">420</td>
<td width="102">L-GulL (25)</td>
<td width="72">124</td>
<td width="48">[30]</td>
</tr>
<tr>
<td width="126">Peach<p />
<p />(<em>Prunus persica</em> L.)</td>
<td width="102">Immature whole fruits</td>
<td width="102">L-GalL (25)<p />
<p />L-Gal (50)</td>
<td width="72">200<p />
<p />350</td>
<td width="102">L-GulL (25)</td>
<td width="72">125</td>
<td width="48">[57]</td>
</tr>
<tr>
<td width="126">Tobacco<p />
<p />(<em>Nicotiana tabacum</em>)</td>
<td width="102">Young leaves</td>
<td width="102">L-GalL (30)</td>
<td width="72">6932</td>
<td width="102">L-GulL (30)</td>
<td width="72">1068</td>
<td width="48">[21]</td>
</tr>
<tr>
<td rowspan="2" width="126">Tomato<p />
<p />(<em>Solanum lycopersicum</em>)</td>
<td width="102">Mature fruit discs</td>
<td width="102">L-GalL (15)<p />
<p />L-Gal (15)</td>
<td width="72">287<p />
<p />246</td>
<td width="102">L-GulL (15)</td>
<td width="72">180</td>
<td rowspan="2" width="48">[58]</td>
</tr>
<tr>
<td width="102">Red fruit discs</td>
<td width="102">L-GalL (15)<p />
<p />L-Gal (15)</td>
<td width="72">192<p />
<p />201</td>
<td width="102">L-GulL (15)</td>
<td width="72">151</td>
</tr>
</tbody>
</table>
<p><strong>4. DISCUSSION</strong></p>
<p>Conversion of L-GulL to AsA has been observed in plants since 1954 [26] (<strong>Table 4</strong>) and corresponding enzyme activity has also been detected in hypocotyl homogenates of kidney beans [31], cytosolic and mitochondrial fractions of <em>Arabidopsis</em> cell suspension [29]and potato tubers [9, 33] but no enzyme has been characterized in detail until now. Recently, Wheeler et al. [52] proposed that plants have been replaced with the GLDH enzyme instead of a GulLO. However, we argue that plants possess a GulLO enzyme in addition to GLDH because of the many precursor feeding reports (<strong>Table 4</strong>), the above-mentioned enzyme activity detection, and the evidence presented in this paper.</p>
<p>We characterized the AtGulLO5 enzyme in detail and calculated its kinetic parameters. The K<sub>m</sub> and V<sub>max</sub> with L-GulL were found to be 33.8 mM and 4.5 nmol AsA per min per mg protein, respectively. The Michealis‒Menten constant for AtGulLO5 appears to be higher, in comparison to the reported values for GulLO isozymes (0.05 mM to 5.5 mM) from rat, goat, chicken, fungi, bacteria and trypanosomes (<strong>Table 3</strong>). However, some isozymes with high K<sub>m</sub>, similar to our report, have also been documented in <em>Candida albicans</em>, <em>Gluconobacter oxydans</em> and <em>Grifola frondosa</em> (<strong>Table 3</strong>). The specific activity (4.5 mU/mg; K<sub>cat</sub> = 0.005 s<sup>‒1</sup>), we observed for AtGulLO5, is lower compared to other reports for isozymes from other species and it could have been increased by another 2-fold if the assay were conducted in 40<sup>o</sup>C and pH 9.0 (<strong>Figure 8A</strong> and <strong>8B</strong>). A specific activity of 66 mU/mg for L-GulL dehydrogenase (GulLDH) from <em>M. tuberculosis</em> was the lowest reported value thus far among all the previous studies of aldonolactone oxidoreducatases [14].</p>
<p>The reason for the observed low specific activity of AtGulLO5 may be due to the absence of a flavin prosthetic group. A flavin prosthetic group is also absent in <em>M. tuberculosis</em> GulLDH [14] as evidenced in both by the lack of a flavin spectrum and insensitivity of exogenous  FAD addition  in activity  assay, even though a histidine is present in the N-terminus FAD-binding domain (<strong>Figure 3</strong>). The flavin moiety is involved in the enzyme catalysis as observed by the disappearance of 450 nm peak upon substrate addition in previous reports [12, 16, 20, 51, 59, 60]. Flavoprotein inhibitors such as riboflavin and atebrin inhibited cytochrome activity of cauliflower GLDH but not phenazine activity [16]. Interestingly, <em>M. tuberculosis</em> GulLDH had higher activity when phenazine methosulfate was used as an electron acceptor similar to the findings of Mapson and Breslow [16]. No flavin prosthetic group is found when aldonolactone oxidoreductases—known to have a covalent FAD but not non-covalent FAD—are expressed recombinantly in <em>E. coli</em> [61, 62]. However, no problem with the attachment of the flavin group was experienced when aldonolactone oxidoreductases were expressed in eukaryotic systems [63‒65]. Therefore, AtGulLO5 seems to be an exception to this observation when expressed in a eukaryotic system.</p>
<p>The catalytic efficiency of AtGulLO5 (K<sub>cat</sub> = 0.005 s<sup>‒1</sup>) is significantly lower compared to that of the other enzymes in the plant AsA pathway such as AtVTC2 (27 s<sup>‒1</sup>) and AtVTC5 (13 s<sup>‒1</sup>; [66]). However, it is similar to that of recombinant GME from <em>Arabidopsis</em> (K<sub>cat</sub> = 0.007 s<sup>‒1</sup>; [9]). The authors that studied GME proposed that this enzyme needs an effector molecule such as Hsc70.3 for its high catalytic efficiency. Thus, AtGulLO5, similar to GME, may need an effector molecule that is produced during certain conditions, for example, stress and thus may be regulated post-transcriptionally.</p>
<p>On the other hand, AtGulLO3 protein expression is absent in <em>A. thaliana</em> (stable expression; <strong>Figure 13B</strong>) and its expression appears only after 48 h post-infiltration in <em>N. benthamiana</em> (transient expression; <strong>Figure 5B</strong>). The expression is not found thereafter up to 96 h post-infiltration. These results are indicative of rapid protein turnover. Thus, it seems that both AtGulLO3 and AtGulLO5 are regulated post-transcriptionally, however, by different mechanisms, i.e, an effector molecule may increase the catalytic efficiency of AtGulLO5 while AtGulLO3 may use another mechanism to maintain its stability from proteolytic degradation.</p>
<p>Our attempts to prove <em>in planta</em> enzyme function of AtGulLO3 and AtGulLO5 did not succeed. No increase in foliar AsA content was observed when <em>AtGulLO3</em> or <em>AtGulLO5</em> constructs were expressed (<strong>Figure 10</strong>). It was presumed that L-GulL availability was limiting in these leaves. Feeding L-GulL to <em>N. benthamiana</em> or <em>A. thaliana</em> leaves expressing <em>AtGulLO3</em> or <em>AtGulLO5</em> constructs did not show a distinctive increase in AsA compared to that of empty vector (<strong>Figures 11 </strong>and<strong> 12</strong>). In contrast to our observations, tobacco BY-2 cell suspension cultures over-expressing <em>AtGulLO3</em> and <em>AtGulLO5</em> had an increased AsA level after L-GulL feeding suggesting GulLO function in BY-2 cells [34][34]. Interestingly, the results of Tokunaga et al. [67] and Imai et al. [25] are also contrasting when GLDH was overexpressed in tobacco cell cultures and in tobacco plants, respectively. GLDH over-expression increased AsA levels in BY-2 cells [67] but no increase in foliar AsA levels were found when GLDH was over-expressed in tobacco plants [25]. These observations may be partly attributed to the source material used in our study and in the study of Maruta et al. [34] as was the case observed in studies by Imai et al. [25] and Tokunaga et al. [67] with GLDH over-expression and L-GalL feeding.</p>
<p>The failure to increase AsA in <em>N. benthamiana</em> or <em>A. thaliana</em> leaves indicates that enzyme activity was a limiting factor. The proposed post-transcriptional regulation of AtGulLO3 and AtGulLO5 offers a reason for the limited GulLO enzyme availability. Although the above-mentioned experiments with GLDH and GulLO overexpression did not increase foliar AsA content, the case with GLDH is different than with GulLO. Feeding the same concentration of L-GulL does not amount to the same AsA accumulation as with L-galactose feeding in <em>A. thaliana</em> and <em>N. benthamiana</em> (<strong>Figure 14</strong>). This is true in many plant species, where AsA content readily increases upon L-GalL or L-galactose feeding whereas L-GulL feeding does not increase AsA content to the same extent (<strong>Table 4</strong>). Differential substrate uptake is not the reason for this difference [28, 29]. It is not the case that there is a general increase in carbohydrate pool, which indirectly stimulates AsA synthesis, since radiolabeled L-GulL was directly converted to AsA without any carbon skeleton break or inversion [28, 68]. It should be noted that the GLDH enzyme is very efficient and even a small amount of GLDH activity is enough to catalyze L-GalL to AsA. For example, RNAi tomato plants, with only 30% of wild type GLDH activity, had similar AsA content as wild type [69]. In isotope dilution experiments conducted in pea embryonic axes, it was found that the pool size of L-GalL is very small and its conversion to AsA is very fast indicating the high efficiency of GLDH [30]. The limiting factor for AsA synthesis with GLDH was the substrate, L-GalL (<strong>Figure 14</strong>; <strong>Table 4</strong>). However, considering the above-mentioned facts our findings indicate that AsA synthesis is limited by GulLO enzyme availability via L-GulL.</p>
<p><a href="wp-content/uploads/2017/09/figure-14.png"><img class="alignnone size-medium wp-image-1578" src="wp-content/uploads/2017/09/figure-14-300x187.png" alt="" width="300" height="187" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-14-300x187.png 300w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-14-768x479.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-14-1024x638.png 1024w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-14-400x250.png 400w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-14-1080x673.png 1080w, http://www.aimsci.com/ros/html/wp-content/uploads/2017/09/figure-14.png 1592w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p><strong>FIGURE 14.</strong><strong>  </strong><strong>Foliar AsA content of wild type <em>A. thaliana </em>(panel A) and <em>N. benthamiana </em>(panel B) after exogenous substrate feeding</strong>. Leaves at the developmental stage 5.10 (panel A) or from 6 weeks old plants (panel B) were used. Substrates, at an indicated concentration of 15 mM (panel A) or 50 mM (panel B), were exogenously added for 16 h (panel A) or 9 h (panel B) as described in “materials and methods”. Data are means ± SD (n = 3 in panel A; n = 8 in panel B) and are representative of at least three independent experiments. One-way ANOVA was performed at 95% confidence interval and means that do not share a letter were significantly different (P &lt; 0.001).</p>
<p>Our results also indicate that the post-transcriptional regulation for AtGulLO3 and AtGulLO5 is present only in whole plants but not in cell cultures. Yabuta et al. [70] have also suggested post-transcriptional regulation of AtGLDH activity <em>in planta</em>, which is not present in cultured cells [67]. Data mined from the Genevestigator depository indicates that the expression of <em>AtGulLO3</em> and <em>AtGulLO5</em> in cell cultures is higher compared to leaves (<strong>Supplemental Figure 2</strong>). This also points to the fact that the regulation is present in whole plants but not in cell cultures. Interestingly, the previous studies that detected GulLO activity were in hypocotyl [31], tubers [9, 33] and cell cultures [29]. This is similar to the Genevestigator expression data that shows the abundance of GulLO transcripts in early developmental stages, roots, and cell cultures (<strong>Supplemental Figures 1 </strong>and<strong> 2</strong>). Therefore, in addition to cell cultures, early developmental stage and roots are probably the other conditions where the proposed regulation is not present.</p>
<p>On the other hand, RnGulLO may not undergo such a post-transcriptional regulation as plant GulLOs undergo, because RnGulLO was able to use the available L-GulL pool, as evidenced by an increase in AsA content in several plant species when RnGulLO was overexpressed [21‒23], <strong>Figure 10</strong>). Although RnGulLO can use both L-GalL and L-GulL as substrates [12], it is conceivable that RnGulLO uses L-GulL for two reasons. First, GLDH is very efficient in using L-GalL [20, 33] and it was only the limitation of L-GalL substrate that prevented GLDH overexpression to increase AsA [25]. Second, RnGulLO successfully rescued vitamin C deficient (<em>vtc1</em>) mutant [71] and the level of L-GalL is reduced in <em>vtc1</em> mutant [72]. These observations also indicate the limited availability of GulLO enzyme activity to synthesize AsA via L-GulL.</p>
<p>Another interesting result, we obtained, is that AtGulLO5 has an absolute specificity for L-GulL. To our knowledge, this is the first report of a plant enzyme to have such specificity for L-GulL and it supports the earlier suggestion of Baig et al. [28] that a specific enzyme for L-GulL may exist in plants. The substrate specificity of GulLO isozymes is diverse. Plant GLDHs—characterized from different plant species—are strictly specific to L-GalL [16‒19, 32]. Exceptions to this are GLDHs from <em>Arabidopsis</em> [20], white potato [33], and a recombinant GLDH of tobacco but not the native [73], all of which can also use L-GulL at a very low efficiency. Similar to plant GLDHs, protozoan parasites are specific only to L-GalL and/or D-arabinono-1,4-lactone (D-AL) but not to L-GulL [15, 60, 74]. On the other hand, vertebrate enzymes from rat, goat and chicken can utilize both L-GulL and L-GalL as substrates [12, 74, 75]. Similar to vertebrates is the enzyme from <em>Euglena gracilis</em> [76] that uses both L-GulL and L-GalL as substrates. Yeast isozymes can use D-AL in addition to L-GulL and L-GalL [13, 77]. Bacterial GulLDHs are specific to L-GulL and do not use L-GalL as substrate [14, 49]. Interestingly, our phylogenetic analysis shows AtGulLOs, including AtGulLO5 (specific to L-GulL) and GLDHs from plants (specific to L-GalL), in two separate clades (<strong>Figure 2B</strong>). However, it is not known if the other AtGulLOs are specific only to L-GulL. RnGulLO, utilizing both L-GulL and L-GalL lies in between the two clades (<strong>Figure 2B</strong>) and it would be interesting to test whether AtGulLO3 has similar substrate specificity as RnGulLO. Furthermore, AtGulLO5 and GulLDH of <em>M. tuberculosis</em> [14]—both specific only to L-GulL—have a glycine in the HW<u>X</u>K motif whereas, AtGLDH (specific only to L-GalL) has an alanine (<strong>Figure 3</strong>). But it is not known if Ala and Gly in the motif play a role in substrate specificity.</p>
<p>We found AtGuLO5 to be an exclusive dehydrogenase with no oxidase activity (<strong>Table 1</strong>). Similar to AtGulLO5, there are two exclusive dehydrogenases specific to L-GulL—among aldonolactone oxidoreductases—in <em>G. oxydans</em> [49], <em>M. tuberculosis</em> [14]. Plant GLDHs [17, 20] and <em>E. gracilis</em> GulLDH [76] are also exclusive dehydrogenases but their substrate specificities are different. A gatekeeper amino acid residue was identified in AtGLDH, which determines whether the oxidoreductase is a dehydrogenase or an oxidase. It was found that glycine or proline can be present in oxidases while an alanine is present in dehydrogenases with some exceptions ([78], <strong>Figure 3</strong>). Although, AtGulLO5 and <em>M. tuberculosis</em> contain a proline and a glycine, respectively (<strong>Figure 3</strong>), they do not possess oxidase activity. This may be due to the absence of a flavin prosthetic group. The GulLO isozymes in animals, yeast and fungi are oxidases and in addition, they also possess dehydrogenase activity (<strong>Table 3</strong>).</p>
<p>AtGulLO5 is localized in the cell wall ([44] and our unpublished observation), similar to that of GulLDH in <em>M. tuberculosis</em> [14, 50]. The GulLDH of <em>E. gracilis</em> is localized in both mitochondria and cytosol [75]. Mammalian GulLOs are localized in the microsomes, protozoan isozymes are localized in glycosomes, while yeast and plant isozymes are localized in mitochondria (<strong>Table 3</strong>).</p>
<p>Microarray data from Genevestigator database together with our PCR amplification experiment (data not shown) showed that <em>AtGulLO5</em> is abundantly expressed in roots (<strong>Supplemental Figure 2</strong>). However, Franceschi and Tarlyn [79] found very low level of AsA synthesis with L-GalL feeding in roots of <em>Medicago sativa</em>, suggesting that the L-galactose pathway is not operational in the roots of <em>M. sativa</em> and the alternate pathway using L-GulL may be operational in roots. It would be interesting to characterize AtGulLO2, the next best expressed candidate in roots (<strong>Supplemental Figure 3</strong>).</p>
<p>Gene expression data from Genevestigator suggest that AsA synthesis through <em>AtGLDH</em> appears to be constitutive and not inducible, while the L-gulose and <em>myo</em>-inositol routes to ascorbate involving <em>AtGulLO</em> may be operational only under specific developmental stages such as the early developmental stage (<strong>Supplemental Figure 1</strong>). It may also be operational in specific parts of the plants such as roots (<strong>Supplemental Figure 2</strong>) and is induced upon certain stimuli such as hormones, germination, callus formation, plant development, nematode attack and some abiotic stresses (<strong>Supplemental Figure 3B</strong>). It is interesting to note that <em>MIOX</em> transcripts have been reported to be induced upon certain stimuli such as anoxia [80], syncytia [81], low nutrients [82], and zinc deficiency [83]. In addition, increases in <em>MIOX</em> transcripts have been observed, with an associated increase in AsA, during ripening in strawberry [84] and at the first stages of development in citrus fruits [85]. Wolucka et al. [86] have also reported the induction of <em>AtGulLO2</em> and GME transcripts when <em>Arabidopsis</em> cell suspensions were treated with methyl jasmonate suggesting the inducible nature of the L-gulose route.</p>
<p><strong style="line-height: 1.7em;">5. CONCLUSIONS</strong></p>
<p>In conclusion, we characterized two members of the <em>Arabidopsis</em> AtGulLO family, AtGulLO3 and AtGulLO5. Our detailed characterization of AtGulLO5 showed that it has an absolute specificity for L-GulL differing from the plant GLDHs that are highly specific to L-GalL and mammalian GulLOs that use both substrates. However, the biochemical properties of the plant enzyme are more similar to the ones of <em>M. tuberculosis</em> GulLDH in terms of its substrate specificity, subcellular localization, use of electron acceptor and specific activity. Our results indicate that both AtGulLO3 and AtGulLO5 are regulated post-transcriptionally by two different mechanisms. AtGulLO5 may need an effector molecule to increase its catalytic efficiency while AtGulLO3 may need to be protected from the proposed rapid turnover. The proposed regulation limits GulLO enzyme availability in plants and thus their over-expression in <em>N. benthamiana</em> or <em>A. thaliana</em> did not lead to elevated foliar AsA levels. Future research aimed at understanding the regulatory mechanism(s) will aid in designing a strategy to increase AsA pool in plants utilizing the L-GulL substrate pool.</p>
<p><strong>ACKNOWLEDGMENTS</strong></p>
<p>The authors would like to thank K Lee for help with plant care, B Savary, P Vasu and J Tovar for suggestions on enzyme purification, JP Yactayo-Chang for help with cloning, and K Lisko and Z Campbell for critical reading of the manuscript. This study was supported with funds from the Arkansas Biosciences Institute, the major research component of the Arkansas Tobacco Settlement Proceeds Act of 2000, and a sub-award from the Arkansas INBRE program [National Center for Research Resources (5P20RR016460-11) and the National Institute of General Medical Sciences (8P20GM103429-11) from the National Institutes of Health]. S.I.A. thanks the Molecular Biosciences PhD program at Arkansas State University for a scholarship. The authors declare no conflicts of interest.</p>
<p><strong>REFERENCES</strong></p>
<ol>
<li>De Tullio MC, Arrigoni O. Hopes, disillusions and more hopes from vitamin C. <em>Cell Mol Life Sci</em> 2004; 61(2):209‒19. doi: 10.1007/s00018-003-3203-8.</li>
<li>Nishikimi M, Koshizaka T, Ozawa T, Yagi K. Occurrence in humans and guinea pigs of the gene related to their missing enzyme L-gulono-gamma-lactone oxidase. <em>Arch Biochem Biophys</em> 1988; 267(2):842‒6.</li>
<li>Noctor G, Foyer CH. Ascorbate and glutathione: keeping active oxygen under control. <em>Annu Rev Plant Physiol Plant Mol Biol</em> 1998; 49:249‒79. doi: 10.1146/annurev.arplant.49.1.249.</li>
<li>Conklin PL, Williams EH, Last RL. Environmental stress sensitivity of an ascorbic acid-deficient <em>Arabidopsis</em> mutant. <em>Proc Natl Acad Sci USA</em> 1996; 93(18):9970‒4.</li>
<li>Barth C, Moeder W, Klessig DF, Conklin PL. The timing of senescence and response to pathogens is altered in the ascorbate-deficient <em>Arabidopsis</em> mutant vitamin c-1. <em>Plant Physiol</em> 2004; 134(4):1784‒92. doi: 10.1104/pp.103.032185.</li>
<li>Gallie DR. Increasing vitamin C content in plant foods to improve their nutritional value-successes and challenges. <em>Nutrients</em> 2013; 5(9):3424‒46. doi: 10.3390/nu5093424.</li>
<li>Wheeler GL, Jones MA, Smirnoff N. The biosynthetic pathway of vitamin C in higher plants. <em>Nature</em> 1998; 393(6683):365‒9. doi: 10.1038/30728.</li>
<li>Agius F, Gonzalez-Lamothe R, Caballero JL, Munoz-Blanco J, Botella MA, Valpuesta V. Engineering increased vitamin C levels in plants by overexpression of a D-galacturonic acid reductase. <em>Nat Biotechnol</em> 2003; 21(2):177‒81. doi: 10.1038/nbt777.</li>
<li>Wolucka BA, Van Montagu M. GDP-mannose 3&#8242;,5&#8242;-epimerase forms GDP-L-gulose, a putative intermediate for the de novo biosynthesis of vitamin C in plants. <em>J Biol Chem</em> 2003; 278(48):47483‒90. doi: 10.1074/jbc.M309135200.</li>
<li>Lorence A, Chevone BI, Mendes P, Nessler CL. myo-inositol oxygenase offers a possible entry point into plant ascorbate biosynthesis. <em>Plant Physiol</em> 2004; 134(3):1200‒5. doi: 10.1104/pp.103.033936.</li>
<li>Zhang W, Gruszewski HA, Chevone BI, Nessler CL. An Arabidopsis purple acid phosphatase with phytase activity increases foliar ascorbate. <em>Plant Physiol</em> 2008; 146(2):431‒40. doi: 10.1104/pp.107.109934.</li>
<li>Kiuchi K, Nishikimi M, Yagi K. Purification and characterization of L-gulonolactone oxidase from chicken kidney microsomes. <em>Biochemistry</em> 1982; 21(20):5076‒82.</li>
<li>Huh WK, Kim ST, Yang KS, Seok YJ, Hah YC, Kang SO. Characterisation of D-arabinono-1,4-lactone oxidase from <em>Candida albicans</em> ATCC 10231. <em>Eur J Biochem</em> 1994; 225(3):1073‒9.</li>
<li>Wolucka BA, Communi D. <em>Mycobacterium tuberculosis</em> possesses a functional enzyme for the synthesis of vitamin C, L-gulono-1,4-lactone dehydrogenase. <em>FEBS J</em> 2006; 273(19):4435‒45. doi: 10.1111/j.1742-4658.2006.05443.x.</li>
<li>Logan FJ, Taylor MC, Wilkinson SR, Kaur H, Kelly JM. The terminal step in vitamin C biosynthesis in <em>Trypanosoma cruzi</em> is mediated by a FMN-dependent galactonolactone oxidase. <em>Biochem J</em> 2007; 407(3):419‒26. doi: 10.1042/BJ20070766.</li>
<li>Mapson LW, Breslow E. Biological synthesis of ascorbic acid: L-galactono-gamma-lactone dehydrogenase. <em>Biochem J</em> 1958; 68(3):395‒406.</li>
<li>Mutsuda M, Ishikawa T, Takeda T, Shigeoka S. Subcellular localization and properties of L-galactono-γ-lactone dehydrogenase in spinach leaves. <em>Biosci Biotechnol Biochem</em> 1995; 59:1983‒4.</li>
<li>Oba K, Ishikawa S, Nishikawa M, Mizuno H, Yamamoto T. Purification and properties of L-galactono-gamma-lactone dehydrogenase, a key enzyme for ascorbic acid biosynthesis, from sweet potato roots. <em>J Biochem</em> 1995; 117(1):120‒4.</li>
<li>Ostergaard J, Persiau G, Davey MW, Bauw G, Van Montagu M. Isolation of a cDNA coding for L-galactono-gamma-lactone dehydrogenase, an enzyme involved in the biosynthesis of ascorbic acid in plants: purification, characterization, cDNA cloning, and expression in yeast. <em>J Biol Chem</em> 1997; 272(48):30009‒16.</li>
<li>Leferink NG, van den Berg WA, van Berkel WJ. l-Galactono-gamma-lactone dehydrogenase from <em>Arabidopsis thaliana</em>, a flavoprotein involved in vitamin C biosynthesis. <em>FEBS J</em> 2008; 275(4):713‒26. doi: 10.1111/j.1742-4658.2007.06233.x.</li>
<li>Jain AK, Nessler CL. Metabolic engineering of an alternative pathway for ascorbic acid biosynthesis in plants. <em>Mol Breed</em> 2000; 6:73‒8.</li>
<li>Hemavathi, Upadhyaya CP, Akula N, Young KE, Chun SC, Kim DH, et al. Enhanced ascorbic acid accumulation in transgenic potato confers tolerance to various abiotic stresses. <em>Biotechnol Lett</em> 2010; 32(2):321‒30. doi: 10.1007/s10529-009-0140-0.</li>
<li>Lim MY, Pulla RK, Park JM, Harn CH, Jeong BR. Over-expression of L-gulono-γ-lactone oxidase (GLOase) gene leads to ascorbate accumulation with enhanced abiotic stress tolerance in tomato. <em>In Vitro Cell Dev Biol Anim</em> 2012; 48:453‒61.</li>
<li>Lisko KA, Torres R, Harris RS, Belisle M, Vaughan MM, Jullian B, et al. Elevating vitamin C content via overexpression of <em>myo</em>-inositol oxygenase and L-gulono-1,4-lactone oxidase in <em>Arabidopsis</em> leads to enhanced biomass and tolerance to abiotic stresses. <em>In Vitro Cell Dev Biol Plant</em> 2013; 49(6):643‒55. doi: 10.1007/s11627-013-9568-y.</li>
<li>Imai T, Niwa M, Ban Y, Hirai M, Ôba K, Moriguchi T. Importance of the L-galactonolactone pool for enhancing the ascorbate content revealed by L-galactonolactone dehydrogenase-overexpressing tobacco plants. <em>Plant Cell Tiss Organ Cult</em> 2009; 96:105‒12.</li>
<li>Isherwood FA, Chen YT, Mapson LW. Synthesis of L-ascorbic acid in plants and animals. <em>Biochem J</em> 1954; 56(1):1‒15.</li>
<li>Mapson LW, Isherwood FA, Chen YT. Biological synthesis of L-ascorbic acid: the conversion of L-galactono-gamma-lactone into L-ascorbic acid by plant mitochondria. <em>Biochem J</em> 1954; 56(1):21‒8.</li>
<li>Baig MM, Kelly S, Loewus F. L-ascorbic acid biosynthesis in higher plants from L-gulono-1, 4-lactone and L-galactono-1, 4-lactone. <em>Plant Physiol</em> 1970; 46(2):277‒80.</li>
<li>Davey MW, Gilot C, Persiau G, Ostergaard J, Han Y, Bauw GC, et al. Ascorbate biosynthesis in <em>Arabidopsis</em> cell suspension culture. <em>Plant Physiol</em> 1999; 121(2):535‒43.</li>
<li>Pallanca JE, Smirnoff N. Ascorbic acid metabolism in pea seedlings. A comparison of D-glucosone, L-sorbosone, and L-galactono-1,4-lactone as ascorbate precursors. <em>Plant Physiol</em> 1999; 120(2):453‒62.</li>
<li>Siendones E, Gonzalez-Reyes JA, Santos-Ocana C, Navas P, F Cr. Biosynthesis of ascorbic acid in kidney bean. L-galactono-gamma-lactone dehydrogenase is an intrinsic protein located at the mitochondrial inner membrane. <em>Plant Physiol</em> 1999; 120(3):907‒12.</li>
<li>Imai T, Karita S, Shiratori G, Hattori M, Nunome T, Oba K, et al. L-galactono-gamma-lactone dehydrogenase from sweet potato: purification and cDNA sequence analysis. <em>Plant Cell Physiol</em> 1998; 39(12):1350‒8.</li>
<li>Ôba K, Fukui M, Imai Y, Iriyama S, Nogami K. L-Galactono-γ-lactone dehydrogenase: partial characterization, induction of activity and role in the synthesis of ascorbic acid in wounded white potato tuber tissue. <em>Plant Cell Physiol</em> 1994; 35:473‒8.</li>
<li>Maruta T, Ichikawa Y, Mieda T, Takeda T, Tamoi M, Yabuta Y, et al. The contribution of <em>Arabidopsis</em> homologs of L-gulono-1,4-lactone oxidase to the biosynthesis of ascorbic acid. <em>Biosci Biotechnol Biochem</em> 2010; 74(7):1494‒7. doi: 10.1271/bbb.100157.</li>
<li>Murashige T, Skoog FA. A revised medium for rapid growth and bioassays with tobacco culture. <em>Physiol Plant</em> 1962; 15:473‒97.</li>
<li>Boyes DC, Zayed AM, Ascenzi R, McCaskill AJ, Hoffman NE, Davis KR, et al. Growth stage-based phenotypic analysis of <em>Arabidopsis</em>: a model for high throughput functional genomics in plants. <em>Plant Cell</em> 2001; 13(7):1499‒510.</li>
<li>Medrano G, Reidy MJ, Liu J, Ayala J, Dolan MC, Cramer CL. Rapid system for evaluating bioproduction capacity of complex pharmaceutical proteins in plants. <em>Methods Mol Biol</em> 2009; 483:51‒67. doi: 10.1007/978-1-59745-407-0_4.</li>
<li>Becker D. Binary vectors which allow the exchange of plant selectable markers and reporter genes. <em>Nucleic Acids Res</em> 1990; 18(1):203.</li>
<li>Bradford MM. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. <em>Anal Biochem</em> 1976; 72:248‒54.</li>
<li>Dabrowski K. Gulonolactone oxidase is missing in teleost fish: the direct spectrophotometric assay. <em>Biol Chem Hoppe Seyler</em> 1990; 371(3):207‒14.</li>
<li>Clough SJ, Bent AF. Floral dip: a simplified method for <em>Agrobacterium</em>-mediated transformation of <em>Arabidopsis thaliana</em>. <em>Plant J</em> 1998; 16(6):735‒43.</li>
<li>Hruz T, Laule O, Szabo G, Wessendorp F, Bleuler S, Oertle L, et al. Genevestigator v3: a reference expression database for the meta-analysis of transcriptomes. <em>Adv Bioinformatics</em> 2008; 2008:420747. doi: 10.1155/2008/420747.</li>
<li>Fraaije MW, van den Heuvel RH, van Berkel WJ, Mattevi A. Covalent flavinylation is essential for efficient redox catalysis in vanillyl-alcohol oxidase. <em>J Biol Chem</em> 1999; 274(50):35514‒20.</li>
<li>Boudart G, Jamet E, Rossignol M, Lafitte C, Borderies G, Jauneau A, et al. Cell wall proteins in apoplastic fluids of Arabidopsis thaliana rosettes: identification by mass spectrometry and bioinformatics. <em>Proteomics</em> 2005; 5(1):212‒21. doi: 10.1002/pmic.200400882.</li>
<li>Laing WA, Wright MA, Cooney J, Bulley SM. The missing step of the L-galactose pathway of ascorbate biosynthesis in plants, an L-galactose guanyltransferase, increases leaf ascorbate content. <em>Proc Natl Acad Sci USA</em> 2007; 104(22):9534‒9. doi: 10.1073/pnas.0701625104.</li>
<li>Bulley SM, Rassam M, Hoser D, Otto W, Schunemann N, Wright M, et al. Gene expression studies in kiwifruit and gene over-expression in Arabidopsis indicates that GDP-L-galactose guanyltransferase is a major control point of vitamin C biosynthesis. <em>J Exp Bot</em> 2009; 60(3):765‒78. doi: 10.1093/jxb/ern327.</li>
<li>Zhang W, Lorence A, Gruszewski HA, Chevone BI, Nessler CL. AMR1, an Arabidopsis gene that coordinately and negatively regulates the mannose/L-galactose ascorbic acid biosynthetic pathway. <em>Plant Physiol</em> 2009; 150(2):942‒50. doi: 10.1104/pp.109.138453.</li>
<li>Lisko KA, Hubstenberger JF, Phillips GC, Belefant-Miller H, McClung A, Lorence A. Ontogenetic changes in vitamin C in selected rice varieties. <em>Plant Physiol Biochem</em> 2013; 66:41‒6. doi: 10.1016/j.plaphy.2013.01.016.</li>
<li>Sugisawa T, Setsuko O, Matzinger PK, Hoshino T. Isolation and characterization of a new vitamin C producing enzyme (L-gulono-γ-lactone dehydrogenase) of bacterial origin. <em>Biosci Biotech Biochem</em> 1995; 59:190‒6.</li>
<li>Mawuenyega KG, Forst CV, Dobos KM, Belisle JT, Chen J, Bradbury EM, et al. <em>Mycobacterium tuberculosis</em> functional network analysis by global subcellular protein profiling. <em>Mol Biol Cell</em> 2005; 16(1):396‒404. doi: 10.1091/mbc.E04-04-0329.</li>
<li>Okamura M. Purification and properties of L-gulono-1,4-lactone oxidase from <em>Grifola frondosa</em>. <em>J Nutr Sci Vitaminol (Tokyo)</em> 2001; 47(3):258‒62.</li>
<li>Wheeler G, Ishikawa T, Pornsaksit V, Smirnoff N. Evolution of alternative biosynthetic pathways for vitamin C following plastid acquisition in photosynthetic eukaryotes. <em>Elife</em> 2015; 4. doi: 10.7554/eLife.06369.</li>
<li>Li MJ, Ma FW, Zhang M, Pu F. Distribution and metabolism of ascorbic acid in apple fruits (<em>Malus domestica</em> Borkh cv. Gala). <em>Plant Sci</em> 2008; 174:606‒12.</li>
<li>Davey MW, Franck C, Keulemans J. Distribution, developmental and stress responses of antioxidant metabolism in Malus. <em>Plant Cell Environ</em> 2004; 27:1309‒20.</li>
<li>Hancock RD, Walker PG, Pont SDA, Marquis N, Vivera S, Gordon SL, et al. L-Ascorbic acid accumulation in fruit of <em>Ribes nigrum</em> occurs by in situ biosynthesis via the L-galactose pathway. <em>Funct Plant Biol</em> 2007; 34:1080‒109.</li>
<li>Li M, Ma F, Liang D, Li J, Wang Y. Ascorbate biosynthesis during early fruit development is the main reason for its accumulation in kiwi. <em>PLoS One</em> 2010; 5(12):e14281. doi: 10.1371/journal.pone.0014281.</li>
<li>Imai T, Ban Y, Terakami S, Yamamoto T, Moriguchi T. L-Ascorbate biosynthesis in peach: cloning of six L-galactose pathway-related genes and their expression during peach fruit development. <em>Physiol Plant</em> 2009; 136(2):139‒49. doi: 10.1111/j.1399-3054.2009.01213.x.</li>
<li>Mellidou I, Keulemans J, Kanellis AK, Davey MW. Regulation of fruit ascorbic acid concentrations during ripening in high and low vitamin C tomato cultivars. <em>BMC Plant Biol</em> 2012; 12:239. doi: 10.1186/1471-2229-12-239.</li>
<li>Bleeg HS, Christensen F. Biosynthesis of ascorbate in yeast. Purification of L-galactono-1,4-lactone oxidase with properties different from mammalian L-gulonolactone oxidase. <em>Eur J Biochem</em> 1982; 127(2):391‒6.</li>
<li>Wilkinson SR, Prathalingam SR, Taylor MC, Horn D, Kelly JM. Vitamin C biosynthesis in trypanosomes: a role for the glycosome. <em>Proc Natl Acad Sci USA</em> 2005; 102(33):11645-50. doi: 10.1073/pnas.0504251102.</li>
<li>Salusjärvi. Characterisation, cloning and production of industrially interesting enzymes: gluconolactone oxidase of <em>Penicillium cyaneofulvum</em> and gluconate 5-dehydrogenase of <em>Gluconobacter suboxydans</em>. <em>PhD Thesis.</em> University of Helsinki, Finland. 2006.</li>
<li>Aboobucker SI, Lorence A. Recent progress on the characterization of aldonolactone oxidoreductases. <em>Plant Physiol Biochem</em> 2016; 98:171‒85. doi: 10.1016/j.plaphy.2015.11.017.</li>
<li>Yagi K, Koshizaka T, Kito M, Ozawa T, Nishikimi M. Expression in monkey cells of the missing enzyme in L-ascorbic acid biosynthesis, L-gulono-gamma-lactone oxidase. <em>Biochem Biophys Res Commun</em> 1991; 177(2):659‒63.</li>
<li>Nishikimi M, Kobayashi J, Yagi K. Production by a baculovirus expression system of the APO-protein of L-gulono-gamma-lactone oxidase, a flavoenzyme possessing a covalently-bound FAD. <em>Biochem Mol Biol Int</em> 1994; 33(2):313‒9.</li>
<li>Ha MN, Graham FL, D&#8217;Souza CK, Muller WJ, Igdoura SA, Schellhorn HE. Functional rescue of vitamin C synthesis deficiency in human cells using adenoviral-based expression of murine l-gulono-gamma-lactone oxidase. <em>Genomics</em> 2004; 83(3):482‒92. doi: 10.1016/j.ygeno.2003.08.018.</li>
<li>Linster CL, Adler LN, Webb K, Christensen KC, Brenner C, Clarke SG. A second GDP-L-galactose phosphorylase in arabidopsis en route to vitamin C: covalent intermediate and substrate requirements for the conserved reaction. <em>J Biol Chem</em> 2008; 283(27):18483‒92. doi: 10.1074/jbc.M802594200.</li>
<li>Tokunaga T, Miyahara K, Tabata K, Esaka M. Generation and properties of ascorbic acid-overproducing transgenic tobacco cells expressing sense RNA for L-galactono-1,4-lactone dehydrogenase. <em>Planta</em> 2005; 220(6):854‒63. doi: 10.1007/s00425-004-1406-3.</li>
<li>Loewus FA. Tracer studies of ascorbic acid formation in plants. <em>Phytochem</em> 1963; 2:109‒28.</li>
<li>Alhagdow M, Mounet F, Gilbert L, Nunes-Nesi A, Garcia V, Just D, et al. Silencing of the mitochondrial ascorbate synthesizing enzyme L-galactono-1,4-lactone dehydrogenase affects plant and fruit development in tomato. <em>Plant Physiol</em> 2007; 145(4):1408‒22. doi: 10.1104/pp.107.106500.</li>
<li>Yabuta Y, Maruta T, Nakamura A, Mieda T, Yoshimura K, Ishikawa T, et al. Conversion of L-galactono-1,4-lactone to L-ascorbate is regulated by the photosynthetic electron transport chain in <em>Arabidopsis</em>. <em>Biosci Biotechnol Biochem</em> 2008; 72(10):2598‒607. doi: 10.1271/bbb.80284.</li>
<li>Radzio JA, Lorence A, Chevone BI, Nessler CL. L-Gulono-1,4-lactone oxidase expression rescues vitamin C-deficient <em>Arabidopsis</em> (vtc) mutants. <em>Plant Mol Biol</em> 2003; 53(6):837‒44. doi: 10.1023/B:PLAN.0000023671.99451.1d.</li>
<li>Gallie DR. L-Ascorbic acid: a multifunctional molecule supporting plant growth and development. <em>Scientifica (Cairo)</em> 2013; 2013:795964. doi: 10.1155/2013/795964.</li>
<li>Yabuta Y, Yoshimura K, Takeda T, Shigeoka S. Molecular characterization of tobacco mitochondrial L-galactono-gamma-lactone dehydrogenase and its expression in <em>Escherichia coli</em>. <em>Plant Cell Physiol</em> 2000; 41(6):666‒75.</li>
<li>Biyani N, Madhubala R. <em>Leishmania donovani</em> encodes a functional enzyme involved in vitamin C biosynthesis: arabino-1,4-lactone oxidase. <em>Mol Biochem Parasitol</em> 2011; 180(2):76‒85. doi: 10.1016/j.molbiopara.2011.08.005.</li>
<li>Nishikimi M, Tolbert BM, Udenfriend S. Purification and characterization of L-gulono-gamma-lactone oxidase from rat and goat liver. <em>Arch Biochem Biophys</em> 1976; 175(2):427‒35.</li>
<li>Shigeoka S, Yokota A, Nakano Y, Kitaoka S. The effect of illumination on the L-ascorbic acid content in <em>Euglena gracilis</em> Z. <em>Agric Biol Chem</em> 1979; 43:2053‒8.</li>
<li>Nishikimi M, Noguchi E, Yagi K. Occurrence in yeast of L-galactonolactone oxidase which is similar to a key enzyme for ascorbic acid biosynthesis in animals, L-gulonolactone oxidase. <em>Arch Biochem Biophys</em> 1978; 191(2):479‒86.</li>
<li>Leferink NG, Fraaije MW, Joosten HJ, Schaap PJ, Mattevi A, van Berkel WJ. Identification of a gatekeeper residue that prevents dehydrogenases from acting as oxidases. <em>J Biol Chem</em> 2009; 284(7):4392‒7. doi: 10.1074/jbc.M808202200.</li>
<li>Franceschi VR, Tarlyn NM. L-Ascorbic acid is accumulated in source leaf phloem and transported to sink tissues in plants. <em>Plant Physiol</em> 2002; 130(2):649‒56. doi: 10.1104/pp.007062.</li>
<li>Ioannidi E, Kalamaki MS, Engineer C, Pateraki I, Alexandrou D, Mellidou I, et al. Expression profiling of ascorbic acid-related genes during tomato fruit development and ripening and in response to stress conditions. <em>J Exp Bot</em> 2009; 60(2):663‒78. doi: 10.1093/jxb/ern322.</li>
<li>Siddique S, Endres S, Atkins JM, Szakasits D, Wieczorek K, Hofmann J, et al. Myo-inositol oxygenase genes are involved in the development of syncytia induced by <em>Heterodera schachtii</em> in <em>Arabidopsis</em> roots. <em>New Phytol</em> 2009; 184(2):457‒72. doi: 10.1111/j.1469‒8137.2009.02981.x.</li>
<li>Alford SR, Rangarajan P, Williams P, Gillaspy GE. myo-Inositol oxygenase is required for responses to low energy conditions in <em>Arabidopsis thaliana</em>. <em>Front Plant Sci</em> 2012; 3:69. doi: 10.3389/fpls.2012.00069.</li>
<li>Holler S, Hajirezaei MR, von Wiren N, Frei M. Ascorbate metabolism in rice genotypes differing in zinc efficiency. <em>Planta</em> 2014; 239(2):367‒79. doi: 10.1007/s00425-013-1978-x.</li>
<li>Cruz-Rus E, Amaya I, Sanchez-Sevilla JF, Botella MA, Valpuesta V. Regulation of L-ascorbic acid content in strawberry fruits. <em>J Exp Bot</em> 2011; 62(12):4191‒201. doi: 10.1093/jxb/err122.</li>
<li>Alos E, Rodrigo MJ, Zacarias L. Differential transcriptional regulation of L-ascorbic acid content in peel and pulp of citrus fruits during development and maturation. <em>Planta</em> 2014; 239(5):1113‒28. doi: 10.1007/s00425-014-2044-z.</li>
<li>Wolucka BA, Goossens A, Inze D. Methyl jasmonate stimulates the de novo biosynthesis of vitamin C in plant cell suspensions. <em>J Exp Bot</em> 2005; 56(419):2527‒38. doi: 10.1093/jxb/eri246.</li>
</ol>
					</div> <!-- .entry-content -->

				
				</article> <!-- .et_pb_post -->

			

			</div> <!-- #left-area -->

					</div> <!-- #content-area -->
	</div> <!-- .container -->


</div> <!-- #main-content -->


	<span class="et_pb_scroll_top et-pb-icon"></span>


			<footer id="main-footer">
				

		
				<div id="footer-bottom">
					<div class="container clearfix">
				<div id="footer-info">Reactive Oxygen Species ◊ ©2016‒2019 Cell Med Press, AIMSCI Inc. All rights reserved.</div>					</div>	<!-- .container -->
				</div>
			</footer> <!-- #main-footer -->
		</div> <!-- #et-main-area -->


	</div> <!-- #page-container -->

		<script type="text/javascript">
			</script>
	<script type='text/javascript'>
/* <![CDATA[ */
var DIVI = {"item_count":"%d Item","items_count":"%d Items"};
var et_shortcodes_strings = {"previous":"Previous","next":"Next"};
var et_pb_custom = {"ajaxurl":"http:\/\/www.aimsci.com\/ros\/html\/wp-admin\/admin-ajax.php","images_uri":"http://www.aimsci.com/ros/html/wp-content/themes/Divi/images/","builder_images_uri":"http://www.aimsci.com/ros/html/wp-content/themes/Divi/includes/builder/images/","et_frontend_nonce":"8606b18c1a","subscription_failed":"Please, check the fields below to make sure you entered the correct information.","et_ab_log_nonce":"d44b95b2c4","fill_message":"Please, fill in the following fields:","contact_error_message":"Please, fix the following errors:","invalid":"Invalid email","captcha":"Captcha","prev":"Prev","previous":"Previous","next":"Next","wrong_captcha":"You entered the wrong number in captcha.","ignore_waypoints":"no","is_divi_theme_used":"1","widget_search_selector":".widget_search","is_ab_testing_active":"","page_id":"1560","unique_test_id":"","ab_bounce_rate":"5","is_cache_plugin_active":"no","is_shortcode_tracking":"","tinymce_uri":""};
var et_pb_box_shadow_elements = [];
/* ]]> */
</script>
<script type='text/javascript' src='wp-content/themes/Divi/js/custom.min_1e2ae7de.js'></script>
<script type='text/javascript' src='wp-content/themes/Divi/core/admin/js/common_1e2ae7de.js'></script>
<script type='text/javascript' src='wp-includes/js/wp-embed.min_e0220a6d.js'></script>
</body>
</html>
