<!DOCTYPE html>
<html lang="en-US">
<head>
<!-- Original URL: http://www.aimsci.com/ros/html/?page_id=748
Date Downloaded: 12/25/2019 3:39:58 PM !-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	<meta charset="UTF-8" />
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
	<link rel="pingback" href="http://www.aimsci.com/ros/html/xmlrpc.php" />

	<script type="text/javascript">
		document.documentElement.className = 'js';
	</script>

	<script>var et_site_url='index.html';var et_post_id='748';function et_core_page_resource_fallback(a,b){"undefined"===typeof b&&(b=a.sheet.cssRules&&0===a.sheet.cssRules.length);b&&(a.onerror=null,a.onload=null,a.href?a.href=et_site_url+"/?et_core_page_resource="+a.id+et_post_id:a.src&&(a.src=et_site_url+"/?et_core_page_resource="+a.id+et_post_id))}
</script><title>2016; 2(5):325–338 | ss_site_title</title>
<link rel='dns-prefetch' href='//fonts.googleapis.com' />
<link rel='dns-prefetch' href='//s.w.org' />
<link rel="alternate" type="application/rss+xml" title="ss_site_title &raquo; Feed" href="_c0b1fbb3" />
<link rel="alternate" type="application/rss+xml" title="ss_site_title &raquo; Comments Feed" href="_387b1518" />
		<script type="text/javascript">
			window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/11\/72x72\/","ext":".png","svgUrl":"https:\/\/s.w.org\/images\/core\/emoji\/11\/svg\/","svgExt":".svg","source":{"concatemoji":"wp-includes/js/wp-emoji-release.min_e0220a6d.js"}};
			!function(a,b,c){function d(a,b){var c=String.fromCharCode;l.clearRect(0,0,k.width,k.height),l.fillText(c.apply(this,a),0,0);var d=k.toDataURL();l.clearRect(0,0,k.width,k.height),l.fillText(c.apply(this,b),0,0);var e=k.toDataURL();return d===e}function e(a){var b;if(!l||!l.fillText)return!1;switch(l.textBaseline="top",l.font="600 32px Arial",a){case"flag":return!(b=d([55356,56826,55356,56819],[55356,56826,8203,55356,56819]))&&(b=d([55356,57332,56128,56423,56128,56418,56128,56421,56128,56430,56128,56423,56128,56447],[55356,57332,8203,56128,56423,8203,56128,56418,8203,56128,56421,8203,56128,56430,8203,56128,56423,8203,56128,56447]),!b);case"emoji":return b=d([55358,56760,9792,65039],[55358,56760,8203,9792,65039]),!b}return!1}function f(a){var c=b.createElement("script");c.src=a,c.defer=c.type="text/javascript",b.getElementsByTagName("head")[0].appendChild(c)}var g,h,i,j,k=b.createElement("canvas"),l=k.getContext&&k.getContext("2d");for(j=Array("flag","emoji"),c.supports={everything:!0,everythingExceptFlag:!0},i=0;i<j.length;i++)c.supports[j[i]]=e(j[i]),c.supports.everything=c.supports.everything&&c.supports[j[i]],"flag"!==j[i]&&(c.supports.everythingExceptFlag=c.supports.everythingExceptFlag&&c.supports[j[i]]);c.supports.everythingExceptFlag=c.supports.everythingExceptFlag&&!c.supports.flag,c.DOMReady=!1,c.readyCallback=function(){c.DOMReady=!0},c.supports.everything||(h=function(){c.readyCallback()},b.addEventListener?(b.addEventListener("DOMContentLoaded",h,!1),a.addEventListener("load",h,!1)):(a.attachEvent("onload",h),b.attachEvent("onreadystatechange",function(){"complete"===b.readyState&&c.readyCallback()})),g=c.source||{},g.concatemoji?f(g.concatemoji):g.wpemoji&&g.twemoji&&(f(g.twemoji),f(g.wpemoji)))}(window,document,window._wpemojiSettings);
		</script>
		<meta content="Divi v.3.19.7" name="generator" /><style type="text/css">
img.wp-smiley,
img.emoji {
	display: inline !important;
	border: none !important;
	box-shadow: none !important;
	height: 1em !important;
	width: 1em !important;
	margin: 0 .07em !important;
	vertical-align: -0.1em !important;
	background: none !important;
	padding: 0 !important;
}
</style>
<link rel='stylesheet' id='wp-block-library-css' href='wp-includes/css/dist/block-library/style.min_e0220a6d.css' type='text/css' media='all' />
<link rel='stylesheet' id='divi-fonts-css' href='../../../fonts.googleapis.com/css_6b0aa6b1.css' type='text/css' media='all' />
<link rel='stylesheet' id='divi-style-css' href='wp-content/themes/Divi/style_1e2ae7de.css' type='text/css' media='all' />
<link rel='stylesheet' id='dashicons-css' href='wp-includes/css/dashicons.min_e0220a6d.css' type='text/css' media='all' />
<script type='text/javascript' src='wp-includes/js/jquery/jquery_85ec8bd8.js'></script>
<script type='text/javascript' src='wp-includes/js/jquery/jquery-migrate.min_e021f9e6.js'></script>
<link rel='https://api.w.org/' href='index_ba315afa.php' />
<link rel="EditURI" type="application/rsd+xml" title="RSD" href="xmlrpc_b9b66041.php" />
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="wp-includes/wlwmanifest.xml" /> 
<meta name="generator" content="WordPress 5.0.8" />
<link rel="canonical" href="_73d2238b.html" />
<link rel='shortlink' href='_73d2238b.html' />
<link rel="alternate" type="application/json+oembed" href="index_66ce9c50.php" />
<link rel="alternate" type="text/xml+oembed" href="index_e48673fd.php" />
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" /><link rel="stylesheet" id="et-divi-customizer-global-cached-inline-styles" href="wp-content/cache/et/global/et-divi-customizer-global-15766348400072.min.css" onerror="et_core_page_resource_fallback(this, true)" onload="et_core_page_resource_fallback(this)" /></head>
<body class="page-template-default page page-id-748 et_pb_button_helper_class et_fixed_nav et_show_nav et_cover_background et_pb_gutter windows et_pb_gutters3 et_primary_nav_dropdown_animation_fade et_secondary_nav_dropdown_animation_fade et_pb_footer_columns4 et_header_style_left et_right_sidebar et_divi_theme et-db et_minified_js et_minified_css">
	<div id="page-container">

	
	
			<header id="main-header" data-height-onload="66">
			<div class="container clearfix et_menu_container">
							<div class="logo_container">
					<span class="logo_helper"></span>
					<a href="index.html">
						<img src="wp-content/uploads/2016/03/ROS-blue-square.png" alt="ss_site_title" id="logo" data-height-percentage="54" />
					</a>
				</div>
							<div id="et-top-navigation" data-height="66" data-fixed-height="40">
											<nav id="top-menu-nav">
						<ul id="top-menu" class="nav"><li id="menu-item-1503" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-ancestor current_page_ancestor menu-item-has-children menu-item-1503"><a href="_ad6635bd.html">2016</a>
<ul class="sub-menu">
	<li id="menu-item-168" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-168"><a href="_2e83399d.html">2016 Vol 1 No 1</a>
	<ul class="sub-menu">
		<li id="menu-item-155" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-155"><a href="_468cdbca.html">2016; 1(1):1–8</a></li>
		<li id="menu-item-154" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-154"><a href="_468cdbcc.html">2016; 1(1):9–21</a></li>
		<li id="menu-item-153" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-153"><a href="_468cdbbe.html">2016; 1(1):22–37</a></li>
		<li id="menu-item-152" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-152"><a href="_73d222cd.html">2016; 1(1):38–52</a></li>
		<li id="menu-item-151" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-151"><a href="_73d222cf.html">2016; 1(1):53–58</a></li>
		<li id="menu-item-150" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-150"><a href="_73d222d1.html">2016; 1(1):59–64</a></li>
		<li id="menu-item-149" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-149"><a href="_73d222d3.html">2016; 1(1):65–80</a></li>
		<li id="menu-item-165" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-165"><a href="_2e833998.html">2016; 1(1):81–98</a></li>
	</ul>
</li>
	<li id="menu-item-208" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-208"><a href="_d12aae32.html">2016 Vol 1 No 2</a>
	<ul class="sub-menu">
		<li id="menu-item-207" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-207"><a href="_d12aae34.html">2016; 1(2):99–109</a></li>
		<li id="menu-item-206" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-206"><a href="_d12aae36.html">2016; 1(2):110–116</a></li>
		<li id="menu-item-205" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-205"><a href="_d12aae38.html">2016; 1(2):117–130</a></li>
		<li id="menu-item-204" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-204"><a href="_d12aae2a.html">2016; 1(2):131–140</a></li>
		<li id="menu-item-203" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-203"><a href="_d3f8ab89.html">2016; 1(2):141–156</a></li>
		<li id="menu-item-202" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-202"><a href="_d3f8ab8b.html">2016; 1(2):157–164</a></li>
		<li id="menu-item-201" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-201"><a href="_d3f8ab8d.html">2016; 1(2):165–177</a></li>
	</ul>
</li>
	<li id="menu-item-289" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-289"><a href="_d3f8aba4.html">2016 Vol 1 No 3</a>
	<ul class="sub-menu">
		<li id="menu-item-273" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-273"><a href="_d12aae53.html">2016; 1(3):178–188</a></li>
		<li id="menu-item-277" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-277"><a href="_d12aae57.html">2016; 1(3):189–198</a></li>
		<li id="menu-item-280" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-280"><a href="_d12aae4c.html">2016; 1(3):199–206</a></li>
		<li id="menu-item-517" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-517"><a href="_8bdbc57e.html">2016; 1(3):207–215</a></li>
		<li id="menu-item-286" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-286"><a href="_d3f8abaf.html">2016; 1(3):216–227</a></li>
		<li id="menu-item-292" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-292"><a href="_76a02046.html">2016; 1(3):228–244</a></li>
	</ul>
</li>
	<li id="menu-item-587" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-587"><a href="_d3f8ac0b.html">2016 Vol 2 No 4</a>
	<ul class="sub-menu">
		<li id="menu-item-607" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-607"><a href="_e9345108.html">2016; 2(4):245–263</a></li>
		<li id="menu-item-576" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-576"><a href="_d12aaeb3.html">2016; 2(4):264–271</a></li>
		<li id="menu-item-677" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-677"><a href="_2e833a37.html">2016; 2(4):272–289</a></li>
		<li id="menu-item-651" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-651"><a href="_73d2236e.html">2016; 2(4):290–297</a></li>
		<li id="menu-item-641" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-641"><a href="_468cdc66.html">2016; 2(4):298–307</a></li>
		<li id="menu-item-692" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-692"><a href="_d3f8ac2e.html">2016; 2(4):308–314</a></li>
	</ul>
</li>
	<li id="menu-item-735" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-ancestor current-menu-parent current_page_parent current_page_ancestor menu-item-has-children menu-item-735"><a href="_468cdc8b.html">2016 Vol 2 No 5</a>
	<ul class="sub-menu">
		<li id="menu-item-741" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-741"><a href="_468cdc85.html">2016; 2(5):315–324</a></li>
		<li id="menu-item-751" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-748 current_page_item menu-item-751"><a href="_73d2238b.html">2016; 2(5):325–338</a></li>
		<li id="menu-item-766" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-766"><a href="_2e833a59.html">2016; 2(5):339–354</a></li>
		<li id="menu-item-781" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-781"><a href="_d12aaef1.html">2016; 2(5):355–360</a></li>
		<li id="menu-item-807" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-807"><a href="_e9344f3d.html">2016; 2(5):361–370</a></li>
		<li id="menu-item-713" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-713"><a href="_8bdbc5c3.html">2016; 2(5):371–387</a></li>
	</ul>
</li>
	<li id="menu-item-844" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-844"><a href="_73d221a8.html">2016 Vol 2 No 6</a>
	<ul class="sub-menu">
		<li id="menu-item-868" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-868"><a href="_2e833876.html">2016; 2(6):388–395</a></li>
		<li id="menu-item-879" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-879"><a href="_d12aad11.html">2016; 2(6):396–403</a></li>
		<li id="menu-item-914" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-914"><a href="_8bdbc3f6.html">2016; 2(6):404–416</a></li>
		<li id="menu-item-886" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-886"><a href="_d3f8aa66.html">2016; 2(6):417–420</a></li>
		<li id="menu-item-834" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-834"><a href="_468cdaa1.html">2016; 2(6):421–431</a></li>
		<li id="menu-item-897" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-897"><a href="_76a01f00.html">2016; 2(6):432–439</a></li>
	</ul>
</li>
</ul>
</li>
<li id="menu-item-1507" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1507"><a href="_22c86351.html">2017</a>
<ul class="sub-menu">
	<li id="menu-item-940" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-940"><a href="_468cdac6.html">2017 Vol 3 No 7</a>
	<ul class="sub-menu">
		<li id="menu-item-932" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-932"><a href="_a3e5662c.html">2017; 3(7):1–11</a></li>
		<li id="menu-item-1007" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1007"><a href="_5141f038.html">2017; 3(7):12–25</a></li>
		<li id="menu-item-1017" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1017"><a href="_5141f039.html">2017; 3(7):26–37</a></li>
		<li id="menu-item-1030" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1030"><a href="_ec1045b.html">2017; 3(7):38–46</a></li>
		<li id="menu-item-944" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-944"><a href="_73d221c7.html">2017; 3(7):47–65</a></li>
		<li id="menu-item-1039" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1039"><a href="_bf30705.html">2017; 3(7):66–80</a></li>
	</ul>
</li>
	<li id="menu-item-1074" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1074"><a href="_deadc002.html">2017 Vol 3 No 8</a>
	<ul class="sub-menu">
		<li id="menu-item-1069" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1069"><a href="_bf30708.html">2017; 3(8):81–95</a></li>
		<li id="menu-item-1132" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1132"><a href="_71b58a6f.html">2017; 3(8):96–110</a></li>
		<li id="menu-item-1128" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1128"><a href="_b70473a4.html">2017; 3(8):111–120</a></li>
		<li id="menu-item-1112" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1112"><a href="_71b58a6d.html">2017; 3(8):121–126</a></li>
		<li id="menu-item-1098" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1098"><a href="_694b9260.html">2017; 3(8):127–134</a></li>
		<li id="menu-item-1084" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1084"><a href="_deadbff3.html">2017; 3(8):135–147</a></li>
	</ul>
</li>
	<li id="menu-item-1174" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1174"><a href="_2c66a13d.html">2017 Vol 3 No 9</a>
	<ul class="sub-menu">
		<li id="menu-item-1194" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1194"><a href="_2c66a12f.html">2017; 3(9):148–183</a></li>
		<li id="menu-item-1286" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1286"><a href="_1cc6f704.html">2017; 3(9):184–188</a></li>
		<li id="menu-item-1177" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1177"><a href="_cf0e15d8.html">2017; 3(9):189–196</a></li>
		<li id="menu-item-1186" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1186"><a href="_fc535cd0.html">2017; 3(9):197–207</a></li>
		<li id="menu-item-1299" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1299"><a href="_a764c971.html">2017; 3(9):208–217</a></li>
		<li id="menu-item-1312" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1312"><a href="_82897a77.html">2017; 3(9):218–236</a></li>
	</ul>
</li>
	<li id="menu-item-1347" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1347"><a href="_6a7fd84b.html">2017 Vol 4 No 10</a>
	<ul class="sub-menu">
		<li id="menu-item-1352" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1352"><a href="_82897a7b.html">2017; 4(10):237–250</a></li>
		<li id="menu-item-1379" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1379"><a href="_afcec184.html">2017; 4(10):251–265</a></li>
		<li id="menu-item-1400" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1400"><a href="_6d4dd597.html">2017; 4(10):266–274</a></li>
		<li id="menu-item-1360" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1360"><a href="_6d4dd5a3.html">2017; 4(10):275–283</a></li>
		<li id="menu-item-1371" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1371"><a href="_caa66109.html">2017; 4(10):284–289</a></li>
		<li id="menu-item-1388" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1388"><a href="_6a7fd83f.html">2017; 4(10):290–297</a></li>
	</ul>
</li>
	<li id="menu-item-1423" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1423"><a href="_a4fc3dba.html">2017 Vol 4 No 11</a>
	<ul class="sub-menu">
		<li id="menu-item-1419" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1419"><a href="_d50f8217.html">2017; 4(11):298–302</a></li>
		<li id="menu-item-1429" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1429"><a href="_d50f8218.html">2017; 4(11):303–319</a></li>
		<li id="menu-item-1438" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1438"><a href="_32680d7e.html">2017; 4(11):320–335</a></li>
		<li id="menu-item-1458" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1458"><a href="_32680d80.html">2017; 4(11):336–350</a></li>
		<li id="menu-item-1478" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1478"><a href="_32680d82.html">2017; 4(11):351–361</a></li>
		<li id="menu-item-1490" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1490"><a href="_ed19243d.html">2017; 4(11):362–371</a></li>
	</ul>
</li>
	<li id="menu-item-1530" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1530"><a href="_3ad20582.html">2017 Vol 4 No 12</a>
	<ul class="sub-menu">
		<li id="menu-item-1591" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1591"><a href="_982a90dd.html">2017; 4(12):372–381</a></li>
		<li id="menu-item-1551" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1551"><a href="_982a90e9.html">2017; 4(12):382–388</a></li>
		<li id="menu-item-1562" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1562"><a href="_500daa5e.html">2017; 4(12):389–417</a></li>
		<li id="menu-item-1538" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1538"><a href="_8020eeb9.html">2017; 4(12):418–433</a></li>
		<li id="menu-item-1605" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1605"><a href="_fb1f16f8.html">2017; 4(12):434–440</a></li>
		<li id="menu-item-1526" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1526"><a href="_c56fd7ee.html">2017; 4(12):441–444</a></li>
	</ul>
</li>
</ul>
</li>
<li id="menu-item-1679" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1679"><a href="_70814493.html">2018</a>
<ul class="sub-menu">
	<li id="menu-item-1667" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1667"><a href="_888ae6c1.html">2018 Vol 5 No 13</a>
	<ul class="sub-menu">
		<li id="menu-item-1671" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1671"><a href="_a09488f0.html">2018; 5(13):1–14</a></li>
		<li id="menu-item-1757" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1757"><a href="_339c5360.html">2018; 5(13):15–34</a></li>
		<li id="menu-item-1742" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1742"><a href="_a8fe80f2.html">2018; 5(13):35–45</a></li>
		<li id="menu-item-1683" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1683"><a href="_b5d02dba.html">2018; 5(13):46–55</a></li>
		<li id="menu-item-1703" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1703"><a href="_be3a25c7.html">2018; 5(13):56–67</a></li>
		<li id="menu-item-1714" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1714"><a href="_a6308399.html">2018; 5(13):68–77</a></li>
	</ul>
</li>
	<li id="menu-item-1810" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1810"><a href="_7c880e31.html">2018 Vol 5 No 14</a>
	<ul class="sub-menu">
		<li id="menu-item-1814" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1814"><a href="_21fd8024.html">2018; 5(14):78–85</a></li>
		<li id="menu-item-1835" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1835"><a href="_c4a4f4c1.html">2018; 5(14):86–106</a></li>
		<li id="menu-item-1860" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1860"><a href="_7c880e36.html">2018; 5(14):107–117</a></li>
		<li id="menu-item-1802" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1802"><a href="_a8fe80e8.html">2018; 5(14):118–125</a></li>
		<li id="menu-item-1847" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1847"><a href="_9491b064.html">2018; 5(14):126–133</a></li>
		<li id="menu-item-1870" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1870"><a href="_7c880e37.html">2018; 5(14):134–144</a></li>
	</ul>
</li>
	<li id="menu-item-1902" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1902"><a href="_b5054a94.html">2018 Vol 5 No 15</a>
	<ul class="sub-menu">
		<li id="menu-item-1934" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1934"><a href="_6fb66161.html">2018; 5(15):145–158</a></li>
		<li id="menu-item-1956" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1956"><a href="_3fa31d05.html">2018; 5(15):159–166</a></li>
		<li id="menu-item-1898" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1898"><a href="_ac9b5288.html">2018; 5(15):167–175</a></li>
		<li id="menu-item-1911" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1911"><a href="_27997ad1.html">2018; 5(15):176–198</a></li>
		<li id="menu-item-1924" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1924"><a href="_6fb66160.html">2018; 5(15):199–208</a></li>
		<li id="menu-item-1944" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1944"><a href="_6fb66162.html">2018; 5(15):209–219</a></li>
	</ul>
</li>
	<li id="menu-item-1993" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1993"><a href="_125dd5f2.html">2018 Vol 6 No 16</a>
	<ul class="sub-menu">
		<li id="menu-item-2036" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2036"><a href="_ae9a7bbd.html">2018; 6(16):220–247</a></li>
		<li id="menu-item-2007" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2007"><a href="_5141f05b.html">2018; 6(16):248–252</a></li>
		<li id="menu-item-1989" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1989"><a href="_fa5433c2.html">2018; 6(16):253–266</a></li>
		<li id="menu-item-2020" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2020"><a href="_ec1047b.html">2018; 6(16):267–275</a></li>
		<li id="menu-item-2046" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2046"><a href="_ae9a7bc4.html">2018; 6(16):276–288</a></li>
		<li id="menu-item-2056" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2056"><a href="_ae9a7bc3.html">2018; 6(16):289–298</a></li>
	</ul>
</li>
	<li id="menu-item-2086" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2086"><a href="_ae9a7bb8.html">2018 Vol 6 No 17</a>
	<ul class="sub-menu">
		<li id="menu-item-2156" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2156"><a href="_89bf2cc1.html">2018; 6(17):299–310</a></li>
		<li id="menu-item-2082" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2082"><a href="_23fca94c.html">2018; 6(17):311–324</a></li>
		<li id="menu-item-2102" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2102"><a href="_71b58a8f.html">2018; 6(17):325–337</a></li>
		<li id="menu-item-2119" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2119"><a href="_145cff29.html">2018; 6(17):338–348</a></li>
		<li id="menu-item-2134" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2134"><a href="_2c66a156.html">2018; 6(17):349–362</a></li>
		<li id="menu-item-2177" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2177"><a href="_59abe861.html">2018; 6(17):363–370</a></li>
		<li id="menu-item-2169" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2169"><a href="_b70473c7.html">2018; 6(17):371–372</a></li>
	</ul>
</li>
	<li id="menu-item-2200" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2200"><a href="_5c79e5ae.html">2018 Vol 6 No 18</a>
	<ul class="sub-menu">
		<li id="menu-item-2204" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2204"><a href="_7a1f8294.html">2018; 6(18):373–388</a></li>
		<li id="menu-item-2242" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2242"><a href="_34d09962.html">2018; 6(18):389–395</a></li>
		<li id="menu-item-2213" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2213"><a href="_d7780df8.html">2018; 6(18):396–405</a></li>
		<li id="menu-item-2226" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2226"><a href="_bf6e6bc8.html">2018; 6(18):406–413</a></li>
		<li id="menu-item-2257" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2257"><a href="_6215e068.html">2018; 6(18):414–427</a></li>
		<li id="menu-item-2278" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2278"><a href="_4bd5501.html">2018; 6(18):428–444</a></li>
	</ul>
</li>
</ul>
</li>
<li id="menu-item-2315" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2315"><a href="_2530ef33.html">2019</a>
<ul class="sub-menu">
	<li id="menu-item-2311" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2311"><a href="_6d4dd5c1.html">2019 Vol 7 No 19</a>
	<ul class="sub-menu">
		<li id="menu-item-2306" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2306"><a href="_d274d05.html">2019; 7(19):1–9</a></li>
		<li id="menu-item-2386" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2386"><a href="_d274cfd.html">2019; 7(19):10–20</a></li>
		<li id="menu-item-2321" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2321"><a href="_caa66125.html">2019; 7(19):21–29</a></li>
		<li id="menu-item-2338" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2338"><a href="_52763638.html">2019; 7(19):30–46</a></li>
		<li id="menu-item-2362" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2362"><a href="_82897a9b.html">2019; 7(19):47–54</a></li>
		<li id="menu-item-2372" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2372"><a href="_82897a9a.html">2019; 7(19):55–63</a></li>
	</ul>
</li>
	<li id="menu-item-2427" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2427"><a href="_1a5e6b6d.html">2019 Vol 7 No 20</a>
	<ul class="sub-menu">
		<li id="menu-item-2770" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2770"><a href="_a63083be.html">2019; 7(20):64–77</a></li>
		<li id="menu-item-2653" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2653"><a href="_5877a27f.html">2019; 7(20):78–85</a></li>
		<li id="menu-item-2740" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2740"><a href="_cdd9d011.html">2019; 7(20):86–92</a></li>
		<li id="menu-item-2743" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2743"><a href="_5877a27e.html">2019; 7(20):93–105</a></li>
		<li id="menu-item-2895" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2895"><a href="_21fd8046.html">2019; 7(20):106–120</a></li>
		<li id="menu-item-2919" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2919"><a href="_c4a4f4e3.html">2019; 7(20):121–128</a></li>
	</ul>
</li>
	<li id="menu-item-2960" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2960"><a href="_ca40ef92.html">2019 Vol 7 No 21</a>
	<ul class="sub-menu">
		<li id="menu-item-3026" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3026"><a href="_694b92a9.html">2019; 7(21):129–135</a></li>
		<li id="menu-item-3117" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3117"><a href="_c6a41e07.html">2019; 7(21):136–147</a></li>
		<li id="menu-item-3029" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3029"><a href="_fa5433e5.html">2019; 7(21):148–153</a></li>
		<li id="menu-item-3135" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3135"><a href="_59abe87d.html">2019; 7(21):154–160</a></li>
		<li id="menu-item-3047" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3047"><a href="_6fb66183.html">2019; 7(21):161–175</a></li>
		<li id="menu-item-3085" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3085"><a href="_c6a41e10.html">2019; 7(21):176–187</a></li>
	</ul>
</li>
	<li id="menu-item-3197" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3197"><a href="_59abe878.html">2019 Vol 8 No 22</a>
	<ul class="sub-menu">
		<li id="menu-item-3291" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3291"><a href="_4bd5522.html">2019; 8(22):188–195</a></li>
		<li id="menu-item-3234" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3234"><a href="_7a1f82b1.html">2019; 8(22):196–201</a></li>
		<li id="menu-item-3360" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3360"><a href="_c7d863ed.html">2019; 8(22):202–212</a></li>
		<li id="menu-item-3218" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3218"><a href="_b9d27134.html">2019; 8(22):213–221</a></li>
		<li id="menu-item-3270" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3270"><a href="_1cc6f752.html">2019; 8(22):222–230</a></li>
		<li id="menu-item-3330" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3330"><a href="_82897ab8.html">2019; 8(22):231–244</a></li>
	</ul>
</li>
	<li id="menu-item-3418" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3418"><a href="_32680dbe.html">2019 Vol 8 No 23</a>
	<ul class="sub-menu">
		<li id="menu-item-3451" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3451"><a href="_254c95d.html">2019; 8(23):245–257</a></li>
		<li id="menu-item-3500" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3500"><a href="_a4fc3df2.html">2019; 8(23):258–266</a></li>
		<li id="menu-item-3552" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3552"><a href="_3ad205c1.html">2019; 8(23):267–275</a></li>
		<li id="menu-item-3470" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3470"><a href="_77b6f6f8.html">2019; 8(23):276–286</a></li>
		<li id="menu-item-3536" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3536"><a href="_22c86393.html">2019; 8(23):287–296</a></li>
		<li id="menu-item-3579" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3579"><a href="_500daa9c.html">2019; 8(23):297–311</a></li>
	</ul>
</li>
	<li id="menu-item-3644" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3644"><a href="_5877a2a3.html">2019 Vol 8 No 24</a>
	<ul class="sub-menu">
		<li id="menu-item-3732" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3732"><a href="_60e19aa5.html">2019; 8(24):312–322</a></li>
		<li id="menu-item-3666" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3666"><a href="_cdd9d037.html">2019; 8(24):323–331</a></li>
		<li id="menu-item-3705" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3705"><a href="_b5d02e07.html">2019; 8(24):332–340</a></li>
		<li id="menu-item-3756" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3756"><a href="_d643c837.html">2019; 8(24):341–357</a></li>
		<li id="menu-item-3786" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3786"><a href="_78eb3cd5.html">2019; 8(24):358–371</a></li>
	</ul>
</li>
</ul>
</li>
<li id="menu-item-3889" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3889"><a href="_4f42c764.html">2020</a>
<ul class="sub-menu">
	<li id="menu-item-3863" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3863"><a href="_9f3de38.html">2020 Vol 9 No 25</a>
	<ul class="sub-menu">
		<li id="menu-item-3886" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3886"><a href="_9491b0a4.html">2020; 9(25):1–12</a></li>
		<li id="menu-item-3914" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3914"><a href="_9f3de2f.html">2020; 9(25):13–21</a></li>
		<li id="menu-item-4021" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4021"><a href="_27997b14.html">2020; 9(25):22–35</a></li>
		<li id="menu-item-4096" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4096"><a href="_694b92cd.html">2020; 9(25):36–47</a></li>
	</ul>
</li>
</ul>
</li>
</ul>						</nav>
					
					
					
										<div id="et_top_search">
						<span id="et_search_icon"></span>
					</div>
					
					<div id="et_mobile_nav_menu">
				<div class="mobile_nav closed">
					<span class="select_page">Select Page</span>
					<span class="mobile_menu_bar mobile_menu_bar_toggle"></span>
				</div>
			</div>				</div> <!-- #et-top-navigation -->
			</div> <!-- .container -->
			<div class="et_search_outer">
				<div class="container et_search_form_container">
					<form role="search" method="get" class="et-search-form" action="http://www.aimsci.com/ros/html/">
					<input type="search" class="et-search-field" placeholder="Search &hellip;" value="" name="s" title="Search for:" />					</form>
					<span class="et_close_search_field"></span>
				</div>
			</div>
		</header> <!-- #main-header -->
			<div id="et-main-area">
	
<div id="main-content">


	<div class="container">
		<div id="content-area" class="clearfix">
			<div id="left-area">


			
				<article id="post-748" class="post-748 page type-page status-publish hentry">

				
					<h1 class="entry-title main_title">2016; 2(5):325–338</h1>
				
				
					<div class="entry-content">
					<h1><span style="color: #808000;"><strong>REVIEW ARTICLES</strong></span></h1>
<h2><a href="../index.php/ros/article/view/55/139.html"><img class="alignright wp-image-231 " src="wp-content/uploads/2016/03/PDF-logo-blue-30.png" alt="PDF logo-blue 30" width="53" height="67" /></a></h2>
<p>&nbsp;</p>
<h2><span style="color: #000000;"><strong>Huntington’s Disease-Induced Cardiac Disorders Affect Multiple Cellular Pathways </strong></span></h2>
<p>&nbsp;</p>
<p><strong>Girish C. Melkani</strong></p>
<p>Department of Biology, Molecular Biology and Heart Institutes, San Diego State University, San Diego, CA 92182, USA</p>
<p>Correspondence: gmelkani@mail.sdsu.edu (G.C.M.)</p>
<p><em>Melkani GC. Reactive Oxygen Species 2(5):325–338, 2016; </em><em>©2016 Cell Med Press</em></p>
<p><a href="../index.php/ros/article/view/55.html"><em>http://dx.doi.org/10.20455/ros.2016.859</em></a></p>
<p><em>(Received: May 30, 2016; Revised: June 28, 2016; Accepted: June 28, 2016)</em></p>
<p><strong>ABSTRACT</strong> <strong>|</strong> Huntington&#8217;s disease (HD) is a rare, inherited, progressive, and fatal neurological disorder resulting from expanded polyglutamine repeats in the huntingtin protein. While HD is predominately characterized as a disease of the central nervous system, mortality surveys and epidemiological studies reveal heart disease as one of the leading causes of death in HD patients. Emerging evidence supports a link between HD and cardiovascular disease, such as cardiac amyloidosis (accumulation of aggregates in the heart). Experimental animal and clinical studies have attempted to explain the mechanisms of HD-induced cardiac pathology in the association of protein misfolding, autophagic defects, oxidative stress, mitochondrial dysfunction, and cell death. HD is increasingly understood as a complex disease with peripheral components of cardiac and skeletal muscle pathophysiology<strong>. </strong>While the discovery of these linkages and apparent pathological markers is promising, the mechanism of HD-induced cardiac pathology and the nature of its cell autonomy remain elusive<strong>. </strong>Further study of the wide-ranging cardiac function in HD patients is needed. This review highlights published literature on the pathological factors associated with HD-induced cardiac amyloidosis and other cardiovascular diseases, and addresses gaps in this expanding area of study. Through comprehensive experimental and clinical studies, potential drugs can be tested to attenuate and/or ameliorate HD-induced cardiac pathology and mortality.</p>
<p><strong>KEYWORDS</strong> <strong>|</strong> Cardiac amyloidosis; Cardiac diseases; Cardiomyopathy; Huntington’s disease; Oxidative stress; Protein aggregations; Protein unfolding; Reactive oxygen species</p>
<p><strong>ABBREVIATIONS </strong><strong>|</strong> BACHD, bacterial artificial chromosome Huntington&#8217;s disease; CNS, central nervous system; DHE, Dihydroethidium; ER, endoplasmic reticulum; FADD, Fas-associated protein with death domain; GFP, green fluorescent protein; HD, Huntington&#8217;s disease; HTT, huntingtin; PolyQ, polyglutamine; ROS, reactive oxygen species; SOD, superoxide dismutase; TNF, tumor necrosis factor; TNFR, TNF receptor; UPR, unfolded protein response</p>
<hr />
<p><strong>CONTENTS </strong></p>
<p>1. Introduction</p>
<p>2. Peripheral Pathology Associated with Huntington&#8217;s Disease</p>
<p>3. Huntington&#8217;s Disease-Induced Cardiac Abnormalities</p>
<p>4. Huntington&#8217;s Disease-Induced Cardiac Amyloidosis and Protein Misfolding</p>
<p>5. Huntington&#8217;s Disease-Induced Mitochondrial Dysfunction and Oxidative Stress</p>
<p>6. Huntington&#8217;s Disease-Induced Cardiac Disease and Cell Death</p>
<p>7. Huntington&#8217;s Disease-Induced Cardiac Disorders Affect Multiple Cellular Pathways</p>
<p>8. Future Directions and Therapeutic Strategy</p>
<hr />
<p><strong>1. INTRODUCTION</strong></p>
<p>A breakthrough in Huntington&#8217;s disease (HD) etiology came in 1971 when researchers determined its genetic basis via mutations in the huntingtin (HTT) gene. The pathological mechanisms of these genetic mutations remain elusive, and as a result, no sufficient disease modifying treatments are currently available [1, 2]. While we know that HD is a progressive and fatal neurological disorder that causes the breakdown of nerve cells in the brain, researchers are still exploring the pathology as attributed to the expanded polyglutamine (PolyQ) repeats in the HTT protein leading to HTT misfolding and the formation of pathogenic aggregates in neuronal tissue [3, 4]. Normally, the CAG (glutamine) segment is repeated 10 to 35 times within the gene. In HD patients, the CAG segment is repeated on average 36 to 120 times [5]. The precise function of HTT will be informative of the disease course, but remains unclear. However, HTT is known to be associated with cytoskeletal anchoring or transport of mitochondria as well as in vesicle trafficking [3, 6]. The mutant-induced expansion of PolyQ segment has been shown to lead to the production of an abnormally long version of the HTT protein. The elongated protein cannot fold properly or might be cut into smaller, toxic fragments that bind together and accumulate in neurons, which disrupts the normal functions of these cells. This dysfunction and the eventual death of neurons in certain areas of the brain underlie the signs and symptoms of HD. In addition, mutated HTT also interferes with several cellular processes, including disruption of mitochondrial function and glucose metabolism [7–9].</p>
<p>Despite significant progress in understanding the basic mechanisms of HD’s pathology, currently, effective therapies are limited [10, 11]. It is thus essential to look at HD’s role beyond the brain and consider it as a multi-system disorder [10, 11], which may lead to improved therapeutic strategies for this progressive and fatal neurological disease.</p>
<p><strong>2. PERIPHERAL PATHOLOGY ASSOCIATED WITH HUNTINGTON&#8217;S DISEASE </strong></p>
<p>HTT is ubiquitously expressed in human tissues and is involved in many critical cellular processes, suggesting its pivotal roles in and outside of the central nervous system (CNS) (i.e., the peripheral tissues) [3, 6–8, 11–20]. Besides the characteristic HD neurological symptoms, weight loss [21, 22, 23 , 24] and altered glucose homeostasis [25, 26] have been reported in HD pathology. Additional alterations in HD patients include subcellular abnormalities in fibroblasts, circadian dysfunction [27–31], lymphocytes [32, 33], and erythrocytes [9]. Growing evidence now suggests that cells from cardiac and muscle tissues of HD patients bear aberrations related to the expression of mutant HTT [11, 14–19, 34–43]. Besides the cardiac abnormalities (described below), skeletal muscle dysfunction is very common in HD patients and leads to severe muscle wasting [11, 19]. For example, measurement of lower limb strength in 20 HD patients was on average about half of that in healthy matched controls [11, 19, 20]. Muscle strength scores of HD patients correlate with their motor impairment; yet, it is unclear whether skeletal muscle dysfunction observed is an expression of mutant HTT in the skeletal muscle or a non-autonomous effect [20]. In addition, HD patients exhibit compromised limb strength and mitochondrial dysfunction at the ultrastructural level [44], suggesting altered energetic metabolism in skeletal muscle [11, 19]. It has been suggested that intense physical exercise might reveal subclinical HD-related myopathy [11, 19, 20]. Skeletal muscle abnormalities have also been observed in transgenic R6/2 HD mouse models [11, 13–19]. The mutant mice demonstrated skeletal muscle atrophy throughout lifespan, with muscle fibers shrinking to half the size of that of age-matched controls by 16 weeks of age in both type I and type II fibers [1, 17, 18, 45, 46]. Other studies have shown that expression of mutant HTT leads to intracellular aggregate formation in peripheral tissues and gene expression changes in HD peripheral tissues sections [10, 47]. Other abnormalities such as endocrine dysfunction, blood tissue abnormalities, and cell death in peripheral tissues have also been reported in both HD patients and animal models [11, 14–19, 34–43].</p>
<p>Despite all of the evidence from studying mutant phenotypes, it is still an open question as to whether HD-related peripheral abnormalities are secondary events independent of neuronal abnormalities, or partially caused by an intrinsic function of HTT in these tissues. Use of genetic models can address these questions by systematically analyzing the tissue-specific expression of mutant HD.</p>
<p><strong style="line-height: 1.7em;">3. HUNTINGTON&#8217;S DISEASE-INDUCED CARDIAC ABNORMALITIES</strong></p>
<p>In addition to peripheral pathologies, HD patients exhibit a high rate of cardiac events, with heart failure being the second leading cause of death among HD patients (accounting for 20–30% of HD deaths) [13, 43, 48–51]. Epidemiological studies have suggested a link between HD and cardiac dysfunction, including cardiac amyloidosis, a group of disorders characterized by protein misfolding and the accumulation of aggregates in the heart [13, 48, 50]. It is well known that HTT is also expressed in the heart; however, until recently, it was unclear whether cardiac pathology in HD patients is primarily due to expression of mutant HTT in the cardiomyocytes, to the brain dysfunction, or to the CNS malfunctions [34, 35, 37, 39, 43].</p>
<p>Most studies postulated that cardiac dysfunction is a result of the autonomic nervous system dysfunction in HD patients [34, 35, 37, 39, 43]. To demonstrate the role of mutant PolyQ repeats in the genesis of cardiac dysfunction, we generated a model for cardiac amyloidosis caused by expanded PolyQ repeats in HTT using the genetically tractable <em>Drosophila melanogaster</em> (commonly known as fruit fly) system [36, 40]. Heart-specific expression of mutant PolyQ resulted in functional and morphological defects, including floppy, non-contractile ostia, and one or more non-contractile myocardial cells. Expression of mutant PolyQ in the myocardial cells also resulted in PolyQ length-dependent cardiac defects, and hearts with longer PolyQ (PolyQ-72 and PolyQ-102) resulted in recurrent asystolic periods and near-complete lack of cardiac contractility [36, 40]. Our results also established that mutant PolyQ interferes with sarcomeric organization of contractile proteins (possibly due to accumulation of aggregates in the cardiomyocytes), causes increased oxidative stress, and leads to mitochondrial and autophagic defects, all of which eventually result in the destruction of cardiomyocytes. We concluded that there is a direct correlation between the levels of amyloid accumulation, reactive oxygen species (ROS) production, and the severity of cardiac dysfunction. In addition to severe cardiac defects, cardiac-specific expression of the mutant PolyQ proteins significantly impacted lifespan of flies. With our cardiac-specific expression, mutant Htt-PolyQ protein was expressed only in the heart; therefore, it is unlikely that our observations reflect a neuronal contribution to these cardiac defects.</p>
<p>Consistent with our findings in <em>Drosophila</em>, cardiac-specific expression of pre-amyloid oligomers (expended PolyQ without HTT) in mice has been shown to lead to cardiac defects [38]. Furthermore, expression of mutant PolyQ-81 in rat neonatal cardiomyocytes resulted in PolyQ-positive aggregates in the cytoplasm and overexpression of a chaperone αB-crystallin reduced PolyQ-induced aggresomes [52, 53]. These results suggest that the increased risk of cardiac disease in HD patients is possibly due to cardiac amyloid accumulation along with other factors such as mitochondrial defects and oxidative stress. Additional studies have also reported the cardiac phenotype in mouse models of HD; however, most did not observe aggregates in cardiomyocytes [34, 35, 37–39, 42, 43]. Results showed that neuronal expression of mutant HTT protein with expanded PolyQ in mice leads to cardiac defects due to alteration of several pathways including oxidative stress, mitochondrial defects, and cell death [34, 35, 37–39, 42, 43].</p>
<p>Since HTT is expressed in cardiac muscle, and PolyQ aggregates have been reported in non-CNS tissues [13, 40], it is postulated that cardiac expression of mutant PolyQ leads to pathogenic HTT aggregate formation in cardiac tissue and subsequent heart dysfunction. It is critical to further study protein aggregates in the cardiomyocytes of HD patients, especially those with heart disease symptoms. Demarcating how accumulation of aggregates might be toxic to cells will be key to understanding PolyQ-induced cardiomyopathy and neuropathy.</p>
<p><strong>4. HUNTINGTON&#8217;S DISEASE-INDUCED CARDIAC AMYLOIDOSIS AND PROTEIN MISFOLDING </strong></p>
<p>Several studies have shown that mutant Htt-PolyQ associated with neuropathy is prone to misfolding and aggregation [54–56], and is known to interfere with several cellular pathways, including the unfolded protein response (UPR) and the oxidative stress response [57, 58]. However, there are limited studies demonstrating this correlation of mutant PolyQ-induced cardiomyopathy [36, 38, 40, 53, 59, 60]. Recent evidence for amyloid deposition in cardiovascular disease (cardiac amyloidosis) has garnered strong interest [36, 38, 40, 61]. Of the identified amyloid precursor proteins, 19 heterogeneous proteins are prone to misfolding and are involved in cardiac amyloidosis leading to cardiomyopathy [36, 38, 40]. Infiltration of the heart from amyloid deposition often results in restrictive cardiomyopathy (RCM) or dilated-cardiomyopathy (DCM). While several proteins have been shown to be associated with cardiac amyloidosis, the precise mechanism that leads to this disease is poorly understood. We hypothesize that many proteins rely on the cellular folding milieu and that expression of toxic amyloids in the heart due to oxidative stress or global misfolding disrupts the global balance of protein folding and induces cardiac defects. To support our hypothesis, we recently established the link of cardiac amyloidosis with expression of aggregation-prone mutant PolyQ [36, 40]. Similar to methods used for studying HD-induced neuropathy and skeletal myopathy, we used aggregation-prone exon-1 HTT fragments to explore the mechanisms of HD-induced cardiac amyloidosis [11, 62].</p>
<p>Cardiac-specific expression of aggregation-prone exon-1 HTT fragment with enhanced-green fluorescent protein (GFP)-tagged mutant HTT fragments (Q46, Q72, and Q103) resulted in cardiac dilation, contractile defects, and cardiac arrhythmias in a PolyQ length-dependent manner [36, 40], suggesting that primary defects due to PolyQ expansion repeats are cardiac cell autonomous. Interestingly, we found that expression of pathogenic PolyQ repeats with GFP-tagged resulted in aggregation in the form of GFP-positive puncta in cardiomyocytes, whereas in a non-pathogenic control PolyQ-25 was distributed in the heart without forming aggregates. Based upon our structural and functional evidence, we concluded that accumulation of aggregation-prone mutant PolyQ is sufficient to cause cardiac dysfunction. Our study did not provide direct evidence on whether these aggregates are amyloid aggregates; however, our data are supported by mouse models for mutant PolyQ which resulted in the formation of aggregates that were immunoreactive for anti-amyloid in the heart [53, 60]. Additionally, previous studies have shown that expression of a polyglutamine (without HTT protein fragments), a preamyloid oligomer in cardiomyocytes, causes heart failure and shortened lifespan [38, 59]. These studies of cardiac-restricted expression of aggregate-prone mutant PQ83 demonstrated higher content of autophagosomes and lysosomes, as well as markers of necrotic death, including inflammatory cell infiltration and increased sarcolemmal permeability in cardiomyocytes [38, 59]. In addition, mutant PolyQ amyloid formation has been observed in a murine model for desmin-related cardiomyopathy (mutant alphaB crystallin R120G) [53, 60]. Overall, these studies are important in understanding the mechanism(s) by which intracellular amyloids cause cardiomyocyte dysfunction/death. We predict that amyloid deposits accumulate in cardiac tissue of HD patients and contribute to cardiomyopathy. Novel therapeutic strategies targeting reduction of protein aggregation may prove useful in clinical settings.</p>
<p>To further understand the mechanisms of mutant HTT-induced cardiomyopathy, we have explored the pathogenic signaling pathways associated with amyloid-induced cardiomyopathy. Accumulation of mutant PolyQ is known to interfere with the protein folding machinery in neurons, and it is suggested that PolyQ has the same effect in the heart [3, 4]. The roles of several molecular chaperones, including heat shock protein (Hsp)-70, Hsp-40, and Hsp-110, have been shown to modulate HD phenotypes in neurons [5, 55, 56]. These chaperones reduce toxicity by facilitating the folding of aggregation-prone mutant HTT and by reducing protein aggregation. Mutant PolyQ may also interfere with proteins required to maintain sarcomeric protein folding and thereby lead to misfolding of contractile proteins. Although mutant PolyQ was shown to lead to cardiomyocyte death in mouse hearts [39, 42], it was previously unknown whether PolyQ directly targets accumulation of key contractile proteins, e.g., myosin and actin. We have shown that a majority of PolyQ mutant hearts contain non-contractile regions, possibly due to cell death or lack of contractile proteins [40]. Since the proper folding of such proteins is essential for cardiac function, we sought to determine whether the presence of non-contractile regions in the hearts of PolyQ mutant flies could be attributed to a lack of accumulation of contractile proteins [36, 40]. We reasoned that overexpression of chaperone UNC-45 (required for myosin folding), which may enhance proper protein folding, might improve cardiac function in hearts compromised by disease-causing PolyQ expression.</p>
<p>Further study is needed to determine whether mutant PolyQ leads to muscle degeneration by directly interacting with these contractile proteins or by interfering with the unfolded protein response, which is necessary to maintain endoplasmic reticulum (ER) homeostasis and contractile protein turnover [63]. We have shown that transgenic overexpression of UNC-45 significantly suppressed mutant PolyQ-72 induced cardiac dilation and amyloid aggregates, and improved contractility and rhythmicity [40]. We also showed that GFP-tagged HTT protein aggregates direct myosin accusation in the heart [36, 40], signifying that pathogenic aggregate formation is associated with decreased cardiac function in PolyQ mutant hearts by directly interfering with the contractile apparatus.</p>
<p>Although overexpression of UNC-45 in PolyQ mutant hearts restored the integrity of actin-containing myofibrils as well, it is unclear if the loss of actin-containing myofibrils due to mutant PolyQ is caused by decreased accumulation of myosin, or if PolyQ aggregates interfere with the accumulation of other contractile proteins such as actin. Alternatively, mutant PolyQ-induced pathogenic aggregates may inhibit the unfolded protein response by directly interfering with chaperone activity. As shown for amyloid-induced desmin related cardiomyopathy (associated with mutant alphaB crystallin R120G), expression of this mutant in the cardiomyocytes caused defective chaperone activity and misfolding of contractile proteins [38, 59, 60]. Co-expression of both mutant alphaB crystallin and mutant PolyQ resulted in the accumulation of amyloid-positive aggregate in cardiomyocytes and myocyte degeneration [59], suggesting a possible link between protein misfolding and cardiomyocyte death. Collectively, these findings suggest that protein misfolding may play a role in mediating PolyQ-induced cardiomyopathy and cardiac amyloidosis.</p>
<p><strong style="line-height: 1.7em;">5. HUNTINGTON&#8217;S DISEASE-INDUCED MITOCHONDRIAL DYSFUNCTION AND OXIDATIVE STRESS</strong></p>
<p>Strong evidence of a connection between misfolded and/or aggregated proteins with mitochondrial damage and/or oxidative stress in neurological disorders has been reported [64, 65]. Several studies support the linkage of mitochondrial dysfunction and oxidative stress associated with mutant PolyQ-induced neuropathy. This relationship included decreased expression of electron transport chain proteins that have been linked to the presence of misfolded PolyQ aggregates in neurons of HD patients [17, 44, 64, 66–73]. The association of mitochondrial dysfunction and oxidative stress with mutant Poly-Q-induced cardiac diseases has been studied extensively [34–43]. The consensus is that a majority of ROS generation is due to leakage of electrons from the electron transport chain of mitochondria [74]. Mutant Ploy-Q-induced oxidative damage is more relevant to cardiac tissue because mitochondria occupy greater than 25% of cardiomyocyte volume to provide sufficient energy for lifetime contractility [75]. Thus, the damage of cardiomyocytes, which are heavily dependent on mitochondrial function (mitochondrial function itself is also vulnerable to oxidative stress with mutant Poly-Q), results in cardiomyopathy.</p>
<p>Mihm et al. first demonstrated a direct relationship between mutant PolyQ-induced cardiac dysfunction and oxidative stress in the R6/2 HD mouse model [39]. The authors demonstrated that expression of the mutant HTT protein leads to intra-cardiomyocyte distribution of mutant HTT and, thus, the roles of mitochondrial and oxidative stress in the development of progressive cardiac dysfunction [39]. Furthermore, enhancement of nuclear and mitochondrial PolyQ presence in the cardiomyocytes as well as the severe alterations in mitochondrial morphology were also discussed. Additional mechanistic studies showed significant enhancement in lysine acetylation and protein nitration associated with weakening in cardiac performance [39]. Several subsequent studies have demonstrated mutant PolyQ-mediated cardiac dysfunction linked to mitochondrial defects and ROS production [34–37, 40–43]. Additionally, using magnetic resonance imaging of the beating mouse heart, progressive cardiac defects have also been reported in the mouse models of mutant-Htt [35]. Using the Langendorff preparation, the authors demonstrated reduced coronary blood flow, impaired myocardial contractility, and reduced left ventricular developed pressure in HD mouse hearts [35]. More recent studies have also demonstrated the association of mutant PolyQ-induced cardiac dysfunction in a mouse model involving key components of Fas-dependent apoptosis (TNF-alpha, TNFR1, Fas ligand, Fas death receptors, FADD, activated caspase-8, and activated caspase-3) and the key components of mitochondria-dependent apoptosis (Bax, Bax-to-Bcl-2 ratio, cytosolic cytochrome c, activated caspase-9, and activated caspase-3) [34]. Their findings indicate that cardiac Fas-dependent and mitochondria-dependent apoptotic pathways were activated in transgenic mice with HD [34].</p>
<p>Overall, research supports previous epidemiological findings of increased risk of cardiac disease in HD patients. Still, very little is known about the mechanisms of cardiomyocyte dysfunction and mitochondrial involvement in mutant PolyQ-induced cardiomyopathy. Prior to our study, no attempt had been made to suppress PolyQ-induced cardiac defects, a crucial step toward understanding the mechanistic basis of HD disease progression and amelioration [40]. Our genetic <em>Drosophila</em> model of mutant PolyQ in conjunction with physiological, cytological and ultrastructural approaches, demonstrated the linkage of PolyQ-induced cardiomyopathy with oxidative stress, mitochondrial dysfunction, unfolded protein response, and imbalance of proteostasis [36, 40]. We have also shown that treatment with oxidants aggravated mutant PolyQ-46-induced cardiac defects by enhancing oxidative stress and increasing amyloid aggregate density. Using an ultrastructural (electron micrographic) approach, we demonstrated that cardiac restricted expression of mutant PolyQ led to severe mitochondrial cristae fragmentation. Furthermore, using dihydroethidium (DHE) staining we found excess ROS production upon expression of mutant PolyQ and co-localization of some of the DHE staining with PolyQ aggregates, suggesting a causal relationship. We were able to suppress amyloid-induced aggregates as well as the severity of the PolyQ72-induced cardiac defects by overexpression of superoxide dismutase (SOD) or by feeding the antioxidant resveratrol [36, 40]. Several neuronal studies had previously shown that expression of mutant polyQ affects SOD levels [76, 77]. Furthermore, manipulation of SOD was associated with levels of oxidative stress in several neurodegenerative diseases [76, 77]. SOD overexpression has also been shown to reduce diabetic cardiomyopathy and neurodegeneration by ROS [78].</p>
<p>Mitochondrial organization is meticulously controlled by the cardiomyocyte and therefore depends on the proper function and organization of contractile proteins [52]. The vulnerability of mitochondria to oxidative stress was shown in a mouse model for oxidative stress-related dilated cardiomyopathy by genetic ablation of the mitochondrial form of superoxide dismutase, SOD2 (MnSOD), which resulted in an abnormal &#8220;ballooning&#8221; of mitochondria in electron micrographs of dendrites and decreased lifespan [79]. Cardiac-specific expression of mutant αB-crystallin also resulted in the accumulation of amyloid-containing protein aggregates in cardiomyocytes in a cell-autonomous manner, leading to enhanced oxidative stress [52, 60, 80]. Decreasing superoxide production in αB-crystallin mutants by pharmacologically inhibiting the activity of the superoxide-generating xanthine oxidase restored mitochondrial function and membrane potential, suggesting a link between mitochondrial dysfunction and oxidative stress due to cardiac amyloidosis [80]. Thus, enhanced oxidative stress and mitochondrial dysfunction leading to heart failure may have a common mechanism among cardiac amyloidosis-based disorders. Additional studies are required to determine whether myofibrils degeneration/disorganization is due to aggregation of mutant PolyQ directly inhibiting mitochondrial organization, and/or whether oxidative stress produced by PolyQ leads to mitochondrial dysfunction.</p>
<p><strong style="line-height: 1.7em;">6. HUNTINGTON&#8217;S DISEASE-INDUCED CARDIAC DISEASE AND CELL DEATH</strong></p>
<p>In addition to protein misfolding, mitochondrial defects, and oxidative stress, cell death is one of the major events reported in neuronal and peripheral pathologies associated with mutant huntingtin in HD [6, 8, 30, 46, 81–86]. A recent study reported the association between mutant HTT and cell death pathways, particularly evident in cardiac tissue [34]. Activation of cardiac Fas-dependent apoptotic pathway in the heart of R6/2 HD mice was reported [34]. These authors demonstrated that key components of Fas-dependent apoptosis, such as TNF-alpha, Fas ligand, Fas death receptors, FADD, activated caspase-8, and activated caspase-3, were found activated in the HD model compared to age-associated control mice [34]. In addition to activation of the Fas-dependent autophagy pathway, their findings demonstrated activation of the key components associated with mitochondria-dependent apoptosis (Bax, Bax-to-Bcl-2 ratio, cytosolic cytochrome c, activated caspase-9, and activated caspase-3) in HD mice compared to wild-type mice [34]. This study concluded that an activation of both Fas-dependent and mitochondria-dependent apoptotic pathways might be accountable for abnormal myocardial architecture, enlarged cardiac interstitial spaces, and more cardiac TUNEL-positive cells in the HD mice [34]. Of note, this study included mice at 10.5 weeks of age, the stage at which symptoms of neurological disorders are observed. The authors sought to demonstrate whether apoptotic pathways are activated in the cardiac tissue due to the presence of neurological disorder in these HD mice [34]. However, this study did not directly address whether the activation of Fas-dependent and mitochondria-dependent apoptotic pathways is primary and due to expression of mutant HTT in the heart, or secondary and due to their neuronal expression.</p>
<p>Additional findings do support the association of apoptotic pathways with mutant HD-induced cardiac abnormalities [10, 37, 38, 87]. For example, Pattison et al. [38] reported cell death upon cardiac expression of PQ83 amyloid in the cardiomyocytes and not due to apoptosis, mitochondrial swelling, or ER stress. Further assessment revealed that the PQ83 amyloid-induced cell death in the cardiomyocytes is due to cell autophagy and necrosis [38]. Recently, the R6/2 transgenic and HdhQ150 knock-in mouse models of HD revealed that cardiac dysfunction is due to apoptotic loss of cardiomyocytes, and, to some extent, interstitial fibrosis [87]. However, the authors were unable to detect any aggregates and HD-specific transcriptional dysregulation in the cardiomyocytes in this mutant Htt mouse. The authors concluded that HD-related cardiomyopathy is caused by altered central autonomic pathways; however, the process, by which this autonomic signaling results in cardiac dysfunction, is not fully understood [87]. Additionally, a new cardiovascular phenotype of HD was recently reported using the bacterial artificial chromosome Huntington disease (BACHD) mouse model [37]. Besides the appearance of fibrotic lesions, markers of apoptosis were observed in the cardiomyocytes using gene expression analysis of the young mutant heart. Additional alteration of genes associated with cellular metabolism and proliferation were also reported in BACHD mouse heart [37]. Alteration of genes associated with cardiac disease, cellular metabolism, and cellular transport clusters was more drastic in the BACHD upon aging, indicating an age-dependent progression in heart dysfunction and gene alteration.</p>
<p>Our study did not directly perform apoptotic assay upon cardiac-specific expression of mutant HTT; however, we have shown the presence of non-contractile myocardial cells [36, 40]. Due to the accumulation of aggregates in cardiomyocytes, we observed a complete lack or disorganization of myofibrils, and increased oxidative stress which led to mitochondrial and autophagic defects and eventually resulted in the destruction of cardiomyocytes [36, 40]. Overall, these findings show that mutant PolyQ-induced myocardial cell death plays a role in cardiac pathogenesis. Additional studies will determine whether cell death occurs through defects in energy generation or by activation of apoptotic pathways.</p>
<p><strong style="line-height: 1.7em;">7. HUNTINGTON&#8217;S DISEASE-INDUCED CARDIAC DISORDERS AFFECT MULTIPLE CELLULAR PATHWAYS</strong></p>
<p>It is clear from multiple studies that the underlying mechanism of cardiac dysfunction in HD is complex and affects several pathogenic pathways (<strong>Figure 1</strong>), and, more importantly, that further study of their roles in HD is needed, including protein unfolding, oxidative stress, mitochondrial, autophagy, cell death, and transcriptional dysregulation. For example, Mihm et al. [39] did not find evidence of apoptotic cell death in the cardiac model of HD mice; however, they reported metabolic dysregulation along with mitochondrial defects. Mielcarek et al. [87] was unable to find mutant HTT-induced aggregates or a HD-specific transcriptional dysregulation in the hearts of R6/2 mice, but the authors demonstrated that the toxicity caused upon expression of mutant HTT in the cardiomyocytes might be a significant factor for contributing to cardiac dysfunction. Other studies have shown that expression of mutant huntingtin proteins in the heart resulted in the accumulation of aggregates in the cardiomyocytes; these aggregates are one of the main hallmarks of HD pathology [1, 36, 38, 40].</p>
<p><a href="wp-content/uploads/2016/08/figure-1-PNG-2.png"><img class="alignnone size-medium wp-image-754" src="wp-content/uploads/2016/08/figure-1-PNG-2-300x240.png" alt="figure 1 - PNG" width="300" height="240" srcset="http://www.aimsci.com/ros/html/wp-content/uploads/2016/08/figure-1-PNG-2-300x240.png 300w, http://www.aimsci.com/ros/html/wp-content/uploads/2016/08/figure-1-PNG-2-768x614.png 768w, http://www.aimsci.com/ros/html/wp-content/uploads/2016/08/figure-1-PNG-2-1024x819.png 1024w, http://www.aimsci.com/ros/html/wp-content/uploads/2016/08/figure-1-PNG-2.png 1042w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p><strong>FIGURE 1.</strong>   <strong>A summary of mutant-HTT-induced cardiac pathology and its therapeutic intervention. </strong>HTT denotes Huntingtin. CNS denotes central nervous system.</p>
<p>With multiple epidemiological and animal model data, there is solid evidence that mutant HTT leads to cardiac dysfunction and involves several pathways. We have demonstrated that several pathways such as protein unfolding, oxidative stress, mitochondrial dysfunction, and autophagy in mutant HD-induced pathology [36, 40]. Further, using the powerful genetic model <em>Drosophila</em> we manipulated both the oxidative stress and protein misfolding pathways to ameliorate PolyQ-induced cardiac amyloidosis and phenotypes [40]. Our physiological and cytological evidence revealed that manipulation of protein misfolding or oxidative stress alone does not completely ameliorate mutant PolyQ-induced cardiac dysfunction [10]. Subsequent studies demonstrated that manipulation of oxidative stresses in combination with the protein misfolding defects nearly completely ameliorated mutant PolyQ-induced cardiac dysfunction in vivo compared to either single manipulation alone. Based upon our genetic suppression of mutant GFP<sup>+</sup> PolyQ aggregates in mutant PolyQ-72 hearts with combined treatment, which correlates with improvement of cardiac function, we concluded the involvement of more than one pathway in mutant HD-induced cardiac pathology. We also demonstrated that mutant PolyQ-induced mitochondrial and myofibrillar degeneration can be suppressed with combined overexpression of UNC-45 and SOD (manipulation of protein misfolding and oxidative stress), as discussed above. Combined manipulation may also suppress mutant HD-induced cell death or autophagic defects, possibly by improving proteostasis pathways. As shown with in vitro findings [88], overexpression of SOD in conjunction with chaperones Hsp-70/Hsp-40 suppresses PolyQ aggregation in mouse neurons cell model, suggesting that targeting both cellular pathways is efficacious in HD pathogenesis in multiple tissue types.</p>
<p>In general, expression of the mutated HTT protein or expression of pre-amyloid oligomers cause cardiac defects by affecting several pathways including cell death, necrosis, oxidative stress, mitochondrial defects, presence of protein aggregates, and increased autophagosomal content [10, 37, 38, 39, 53, 87]. The involvement of oxidative stress and protein unfolding pathways play a role in PolyQ-induced cardiac pathogenesis, and targeting additional pathways and mechanistic basis of these pathways will be a relevant approach to ameliorate mutant HD-based cardiomyopathy.</p>
<p><strong style="line-height: 1.7em;">8. FUTURE DIRECTIONS AND THERAPEUTIC STRATEGY</strong></p>
<p>Mounting evidence indicates the significance of the cardiac phenotype in HD patients. Questions remain in understanding the mechanistic basis of HD-induced cardiomyopathy that will help in the design of therapies to treat the disease. A full emphasis of all the molecular players involved will require additional study (<strong>Figure 1</strong>). Recent studies aim to identify some of the key players and interactions in vivo in HD-induced cardiac pathology. With HD’s significant peripheral components, therapeutic approaches should not be limited to neuropathy; we should also seek to understand the cross-talk between the cardiac system and CNS, and the skeletal muscle and the CNS.</p>
<p>One of the controversies regarding HD-induced cardiomyopathy in particular is whether cardiac abnormalities are CNS-dependent or -independent. Some researchers believe that the cardiac physiological dysfunction in HD animal models occurs much later than CNS dysfunction, and thus heart dysfunction could be secondary to CNS malformation [34, 37, 39, 43]. However, strong evidence suggests that cardiac-specific expression of mutant HTT leads to the accumulation of aggregates and results in cardiac failure without neuronal involvement [34, 37, 39, 43]. This logical approach is promising to understand the mechanisms of pathophysiology as HTT is expressed strongly in the peripheral tissues including cardiac tissue [7, 11, 13, 89]. Furthermore, aggregation of the toxic HTT is a hallmark of HD pathology, and potentially similar pathology may occur in cardiomyocyte destruction as reported in mouse and fly models [40, 59, 61]. Some have proposed that mutant PolyQ forms aggregates in the heart, causing mitochondrial abnormalities associated with enhanced ROS production [39, 40]. Regardless of the detection of aggregates in the cardiomyocytes or not, most studies of the cardiac model showed mitochondrial dysfunction in HD-induced cardiac pathology [34, 39, 40]. We and others hypothesize that since mitochondria occupy a large portion of cardiomyocyte volume to provide sufficient energy for contractility [75, 90], myofibrillar degeneration and disorganization in PolyQ mutant hearts may occur due to defective ATP generation by dysfunctional mitochondria or enhanced ER stress. Alternatively, mitochondrial dysfunction may trigger the activation of apoptotic pathways, causing cell death in PolyQ mutant hearts. A role for ROS in inducing cardiac dysfunction has also been suggested; it is unknown if accumulation of PolyQ aggregates generates ROS by interfering with mitochondrial function directly, and/or if mitochondrial damage is a result of ROS generation due to PolyQ aggregation [91]. The exact mechanisms regulating mitochondrial damage in association with ROS will be deciphered in future studies.</p>
<p>The limited number of studies performed in HD human subjects creates a challenge in drawing conclusions regarding HD-related heart dysfunction in humans. For example, although dense granular deposits—immunoreactive to an anti-HTT antibody—have been found in muscle tissue from HD patients, no such study has been performed on HD heart biopsy samples [92]. First, we need to address the granular deposits of accumulation amyloid that have been observed in cardiomyocyte. We also need to use proteomics and/or transcriptomics approaches to obtain unbiased results of cardiac abnormalities associated with HD pathology. Further, we need to generate more precise mutant (including full-length HTT with PolyQ) by manipulating endogenous HTT gene using CRISPR/CAS-9 approaches. Outcomes from these studies will elucidate the molecular players associated with the accumulation of PolyQ-induced cardiomyopathy.</p>
<p>To develop more effective therapeutic interventions for HD, a better understanding of the mechanistic basis and pathological pathways of cardiac disorders is required. Once the association of molecular players is well understood (<strong>Figure 1</strong>), we can use the small molecules screening approach to ameliorate HD pathology as well as HD-induced cardiac pathology. Our focus using small molecules should be in suppressing aggregates-induced pathology, mitochondrial dysfunction, oxidative stress, and cell death. Additional genes/pathways obtained from manipulation of the CRISPR/CAS-9 approach can also be studied. Delineating how amyloid might be toxic to cells will be critical not only for an understanding of PolyQ-induced cardiomyopathy, but also for insights into amyloid-based neural degeneration.</p>
<p><strong>ACKNOWLEDGMENTS</strong></p>
<p>This work was supported by the National Institutes of Health grants RR032100, AG049494, and American Heart Association grant 17260057 to G.C.M.</p>
<p><strong>REFERENCES</strong></p>
<ol>
<li>Sathasivam K, Hobbs C, Turmaine M, Mangiarini L, Mahal A, Bertaux F, et al. Formation of polyglutamine inclusions in non-CNS tissue. <em>Hum Mol Genet</em> 1999; 8(5):813–22.</li>
<li>Petersen A, Hansson O, Brundin P. Huntington disease‒yet another mad protein? <em>Lakartidningen</em> 2001; 98(50):5756–8, 61.</li>
<li>Dyer RB, McMurray CT. Mutant protein in Huntington disease is resistant to proteolysis in affected brain. <em>Nat Genet</em> 2001; 29(3):270–8. doi: 10.1038/ng745.</li>
<li>Hoop CL, Lin HK, Kar K, Magyarfalvi G, Lamley JM, Boatz JC, et al. Huntingtin exon 1 fibrils feature an interdigitated beta-hairpin-based polyglutamine core. <em>Proc Natl Acad Sci USA</em> 2016; 113(6):1546–51. doi: 10.1073/pnas.1521933113.</li>
<li>Arrasate M, Finkbeiner S. Protein aggregates in Huntington&#8217;s disease. <em>Exp Neurol</em> 2012; 238(1):1–11.</li>
<li>Gusella JF, MacDonald ME, Ambrose CM, Duyao MP. Molecular genetics of Huntington&#8217;s disease. <em>Arch Neurol</em> 1993; 50(11):1157–63.</li>
<li>Strong TV, Tagle DA, Valdes JM, Elmer LW, Boehm K, Swaroop M, et al. Widespread expression of the human and rat Huntington&#8217;s disease gene in brain and nonneural tissues. <em>Nat Genet</em> 1993; 5(3):259–65. doi: 10.1038/ng1193-259.</li>
<li>Sassone J, Colciago C, Cislaghi G, Silani V, Ciammola A. Huntington&#8217;s disease: the current state of research with peripheral tissues. <em>Exp Neurol</em> 2009; 219(2):385–97. doi: 10.1016/j.expneurol.2009.05.012.</li>
<li>Zanella A, Izzo C, Meola G, Mariani M, Colotti MT, Silani V, et al. Metabolic impairment and membrane abnormality in red cells from Huntington&#8217;s disease. <em>J Neurol Sci</em> 1980; 47(1):93–103.</li>
<li>Mielcarek M. Huntington&#8217;s disease is a multi-system disorder. <em>Rare Dis</em> 2015; 3(1):e1058464. doi: 10.1080/21675511.2015.1058464.</li>
<li>Waters CW, Varuzhanyan G, Talmadge RJ, Voss AA. Huntington disease skeletal muscle is hyperexcitable owing to chloride and potassium channel dysfunction. <em>Proc Natl Acad Sci USA</em> 2013; 110(22):9160–5. doi: 10.1073/pnas.1220068110.</li>
<li>Kwan W, Trager U, Davalos D, Chou A, Bouchard J, Andre R, et al. Mutant huntingtin impairs immune cell migration in Huntington disease. <em>J Clin Invest</em> 2012; 122(12):4737–47. doi: 10.1172/JCI64484.</li>
<li>Bradford JW, Li S, Li XJ. Polyglutamine toxicity in non-neuronal cells. <em>Cell Res</em> 2010; 20(4):400–7. doi: 10.1038/cr.2010.32.</li>
<li>Robbins JH, Brumback RA, Polinsky RJ, Wirtschafter JD, Tarone RE, Scudiero DA, et al. Hypersensitivity to DNA-damaging agents in abiotrophies: a new explanation for degeneration of neurons, photoreceptors, and muscle in Alzheimer, Parkinson and Huntington diseases, retinitis pigmentosa, and Duchenne muscular dystrophy. <em>Basic Life Sci</em> 1985; 35:315–44.</li>
<li>Ansved T, Lundin A, Anvret M. Larger CAG expansions in skeletal muscle compared with lymphocytes in Kennedy disease but not in Huntington disease. <em>Neurology</em> 1998; 51(5):1442–4.</li>
<li>Ciammola A, Sassone J, Sciacco M, Mencacci NE, Ripolone M, Bizzi C, et al. Low anaerobic threshold and increased skeletal muscle lactate production in subjects with Huntington&#8217;s disease. <em>Mov Disord</em> 2011; 26(1):130–7. doi: 10.1002/mds.23258.</li>
<li>Gizatullina ZZ, Lindenberg KS, Harjes P, Chen Y, Kosinski CM, Landwehrmeyer BG, et al. Low stability of Huntington muscle mitochondria against Ca<sup>2+</sup> in R6/2 mice. <em>Ann Neurol</em> 2006; 59(2):407–11. doi: 10.1002/ana.20754.</li>
<li>Ribchester RR, Thomson D, Wood NI, Hinks T, Gillingwater TH, Wishart TM, et al. Progressive abnormalities in skeletal muscle and neuromuscular junctions of transgenic mice expressing the Huntington&#8217;s disease mutation. <em>Eur J Neurosci</em> 2004; 20(11):3092–114. doi: 10.1111/j.1460-9568.2004.03783.x.</li>
<li>Zielonka D, Piotrowska I, Marcinkowski JT, Mielcarek M. Skeletal muscle pathology in Huntington&#8217;s disease. <em>Front Physiol</em> 2014; 5:380. doi: 10.3389/fphys.2014.00380.</li>
<li>Busse ME, Hughes G, Wiles CM, Rosser AE. Use of hand-held dynamometry in the evaluation of lower limb muscle strength in people with Huntington&#8217;s disease. <em>J Neurol</em> 2008; 255(10):1534–40. doi: 10.1007/s00415-008-0964-x.</li>
<li>Sanberg PR, Fibiger HC, Mark RF. Body weight and dietary factors in Huntington&#8217;s disease patients compared with matched controls. <em>Med J Aust</em> 1981; 1(8):407–9.</li>
<li>Djousse L, Knowlton B, Cupples LA, Marder K, Shoulson I, Myers RH. Weight loss in early stage of Huntington&#8217;s disease. <em>Neurology</em> 2002; 59(9):1325–30.</li>
<li>Fennema-Notestine C, Archibald SL, Jacobson MW, Corey-Bloom J, Paulsen JS, Peavy GM, et al. In vivo evidence of cerebellar atrophy and cerebral white matter loss in Huntington disease. <em>Neurology</em> 2004; 63(6):989–95.</li>
<li>Aziz NA, Jurgens CK, Landwehrmeyer GB, Group ERS, van Roon-Mom WM, van Ommen GJ, et al. Normal and mutant HTT interact to affect clinical severity and progression in Huntington disease. <em>Neurology</em> 2009; 73(16):1280–5. doi: 10.1212/WNL.0b013e3181bd1121.</li>
<li>Podolsky S, Leopold NA. Abnormal glucose tolerance and arginine tolerance tests in Huntington&#8217;s disease. <em>Gerontology</em> 1977; 23(1):55–63.</li>
<li>Podolsky S, Leopold NA, Sax DS. Increased frequency of diabetes mellitus in patients with Huntington&#8217;s chorea. <em>Lancet</em> 1972; 1(7765):1356–8.</li>
<li>Menkes JH, Stein N. Fibroblast cultures in Huntington&#8217;s disease. <em>N Engl J Med</em> 1973; 288(16):856–7. doi: 10.1056/NEJM197304192881624.</li>
<li>Van Putten T, Menkes JH. Huntington&#8217;s disease masquerading as chronic schizophrenia. <em>Dis Nerv Syst</em> 1973; 34(1):54–6.</li>
<li>Leonardi A, De Martini IS, Perdelli F, Mancardi GL, Salvarani S, Bugiani O. Skin fibroblasts in Huntington&#8217;s disease. <em>N Engl J Med</em> 1978; 298(11):632. doi: 10.1056/NEJM197803162981119.</li>
<li>Beverstock GC. The current state of research with peripheral tissues in Huntington disease. <em>Hum Genet</em> 1984; 66(2–3):115–31.</li>
<li>Kudo T, Schroeder A, Loh DH, Kuljis D, Jordan MC, Roos KP, et al. Dysfunctions in circadian behavior and physiology in mouse models of Huntington&#8217;s disease. <em>Exp Neurol</em> 2011; 228(1):80–90. doi: 10.1016/j.expneurol.2010.12.011.</li>
<li>Pettegrew JW, Nichols JS, Stewart RM. Membrane studies in Huntington&#8217;s disease: steady state and time-dependent fluorescence spectroscopy of intact lymphocytes. <em>J Neurochem</em> 1981; 36(6):1966–76.</li>
<li>McGovern D, Webb T. Sensitivity to ionising radiation of lymphocytes from Huntington&#8217;s chorea patients compared to controls. <em>J Med Genet</em> 1982; 19(3):168–74.</li>
<li>Wu BT, Chiang MC, Tasi CY, Kuo CH, Shyu WC, Kao CL, et al. Cardiac Fas-dependent and mitochondria-dependent apoptotic pathways in a transgenic mouse model of Huntington&#8217;s disease. <em>Cardiovasc Toxicol</em> 2016; 16(2):111–21. doi: 10.1007/s12012-015-9318-y.</li>
<li>Wood NI, Sawiak SJ, Buonincontri G, Niu Y, Kane AD, Carpenter TA, et al. Direct evidence of progressive cardiac dysfunction in a transgenic mouse model of Huntington&#8217;s disease. <em>J Huntingtons Dis</em> 2012; 1(1):57–64. doi: 10.3233/JHD-2012-120004.</li>
<li>Trujillo AS, Ramos R, Bodmer R, Bernstein SI, Ocorr K, Melkani GC. Drosophila as a potential model to ameliorate mutant Huntington-mediated cardiac amyloidosis. <em>Rare Dis</em> 2014; 2(1):e968003. doi: 10.4161/2167549X.2014.968003.</li>
<li>Schroeder AM, Wang HB, Park S, Jordan MC, Gao F, Coppola G, et al. Cardiac dysfunction in the bachd mouse model of Huntington&#8217;s disease. <em>PLoS One</em> 2016; 11(1):e0147269. doi: 10.1371/journal.pone.0147269.</li>
<li>Pattison JS, Robbins J. Protein misfolding and cardiac disease: establishing cause and effect. <em>Autophagy</em> 2008; 4(6):821–3.</li>
<li>Mihm MJ, Amann DM, Schanbacher BL, Altschuld RA, Bauer JA, Hoyt KR. Cardiac dysfunction in the R6/2 mouse model of Huntington&#8217;s disease. <em>Neurobiol Dis</em> 2007; 25(2):297–308. doi: 10.1016/j.nbd.2006.09.016.</li>
<li>Melkani GC, Trujillo AS, Ramos R, Bodmer R, Bernstein SI, Ocorr K. Huntington&#8217;s disease induced cardiac amyloidosis is reversed by modulating protein folding and oxidative stress pathways in the <em>Drosophila</em> heart. <em>PLoS Genet</em> 2013; 9(12):e1004024. doi: 10.1371/journal.pgen.1004024.</li>
<li>Lin CL, Wang SE, Hsu CH, Sheu SJ, Wu CH. Oral treatment with herbal formula B307 alleviates cardiac failure in aging R6/2 mice with Huntington&#8217;s disease via suppressing oxidative stress, inflammation, and apoptosis. <em>Clin Interv Aging</em> 2015; 10:1173–87. doi: 10.2147/CIA.S86493.</li>
<li>Kiriazis H, Jennings NL, Davern P, Lambert G, Su Y, Pang T, et al. Neurocardiac dysregulation and neurogenic arrhythmias in a transgenic mouse model of Huntington&#8217;s disease. <em>J Physiol</em> 2012; 590(22):5845–60. doi: 10.1113/jphysiol.2012.238113.</li>
<li>Abildtrup M, Shattock M. Cardiac dysautonomia in Huntington&#8217;s disease. <em>J Huntingtons Dis</em> 2013; 2(3):251–61. doi: 10.3233/JHD-130054.</li>
<li>Browne SE. Mitochondria and Huntington&#8217;s disease pathogenesis: insight from genetic and chemical models. <em>Ann N Y Acad Sci</em> 2008; 1147:358–82. doi: 10.1196/annals.1427.018.</li>
<li>Davies SW, Sathasivam K, Hobbs C, Doherty P, Mangiarini L, Scherzinger E, et al. Detection of polyglutamine aggregation in mouse models. <em>Methods Enzymol</em> 1999; 309:687–701.</li>
<li>Sathasivam K, Hobbs C, Mangiarini L, Mahal A, Turmaine M, Doherty P, et al. Transgenic models of Huntington&#8217;s disease. <em>Philos Trans R Soc Lond B Biol Sci</em> 1999; 354(1386):963–9. doi: 10.1098/rstb.1999.0447.</li>
<li>Mielcarek M, Bondulich MK, Inuabasi L, Franklin SA, Muller T, Bates GP. The Huntington&#8217;s disease-related cardiomyopathy prevents a hypertrophic response in the R6/2 mouse model. <em>PLoS One</em> 2014; 9(9):e108961. doi: 10.1371/journal.pone.0108961.</li>
<li>Chiu E, Alexander L. Causes of death in Huntington&#8217;s disease. <em>Med J Aust</em> 1982; 1(4):153.</li>
<li>Heemskerk AW, Roos RA. Aspiration pneumonia and death in Huntington&#8217;s disease. <em>PLoS Curr</em> 2012; 4:RRN1293. doi: 10.1371/currents.RRN1293.</li>
<li>Lanska DJ, Lavine L, Lanska MJ, Schoenberg BS. Huntington&#8217;s disease mortality in the United States. <em>Neurology</em> 1988; 38(5):769–72.</li>
<li>Sorensen SA, Fenger K. Causes of death in patients with Huntington&#8217;s disease and in unaffected first degree relatives. <em>J Med Genet</em> 1992; 29(12):911–4.</li>
<li>Maloyan A, Sanbe A, Osinska H, Westfall M, Robinson D, Imahashi K, et al. Mitochondrial dysfunction and apoptosis underlie the pathogenic process in alpha-B-crystallin desmin-related cardiomyopathy. <em>Circulation</em> 2005; 112(22):3451–61. doi: 10.1161/CIRCULATIONAHA.105.572552.</li>
<li>Sanbe A, Osinska H, Villa C, Gulick J, Klevitsky R, Glabe CG, et al. Reversal of amyloid-induced heart disease in desmin-related cardiomyopathy. <em>Proc Natl Acad Sci USA</em> 2005; 102(38):13592–7.</li>
<li>Ivan Dimauro NM, Daniela Capoross. Exercise-induced ROS in heat shock proteins response. <em>Free Radical Biology and Medicine</em> 2016; Accepted.</li>
<li>Kobayashi Y, Kume A, Li M, Doyu M, Hata M, Ohtsuka K, et al. Chaperones Hsp70 and Hsp40 suppress aggregate formation and apoptosis in cultured neuronal cells expressing truncated androgen receptor protein with expanded polyglutamine tract. <em>J Biol Chem</em> 2000; 275(12):8772–8.</li>
<li>Kuo Y, Ren S, Lao U, Edgar BA, Wang T. Suppression of polyglutamine protein toxicity by co-expression of a heat-shock protein 40 and a heat-shock protein 110. <em>Cell Death Dis</em> 2013; 4:e833. doi: 10.1038/cddis.2013.351.</li>
<li>Cheng B, Gong H, Xiao H, Petersen RB, Zheng L, Huang K. Inhibiting toxic aggregation of amyloidogenic proteins: a therapeutic strategy for protein misfolding diseases. <em>Biochim Biophys Acta</em> 2013; 1830(10):4860–71. doi: 10.1016/j.bbagen.2013.06.029.</li>
<li>Tashiro E, Zako T, Muto H, Itoo Y, Sorgjerd K, Terada N, et al. Prefoldin protects neuronal cells from polyglutamine toxicity by preventing aggregation formation. <em>J Biol Chem</em> 2013; 288(27):19958–72. doi: 10.1074/jbc.M113.477984.</li>
<li>Pattison JS, Sanbe A, Maloyan A, Osinska H, Klevitsky R, Robbins J. Cardiomyocyte expression of a polyglutamine preamyloid oligomer causes heart failure. <em>Circulation</em> 2008; 117(21):2743–51. doi: 10.1161/CIRCULATIONAHA.107.750232.</li>
<li>Sanbe A, Osinska H, Saffitz JE, Glabe CG, Kayed R, Maloyan A, et al. Desmin-related cardiomyopathy in transgenic mice: a cardiac amyloidosis. <em>Proc Natl Acad Sci USA</em> 2004; 101(27):10132–6. doi: 10.1073/pnas.0401900101.</li>
<li>Heidari R, Monnier V, Martin E, Tricoire H. Methylene blue partially rescues heart defects in a <em>Drosophila</em> model of Huntington&#8217;s disease. <em>J Huntingtons Dis</em> 2015; 4(2):173–86. doi: 10.3233/JHD-140130.</li>
<li>Strand AD, Aragaki AK, Shaw D, Bird T, Holton J, Turner C, et al. Gene expression in Huntington&#8217;s disease skeletal muscle: a potential biomarker. <em>Hum Mol Genet</em> 2005; 14(13):1863–76. doi: 10.1093/hmg/ddi192.</li>
<li>Groenendyk J, Sreenivasaiah PK, Kim do H, Agellon LB, Michalak M. Biology of endoplasmic reticulum stress in the heart. <em>Circ Res</em> 2010; 107(10):1185–97. doi: 10.1161/CIRCRESAHA.110.227033.</li>
<li>Rodolfo C, Ciccosanti F, Giacomo GD, Piacentini M, Fimia GM. Proteomic analysis of mitochondrial dysfunction in neurodegenerative diseases. <em>Expert Rev Proteomics</em> 2010; 7(4):519–42. doi: 10.1586/epr.10.43.</li>
<li>Shirendeb U, Reddy AP, Manczak M, Calkins MJ, Mao P, Tagle DA, et al. Abnormal mitochondrial dynamics, mitochondrial loss and mutant huntingtin oligomers in Huntington&#8217;s disease: implications for selective neuronal damage. <em>Hum Mol Genet</em> 2011; 20(7):1438–55. doi: 10.1093/hmg/ddr024.</li>
<li>Luft R, Luthman H. Physiopathology of mitochondria: from Luft&#8217;s disease to aging and diabetes. <em>Lakartidningen</em> 1993; 90(34):2770–5.</li>
<li>Tabrizi SJ, Schapira AH. Secondary abnormalities of mitochondrial DNA associated with neurodegeneration. <em>Biochem Soc Symp</em> 1999; 66:99–110.</li>
<li>Schapira A, Lodi R. Assessment of in vitro and in vivo mitochondrial function in Friedreich&#8217;s ataxia and Huntington&#8217;s disease. <em>Methods Mol Biol</em> 2004; 277:293–307. doi: 10.1385/1-59259-804-8:293.</li>
<li>Choo YS, Mao Z, Johnson GV, Lesort M. Increased glutathione levels in cortical and striatal mitochondria of the R6/2 Huntington&#8217;s disease mouse model. <em>Neurosci Lett</em> 2005; 386(1):63–8. doi: 10.1016/j.neulet.2005.05.065.</li>
<li>Panov AV, Lund S, Greenamyre JT. Ca<sup>2+</sup>-induced permeability transition in human lymphoblastoid cell mitochondria from normal and Huntington&#8217;s disease individuals. <em>Mol Cell Biochem</em> 2005; 269(1–2):143–52.</li>
<li>Damiano M, Galvan L, Deglon N, Brouillet E. Mitochondria in Huntington&#8217;s disease. <em>Biochim Biophys Acta</em> 2010; 1802(1):52–61. doi: 10.1016/j.bbadis.2009.07.012.</li>
<li>Xun Z, Rivera-Sanchez S, Ayala-Pena S, Lim J, Budworth H, Skoda EM, et al. Targeting of XJB-5-131 to mitochondria suppresses oxidative DNA damage and motor decline in a mouse model of Huntington&#8217;s disease. <em>Cell Rep</em> 2012; 2(5):1137–42. doi: 10.1016/j.celrep.2012.10.001.</li>
<li>Ventura I, Russo MT, De Nuccio C, De Luca G, Degan P, Bernardo A, et al. hMTH1 expression protects mitochondria from Huntington&#8217;s disease-like impairment. <em>Neurobiol Dis</em> 2013; 49:148–58. doi: 10.1016/j.nbd.2012.09.002.</li>
<li>Turrens JF. Mitochondrial formation of reactive oxygen species. <em>J Physiol</em> 2003; 552(Pt 2):335–44. doi: 10.1113/jphysiol.2003.049478.</li>
<li>Barth E, Stammler G, Speiser B, Schaper J. Ultrastructural quantitation of mitochondria and myofilaments in cardiac muscle from 10 different animal species including man. <em>J Mol Cell Cardiol</em> 1992; 24(7):669–81.</li>
<li>Browne SE, Bowling AC, MacGarvey U, Baik MJ, Berger SC, Muqit MM, et al. Oxidative damage and metabolic dysfunction in Huntington&#8217;s disease: selective vulnerability of the basal ganglia. <em>Ann Neurol</em> 1997; 41(5):646–53. doi: 10.1002/ana.410410514.</li>
<li>Santamaria A, Perez-Severiano F, Rodriguez-Martinez E, Maldonado PD, Pedraza-Chaverri J, Rios C, et al. Comparative analysis of superoxide dismutase activity between acute pharmacological models and a transgenic mouse model of Huntington&#8217;s disease. <em>Neurochem Res</em> 2001; 26(4):419–24.</li>
<li>Min D, Kim H, Park L, Kim TH, Hwang S, Kim MJ, et al. Amelioration of diabetic neuropathy by TAT-mediated enhanced delivery of metallothionein and SOD. <em>Endocrinology</em> 2012; 153(1):81–91. doi: 10.1210/en.2011-1639.</li>
<li>Melov S, Schneider JA, Day BJ, Hinerfeld D, Coskun P, Mirra SS, et al. A novel neurological phenotype in mice lacking mitochondrial manganese superoxide dismutase. <em>Nat Genet</em> 1998; 18(2):159–63. doi: 10.1038/ng0298-159.</li>
<li>Maloyan A, Osinska H, Lammerding J, Lee RT, Cingolani OH, Kass DA, et al. Biochemical and mechanical dysfunction in a mouse model of desmin-related myopathy. <em>Circ Res</em> 2009; 104(8):1021–8. doi: 10.1161/CIRCRESAHA.108.193516.</li>
<li>Reddy PH, Charles V, Williams M, Miller G, Whetsell WO, Jr., Tagle DA. Transgenic mice expressing mutated full-length HD cDNA: a paradigm for locomotor changes and selective neuronal loss in Huntington&#8217;s disease. <em>Philos Trans R Soc Lond B Biol Sci</em> 1999; 354(1386):1035–45. doi: 10.1098/rstb.1999.0456.</li>
<li>Zabel C, Chamrad DC, Priller J, Woodman B, Meyer HE, Bates GP, et al. Alterations in the mouse and human proteome caused by Huntington&#8217;s disease. <em>Mol Cell Proteomics</em> 2002; 1(5):366–75.</li>
<li>Zourlidou A, Gidalevitz T, Kristiansen M, Landles C, Woodman B, Wells DJ, et al. Hsp27 overexpression in the R6/2 mouse model of Huntington&#8217;s disease: chronic neurodegeneration does not induce Hsp27 activation. <em>Hum Mol Genet</em> 2007; 16(9):1078–90. doi: 10.1093/hmg/ddm057.</li>
<li>Martinet W, Agostinis P, Vanhoecke B, Dewaele M, De Meyer GR. Autophagy in disease: a double-edged sword with therapeutic potential. <em>Clin Sci (Lond)</em> 2009; 116(9):697–712. doi: 10.1042/CS20080508.</li>
<li>Squitieri F, Falleni A, Cannella M, Orobello S, Fulceri F, Lenzi P, et al. Abnormal morphology of peripheral cell tissues from patients with Huntington disease. <em>J Neural Transm (Vienna)</em> 2010; 117(1):77–83. doi: 10.1007/s00702-009-0328-4.</li>
<li>Di Pardo A, Alberti S, Maglione V, Amico E, Cortes EP, Elifani F, et al. Changes of peripheral TGF-beta1 depend on monocytes-derived macrophages in Huntington disease. <em>Mol Brain</em> 2013; 6:55. doi: 10.1186/1756-6606-6-55.</li>
<li>Mielcarek M, Inuabasi L, Bondulich MK, Muller T, Osborne GF, Franklin SA, et al. Dysfunction of the CNS-heart axis in mouse models of Huntington&#8217;s disease. <em>PLoS Genet</em> 2014; 10(8):e1004550. doi: 10.1371/journal.pgen.1004550.</li>
<li>Goswami A, Dikshit P, Mishra A, Mulherkar S, Nukina N, Jana NR. Oxidative stress promotes mutant huntingtin aggregation and mutant huntingtin-dependent cell death by mimicking proteasomal malfunction. <em>Biochem Biophys Res Commun</em> 2006; 342(1):184–90. doi: 10.1016/j.bbrc.2006.01.136.</li>
<li>van der Burg JM, Bjorkqvist M, Brundin P. Beyond the brain: widespread pathology in Huntington&#8217;s disease. <em>Lancet Neurol</em> 2009; 8(8):765–74. doi: 10.1016/S1474-4422(09)70178-4.</li>
<li>Kim HD, Kim CH, Rah BJ, Chung HI, Shim TS. Quantitative study on the relation between structural and functional properties of the hearts from three different mammals. <em>Anat Rec</em> 1994; 238(2):199–206. doi: 10.1002/ar.1092380206.</li>
<li>Taverne YJ, Bogers AJ, Duncker DJ, Merkus D. Reactive oxygen species and the cardiovascular system. <em>Oxid Med Cell Longev</em> 2013; 2013:862423. doi: 10.1155/2013/862423.</li>
<li>Saft C, Zange J, Andrich J, Muller K, Lindenberg K, Landwehrmeyer B, et al. Mitochondrial impairment in patients and asymptomatic mutation carriers of Huntington&#8217;s disease. <em>Mov Disord</em> 2005; 20(6):674–9. doi: 10.1002/mds.20373.</li>
</ol>
					</div> <!-- .entry-content -->

				
				</article> <!-- .et_pb_post -->

			

			</div> <!-- #left-area -->

					</div> <!-- #content-area -->
	</div> <!-- .container -->


</div> <!-- #main-content -->


	<span class="et_pb_scroll_top et-pb-icon"></span>


			<footer id="main-footer">
				

		
				<div id="footer-bottom">
					<div class="container clearfix">
				<div id="footer-info">Reactive Oxygen Species ◊ ©2016‒2019 Cell Med Press, AIMSCI Inc. All rights reserved.</div>					</div>	<!-- .container -->
				</div>
			</footer> <!-- #main-footer -->
		</div> <!-- #et-main-area -->


	</div> <!-- #page-container -->

		<script type="text/javascript">
			</script>
	<script type='text/javascript'>
/* <![CDATA[ */
var DIVI = {"item_count":"%d Item","items_count":"%d Items"};
var et_shortcodes_strings = {"previous":"Previous","next":"Next"};
var et_pb_custom = {"ajaxurl":"http:\/\/www.aimsci.com\/ros\/html\/wp-admin\/admin-ajax.php","images_uri":"http://www.aimsci.com/ros/html/wp-content/themes/Divi/images/","builder_images_uri":"http://www.aimsci.com/ros/html/wp-content/themes/Divi/includes/builder/images/","et_frontend_nonce":"8606b18c1a","subscription_failed":"Please, check the fields below to make sure you entered the correct information.","et_ab_log_nonce":"d44b95b2c4","fill_message":"Please, fill in the following fields:","contact_error_message":"Please, fix the following errors:","invalid":"Invalid email","captcha":"Captcha","prev":"Prev","previous":"Previous","next":"Next","wrong_captcha":"You entered the wrong number in captcha.","ignore_waypoints":"no","is_divi_theme_used":"1","widget_search_selector":".widget_search","is_ab_testing_active":"","page_id":"748","unique_test_id":"","ab_bounce_rate":"5","is_cache_plugin_active":"no","is_shortcode_tracking":"","tinymce_uri":""};
var et_pb_box_shadow_elements = [];
/* ]]> */
</script>
<script type='text/javascript' src='wp-content/themes/Divi/js/custom.min_1e2ae7de.js'></script>
<script type='text/javascript' src='wp-content/themes/Divi/core/admin/js/common_1e2ae7de.js'></script>
<script type='text/javascript' src='wp-includes/js/wp-embed.min_e0220a6d.js'></script>
</body>
</html>
